{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "data = pd.read_csv('C:/Users/Bhavesh/Desktop/BE Project/Project Files/training_FINAL.csv') "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Text</th>\n",
       "      <th>Class</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>0</td>\n",
       "      <td>0</td>\n",
       "      <td>FAM58A</td>\n",
       "      <td>Truncating Mutations</td>\n",
       "      <td>Cyclin-dependent kinases (CDKs) regulate a var...</td>\n",
       "      <td>Likely Loss-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>1</td>\n",
       "      <td>CBL</td>\n",
       "      <td>W802*</td>\n",
       "      <td>Abstract Background  Non-small cell lung canc...</td>\n",
       "      <td>Likely Gain-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2</td>\n",
       "      <td>2</td>\n",
       "      <td>CBL</td>\n",
       "      <td>Q249E</td>\n",
       "      <td>Abstract Background  Non-small cell lung canc...</td>\n",
       "      <td>Likely Gain-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>3</td>\n",
       "      <td>3</td>\n",
       "      <td>CBL</td>\n",
       "      <td>N454D</td>\n",
       "      <td>Recent evidence has demonstrated that acquired...</td>\n",
       "      <td>Neutral</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>4</td>\n",
       "      <td>4</td>\n",
       "      <td>CBL</td>\n",
       "      <td>L399V</td>\n",
       "      <td>Oncogenic mutations in the monomeric Casitas B...</td>\n",
       "      <td>Loss-of-function</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "   Unnamed: 0  ID    Gene             Variation  \\\n",
       "0           0   0  FAM58A  Truncating Mutations   \n",
       "1           1   1     CBL                 W802*   \n",
       "2           2   2     CBL                 Q249E   \n",
       "3           3   3     CBL                 N454D   \n",
       "4           4   4     CBL                 L399V   \n",
       "\n",
       "                                                Text                    Class  \n",
       "0  Cyclin-dependent kinases (CDKs) regulate a var...  Likely Loss-of-function  \n",
       "1   Abstract Background  Non-small cell lung canc...  Likely Gain-of-function  \n",
       "2   Abstract Background  Non-small cell lung canc...  Likely Gain-of-function  \n",
       "3  Recent evidence has demonstrated that acquired...                  Neutral  \n",
       "4  Oncogenic mutations in the monomeric Casitas B...         Loss-of-function  "
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Unnamed: 0</th>\n",
       "      <th>ID</th>\n",
       "      <th>Gene</th>\n",
       "      <th>Variation</th>\n",
       "      <th>Text</th>\n",
       "      <th>Class</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1048</th>\n",
       "      <td>1048</td>\n",
       "      <td>1048</td>\n",
       "      <td>TSC2</td>\n",
       "      <td>R611Q</td>\n",
       "      <td>Tuberous sclerosis (TSC) is an autosomal domin...</td>\n",
       "      <td>Loss-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2239</th>\n",
       "      <td>2239</td>\n",
       "      <td>2239</td>\n",
       "      <td>PTEN</td>\n",
       "      <td>L108P</td>\n",
       "      <td>Germline mutations in the phosphatase PTEN are...</td>\n",
       "      <td>Loss-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>127</td>\n",
       "      <td>127</td>\n",
       "      <td>SF3B1</td>\n",
       "      <td>H662R</td>\n",
       "      <td>The splicing factor SF3B1 is the most commonly...</td>\n",
       "      <td>Switch-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>827</th>\n",
       "      <td>827</td>\n",
       "      <td>827</td>\n",
       "      <td>BRIP1</td>\n",
       "      <td>Deletion</td>\n",
       "      <td>BRIP1 is a DNA helicase that directly interact...</td>\n",
       "      <td>Likely Loss-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3154</th>\n",
       "      <td>3154</td>\n",
       "      <td>3154</td>\n",
       "      <td>KRAS</td>\n",
       "      <td>G12C</td>\n",
       "      <td>KRAS mutations occur in one third of human can...</td>\n",
       "      <td>Gain-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2819</th>\n",
       "      <td>2819</td>\n",
       "      <td>2819</td>\n",
       "      <td>BRCA2</td>\n",
       "      <td>Y42C</td>\n",
       "      <td>Abstract  Classification of rare missense var...</td>\n",
       "      <td>Likely Neutral</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2339</th>\n",
       "      <td>2339</td>\n",
       "      <td>2339</td>\n",
       "      <td>JAK2</td>\n",
       "      <td>T875N</td>\n",
       "      <td>Abstract The JAK2 V617F mutation present in o...</td>\n",
       "      <td>Gain-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1514</th>\n",
       "      <td>1514</td>\n",
       "      <td>1514</td>\n",
       "      <td>ALK</td>\n",
       "      <td>T1087I</td>\n",
       "      <td>In the era of personalized medicine, understan...</td>\n",
       "      <td>Neutral</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>962</th>\n",
       "      <td>962</td>\n",
       "      <td>962</td>\n",
       "      <td>ESR1</td>\n",
       "      <td>Fusions</td>\n",
       "      <td>The crucial role of recurrent gene fusions in ...</td>\n",
       "      <td>Likely Gain-of-function</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>10</td>\n",
       "      <td>10</td>\n",
       "      <td>CBL</td>\n",
       "      <td>P395A</td>\n",
       "      <td>Oncogenic mutations in the monomeric Casitas B...</td>\n",
       "      <td>Loss-of-function</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      Unnamed: 0    ID   Gene Variation  \\\n",
       "1048        1048  1048   TSC2     R611Q   \n",
       "2239        2239  2239   PTEN     L108P   \n",
       "127          127   127  SF3B1     H662R   \n",
       "827          827   827  BRIP1  Deletion   \n",
       "3154        3154  3154   KRAS      G12C   \n",
       "2819        2819  2819  BRCA2      Y42C   \n",
       "2339        2339  2339   JAK2     T875N   \n",
       "1514        1514  1514    ALK    T1087I   \n",
       "962          962   962   ESR1   Fusions   \n",
       "10            10    10    CBL     P395A   \n",
       "\n",
       "                                                   Text  \\\n",
       "1048  Tuberous sclerosis (TSC) is an autosomal domin...   \n",
       "2239  Germline mutations in the phosphatase PTEN are...   \n",
       "127   The splicing factor SF3B1 is the most commonly...   \n",
       "827   BRIP1 is a DNA helicase that directly interact...   \n",
       "3154  KRAS mutations occur in one third of human can...   \n",
       "2819   Abstract  Classification of rare missense var...   \n",
       "2339   Abstract The JAK2 V617F mutation present in o...   \n",
       "1514  In the era of personalized medicine, understan...   \n",
       "962   The crucial role of recurrent gene fusions in ...   \n",
       "10    Oncogenic mutations in the monomeric Casitas B...   \n",
       "\n",
       "                        Class  \n",
       "1048         Loss-of-function  \n",
       "2239         Loss-of-function  \n",
       "127        Switch-of-function  \n",
       "827   Likely Loss-of-function  \n",
       "3154         Gain-of-function  \n",
       "2819           Likely Neutral  \n",
       "2339         Gain-of-function  \n",
       "1514                  Neutral  \n",
       "962   Likely Gain-of-function  \n",
       "10           Loss-of-function  "
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data.sample(10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'KRAS mutations occur in one third of human cancers and cluster in several hotspots, with codons 12 and 13 being most commonly affected. It has been suggested that the position and type of amino acid exchange influence the transforming capacity of mutant KRAS proteins. We used MCF10A human mammary epithelial cells to establish isogenic cell lines that express different cancer-associated KRAS mutations (G12C, G12D, G12V, G13C, G13D, A18D, Q61H, K117N) at physiological or elevated levels, and investigated the biochemical and functional consequences of the different variants. The overall effects of low-expressing mutants were moderate compared to overexpressed variants, but allowed delineation of biological functions that were related to specific alleles rather than KRAS expression level. None of the mutations induced morphological changes, migratory abilities, or increased phosphorylation of ERK, PDK1, and AKT. KRAS-G12D, G12V, G13D, and K117N mediated EGF-independent proliferation, whereas anchorage-independent growth was primarily induced by K117N and Q61H. Both codon 13 mutations were associated with increased EGFR expression. Finally, global gene expression analysis of MCF10A-G13D versus MCF10A-G12D revealed distinct transcriptional changes. Together, we describe a useful resource for investigating the function of multiple KRAS mutations and provide insights into the differential effects of these variants in MCF10A cells.  Under physiological conditions, the small GTPase KRAS is activated in a controlled manner through the exchange of GDP for GTP upon binding of growth factors to their receptors, which then allows KRAS to bind and activate various effectors mediating a multitude of cellular effects. For example, the well-studied PI3K-PDK1-AKT, RAF-MEK-ERK, and TIAM1-RAC1 cascades regulate cell survival, cell proliferation, and cytoskeletal organization, respectively1.  Point mutations in the KRAS gene occur in approximately 30% of human cancers and are particularly common in adenocarcinomas of the pancreas, lung, and colon2. Thus far, mutant KRAS is considered an undruggable target3,4,5, although new approaches for blocking KRAS activity continue to be developed6,7, and serves as a predictor of non-responsiveness to molecularly targeted therapies such as EGFR inhibitors in lung and colon cancer8,9.  On the molecular level, KRAS mutations result in reduced intrinsic GTPase activity, which in turn leads to permanent activation of KRAS itself and downstream signalling pathways, thereby mediating malignant transformation. These single amino acid substitutions typically affect hotspots at codons 12 and 13. However, KRAS mutations also occur in codons 18, 61, 117, and 146 at low frequencies10. Of note, there is evidence that the type of KRAS mutation determines their biochemical activity and transforming capacity. For example, experiments with murine NIH/3T3 cells showed that codon 12 mutations protected from apoptosis and promoted anchorage-independent growth more strongly than codon 13 mutations11, and KRAS-K117N and A146T are associated with lower levels of GTP-bound RAS compared to G12D in transfected HEK293FT cells12. Of potential clinical relevance, patients with colorectal cancer harbouring G13D mutations were reported to respond better to anti-EGFR therapy compared to patients with G12D mutations13,14, but the biological basis for this observation is currently unclear.  We hypothesized that a systematic comparative analysis would greatly improve our understanding of the differential effects of diverse KRAS mutations on signalling pathways, cellular functions, and possibly clinical outcomes. Thus far, most in vitro studies have been performed in murine or human cell lines overexpressing the RAS mutants under investigation, which makes it difficult to assign specific biochemical or cellular effects to the respective mutation itself, since overexpression of wildtype (WT) KRAS also has transforming properties15. In addition, it remains elusive whether the specific amino acid that replaces glycine at positions 12 and 13 has an influence on the biological effects of mutant KRAS.  To address these questions, we generated isogenic MCF10A human mammary epithelial cell lines harbouring WT KRAS and eight different KRAS mutations that were all expressed at close-to-endogenous levels, and analysed these cell lines for KRAS activity, activation of downstream signalling pathways, and various cellular phenotypes including morphology, proliferation, anchorage-independent growth, and migratory properties. The different mutations clearly varied in their ability to mediate biochemical and cellular responses, and overall caused only moderate effects compared to oncogenic KRAS alleles expressed at supraphysiologic levels.  Go to: Results and Discussion Establishment of KRAS mutant MCF10A isogenic cell lines  To investigate the biochemical and functional consequences of different KRAS mutations, we chose the immortalised human mammary epithelial cell line MCF10A, since these cells are well characterised and represent a useful tool for assessing the transforming activity of oncogenes, such as RAS, ERBB2, and PLK416,17,18. We introduced the four most frequent KRAS mutations (G12D, G12V, G13D, G12C), which account for 83% of all KRAS mutations, KRAS-G13C as the second most common alteration of codon 13, three rare mutations (Q61H, A18D, K117N), and WT KRAS into MCF10A cells by lentiviral transduction (Figure 1a).  Figure 1 Figure 1 Generation of MCF10A isogenic cell lines harbouring various KRAS mutations. Six to eleven clones per cell line were obtained by seeding single cells in 96-well plates, and KRAS protein levels were compared to those of MCF10A cells transduced with an empty control vector (EV) (Figure S1). Clones with KRAS protein levels comparable to those of EV-transduced cells were selected, and side-by-side analysis by western blotting confirmed similar expression levels in all KRAS mutant clones and EV-transduced control cells (Figure 1b). In all clones, the presence of the respective KRAS mutation was confirmed by Sanger sequencing (data not shown).  To quantify the expression of the introduced cDNAs, we measured total (endogenous and exogenous) and endogenous KRAS mRNA levels by quantitative RT-PCR. In the KRAS mutant clones, we observed slightly increased levels of total KRAS mRNA with 1.5 to 3-fold higher expression compared to EV-transduced control cells (Figure 1c, left panel). For comparison, we analysed clones with substantially higher KRAS protein levels (Figure S1), resembling the high expression typically seen in, e.g., NIH/3T3 or HEK293 cells transduced or transfected with KRAS cDNAs that are often used in functional studies. KRAS mRNA expression in these cells was increased up to 60-fold compared to control cells, illustrating that forced expression of mutant cDNAs using viral vectors often results in supraphysiologic KRAS levels, which complicates assessment of the biochemical and functional consequences. Endogenous KRAS mRNA levels remained unchanged in both low and high-expressing clones, indicating that MCF10A cells did not downregulate endogenous KRAS as compensatory mechanism (Figure 1c, right panel).  GTPase activity of KRAS mutants  To characterise the different KRAS mutants, we first measured their activity by determining KRAS-GTP levels in pulldown experiments with the RAS-binding domain of RAF1 as bait in MCF10A cells starved overnight and either left untreated or stimulated with EGF for 10 minutes. Under starved conditions, we detected no GTP-bound KRAS in MCF10A cells expressing the different KRAS variants at low levels (data not shown). Following EGF stimulation, we observed minor, non-significant effects of the different KRAS variants on activated KRAS in low-expressing clones (Figure 2a). MCF10A cells expressing WT KRAS or the mutants G12D, G13D, and A18D showed similar KRAS-GTP levels as EV-transduced cells, pointing to unchanged GTPase activity. In contrast, the exchange of glycine (G) for cysteine (C) at codons 12 and 13 and valine (V) at codon 12 increased the level of GTP-bound KRAS up to 2-fold, indicating that the identity of the replacing amino acid determines the GTPase activity of oncogenic KRAS, with aspartic acid (D) having the least effect. Unexpectedly, cells expressing the rare mutations Q61H and K117N showed the highest levels of KRAS-GTP with a 5 to 6-fold increase compared to EV-transduced cells. These results reveal variable GTP binding properties among KRAS mutants, which may confer different biological functions.  Figure 2 Figure 2 GTPase activity and signal transduction properties of KRAS mutants. Signal transduction properties of KRAS mutants  To analyse the ability of KRAS mutants to activate downstream signalling pathways, we examined the phosphorylation of EGFR, ERK, RSK, PDK1, AKT, p53, and components of the mTOR-S6K1-RPS6 axis in starved and unstarved MCF10A clones (Figure 2b). The effects of different variants expressed at near-physiologic levels were overall surprisingly modest. Compared to MCF10A-EV and MCF10A-WT cells, none of the cell lines expressing mutant KRAS at low levels showed increased phosphorylation of ERK, PDK1, and AKT under starved or unstarved conditions (Figure 2c). As expected, phosphorylated ERK and AKT were readily detectable in MCF10A clones expressing mutant KRAS at high levels (Figure S2). Activation of the mTOR signalling cascade, as determined by phosphorylation of S6K1 and RPS6, was not observed under starvation, whereas addition of EGF to the low-expressing clones or cells overexpressing mutant KRAS resulted in pronounced phosphorylation of RPS6, most prominently in the context of KRAS-G13D and K117N (Figure 2c and S2). However, given the lack of PDK1 and AKT phosphorylation, activation of RPS6 was likely not induced by the PI3K-AKT-mTOR-S6K1 cascade. Instead, MCF10A-G13D cells showed increased RSK1 phosphorylation upon EGF stimulation, suggesting RSK1 as the kinase that phosphorylates RPS6 in this genetic context. In support of this hypothesis, it has been shown previously that RSK can phosphorylate RPS6 at S235 and S236 in response to serum, growth factors, and oncogenic RAS in an mTOR-independent manner19. Since we did not observe activated RSK1 in MCF10A-K117N cells, RPS6 phosphorylation is likely mediated by other signalling proteins in these cells.  Interestingly, we observed increased expression of total and phosphorylated EGFR in response to both codon 13 mutations in MCF10A cells cultured in regular growth medium, an effect that was also evident under starved conditions in MCF10A-G13D cells (Figure 2c). Furthermore, MCF10A-G13D cells displayed strong phosphorylation of p53 at serine 15, a site that is known to be phosphorylated by ATM, suggesting induction of DNA damage response signalling by G13D-mediated replicative stress20. The effects of KRAS-G13D on EGFR and p53 were strikingly different compared to mutations of codons 12, 18, 61, and 117, and could provide a biological explanation for the favourable clinical outcome of patients with G13D-positive colorectal cancer receiving anti-EGFR therapy compared to patients with codon 12 mutations13,14. For example, it is conceivable that higher EGFR levels provide a larger contact surface for therapeutic antibodies targeting EGFR, such as cetuximab and panitumumab, and that the activated senescence program predisposes EGFRhigh cells to death triggered by EGFR blockade. In accordance with this hypothesis, we observed increased sensitivity of MCF10A-G13D cells towards the EGFR inhibitors erlotinib and gefitinib compared to MCF10A-EV, G12D and K117N cells with 2 to 3-fold reduced IC50 values (Figure S3). To investigate this hypothesis further, correlating pretherapeutic EGFR and phospho-p53 levels with outcome in patients undergoing anti-EGFR therapy will be of interest.  Effects of KRAS mutants on cell morphology  We next examined whether the introduction of mutant KRAS into MCF10A cells induces changes in cell morphology. Subconfluent MCF10A cells grow as clusters, and cells display cell protrusions and lamellipodia at the edges under normal growth conditions16. This phenotype was not altered by any of the eight different KRAS mutations expressed at near-physiological levels (Figure 3a). In contrast, high levels of KRAS-G12D and G13D resulted in marked morphological changes, as the formation of cell clusters was disrupted and the cells showed a fibroblastic and spindle-like shape lacking epithelial polarisation (Figure 3a), a phenotype that is characteristic for epithelial-mesenchymal transition21.  Figure 3 Figure 3 Cellular effects of KRAS mutants. We also investigated the influence of EGF depletion on cell morphology according to KRAS mutation status. As in MCF10A-EV and MCF10A-WT cells, EGF withdrawal in MCF10A cells expressing low levels of mutant KRAS led to rounded cell clusters that completely lacked lamellipodia, and the cells assumed a cobblestone-like morphology similar to the changes observed under confluent growth conditions (Figure S4). In contrast, EGF depletion had less or no effect on the morphology of MCF10A cells overexpressing mutant KRAS, which appeared as single elongated cells with unaffected protrusions. These observations indicate that none of the KRAS mutations studied is able to induce morphological changes in MCF10A cells when expressed at physiological levels.  Effects of KRAS mutants on survival and proliferation  It is known that the withdrawal of medium additives is detrimental for the survival and proliferation of MCF10A cells22. Our experiments indicated that overexpression of mutant KRAS is able to rescue MCF10A cells from EGF depletion, whereas cells expressing KRAS mutations at physiological levels remain reliant on EGF supplementation. To further test this, we analysed the ability of different KRAS mutants to confer EGF-independent growth to MCF10A cells. Expression of the three most common KRAS mutations (G12D, G12V, G13D) or the rare K117N allele was associated with significantly increased proliferation compared to EV or WT controls, whereas the other variants were not able to compensate for EGF withdrawal (Figure 3b, right panel). When cultured in the presence of EGF, cells expressing G13C, G13D, K117N, and, to a lesser extent, codon 12 mutations, showed reduced proliferation. This finding might suggest that constitutive KRAS activity in combination with EGF leads to excessive stimulation of signalling pathways and enhanced oncogene-induced senescence, particularly in the context of KRAS-G13D and K117N. In accordance with this assumption, we found increased senescence, as measured by Î²-galactosidase staining, in MCF10A-G13D and K117N cells (and to a lesser extent MCF10A-Q61H, G12C, G12D, and G13C cells) compared to MCF10A-EV and WT cells grown in EGF-containing medium (Figure S5).  Effects of KRAS mutants on anchorage-independent growth  Next, we were interested in the impact of different KRAS mutations on anchorage-independent growth, an important characteristic of transformed epithelial cells. MCF10A cells exhibit several features of normal breast epithelium, including lack of colony formation in soft agar as a measure of anchorage-independent growth22. Compared to MCF10A clones overexpressing KRAS-G12D, G12V, and G13D, which gave very high colony numbers in soft agar, physiological expression of the different KRAS variants had only a weak effect on colony formation (Figure 3c). Whereas KRAS-G12D, G13C, G13D, and A18D did not confer anchorage-independent growth, slightly increased colony numbers were observed, in descending order, with cells expressing KRAS-K117N, Q61H, G12V, and G12C. This correlates with our observation that these clones exhibit the highest levels of GTP-bound KRAS (Figure 2a).  Effects of KRAS mutants on migration  Since enhanced cell migration is required for tumour cell invasion and metastasis, we examined the motility of EGF-deprived MCF10A clones in a monolayer wound healing assay. In contrast to MCF10A cells overexpressing KRAS-G12D, G12V, or G13D, which were able to close the wound almost entirely after 24 hours, none of the KRAS mutations expressed at physiological levels conferred increased migratory abilities compared to EV or WT controls (Figure 3d and S6). Similar to cells overexpressing mutant KRAS, EV-transduced control cells reduced the wound area by nearly 80% when EGF was added (Figure 3d and S6).  Transcriptional changes induced by KRAS mutants  To better understand the differential effects of KRAS-G12D versus G13D on signalling pathways, which might provide insights into the unequal clinical response to anti-EGFR therapy associated with these genotypes13,14, we used Illumina HumanHT-12 Expression BeadChips to profile gene expression in MCF10A-EV cells and MCF10A clones expressing physiological levels of WT KRAS, KRAS-G12D, and KRAS-G13D. Unsupervised hierarchical cluster analysis based on 2,487 genes demonstrated that WT KRAS-expressing cells were more similar to MCF10A-EV cells than cells expressing KRAS mutants, which clustered separately (Figure 4a). Notably, KRAS-G12D and KRAS-G13D-expressing cells exhibited distinct gene expression profiles that clearly separated them not only from EV-transduced cells and cells expressing WT KRAS, but also from each other.  Figure 4 Figure 4 Transcriptional changes induced by KRAS mutants. Class comparison analysis of control (KRAS-WT, KRAS-EV) versus KRAS mutant (KRAS-G12D, KRAS-G13D) samples demonstrated that both mutations caused widespread changes in the transcriptome of MCF10A cells. Using a 1.3-fold cutoff and a P value of 0.05, we found 939 genes to be significantly upregulated and 650 genes to be significantly downregulated (Table S1). To explore whether the transcriptional changes induced by KRAS-G12D and G13D in MCF10A cells were related to activated RAS signalling, we performed gene set enrichment analysis (GSEA) with the top 300 upregulated genes using the C6 oncogenic signatures collection of the Broad Institute Molecular Signatures Database (MSigDB; http://www.broadinstitute.org/gsea/msigdb/index.jsp). We identified several gene signatures derived from cells in which the RAS signalling pathway was activated through different stimuli, such as activation of ERBB2, EGFR, MEK, or KRAS itself (Figure 4b), again confirming the validity of our isogenic cell line models.  We next wanted to identify genes and pathways that are differentially regulated by KRAS-G12D and G13D. Class comparison analysis identified 1,207 significantly upregulated and 1,011 significantly downregulated genes in MCF10A-G13D cells versus MCF10A-G12D cells (Table S2). GSEA of the top 300 upregulated and the top 300 downregulated genes using the C2 curated gene sets of the MSigDB identified signatures from two independent studies that determined the gene expression profiles of luminal and basal/mesenchymal breast cancer23,24, in which KRAS-G13D was associated with the basal/mesenchymal subtype and KRAS-G12D with the luminal subtype (Figure 4c). Genes that were described to be upregulated in the basal subtype23 were highly expressed in MCF10A-G13D compared to MCF10A-G12D, including several cytokeratins (KRT6A, KRT6B, KRT16, KRT17), integrins (ITGA6, ITGB4), and others such as LAMB3, LAMC2, ANXA8, and COL17A1 (Table S2). Similarly, MCF10A-G13D cells showed high expression of genes upregulated in the closely related mesenchymal subtype23, including collagens (COL8A1), proteases (CTSC, PLAU, PLAUR, SERPINE1, SERPINE2), and others such as VIM and FN1 (Table S2). Although the basal subtype of breast cancer is associated with unfavourable clinical outcome25, it has been reported that breast basal-like cell lines are more sensitive to anti-EGFR treatment than luminal cell lines in vitro26. This differential sensitivity is reminiscent of the clinical behaviour of colorectal cancers harbouring KRAS-G13D versus G12D13,14 and suggests that KRAS-G13D induces distinct transcriptional and consequently biological changes that sensitise cancer cells of various tissue origin to anti-EGFR therapy.  In a second approach, we queried the STRING database (http://string-db.org) using the top 300 genes upregulated in MCF10A-G13D versus MCF10A-G12D cells to identify mutation-specific signalling networks. Of the 300 genes, 87 were clearly connected in one large cluster, suggesting activation of a distinct biological process (Figure 4d and Table S2). A search of the 87 genes against the Gene Ontology term â€œBiological Processâ€� using the DAVID bioinformatics tool (http://david.abcc.ncifcrf.gov/home.jsp) revealed several processes that are associated with cytokine-induced cell migration (Figure 4e).  The top upregulated cytokine genes in MCF10A-G13D cells were CXCL1, IL1B, and IL8 with a more than 10-fold increase in transcription compared to MCF10A-G12D cells (Table S2). Cytokines have been implicated in RAS-driven cancer previously, as activated RAS positively regulates the expression of cytokines to induce autocrine and paracrine signals that promote tumourigenesis27. For example, CXCL1 (also known as Gro-1) was transcriptionally upregulated upon RAS activation and rendered essential for survival and malignant transformation in ovarian epithelial cells28. In a murine model of Kras-G12D-driven lung adenocarcinoma, the mouse homolog of IL8 was increased in lung tissue homogenates, and treatment of these mice with an antibody targeting Cxcr2, the receptor for Il8 and Cxcl1, significantly reduced lung tumour burden29. High expression of IL8 was also identified in human lung adenocarcinomas with mutant KRAS, and was associated with poor clinical outcome30. Furthermore, increased expression and secretion of IL8 was recently shown to be essential for the induction of protease-dependent invasion and metastasis of RAS mutant melanoma cells upon BRAF inhibition31. The involved proteases were MMP1, PLAU, and PLAUR31, of which the latter two were also upregulated in MCF10A-G13D cells (Table S2). Thus, while previous investigations into the effects of mutant KRAS on cytokine signalling have primarily focused on codon 12 mutations, reflecting the substantially higher prevalence of these alleles compared to mutations affecting other codons, our gene expression data suggest that the impact of KRAS-G13D is even more pronounced. However, there may be tissue-specific differences in the effects of various KRAS mutations on cytokine signalling as well as cell-extrinsic contributions from the tumour microenvironment in mouse models and human patients that are not accounted for by our in vitro experimental system.  Analysis of the entire complement of genes significantly upregulated in MCF10A-G13D versus MCF10A-G12D cells using the Ingenuity Pathway Analysis tool (http://www.ingenuity.com) confirmed several of the above mentioned cellular processes and signalling cascades (Figure 4f). For example, ILK signalling includes integrins, which are a component of the gene signature associated with the basal subtype of breast cancer that was identified through GSEA (Figure 4c); IL8 signalling was also identified by STRING analysis (Figure 4d and e); and activated p53 signalling was identified through detection of phosphorylated p53 by western blotting (Figure 2c).  Go to: Conclusions In summary, we established isogenic cell lines that allowed characterization of eight different KRAS mutations expressed at physiological levels. Our data support the conclusion that KRAS mutations affecting different codons have distinct effects on KRAS activity, signal transduction, transcriptional programs, and transforming capacity (Table 1). The overall effects in MCF10A cells were weaker than those of KRAS variants expressed at supraphysiological levels, which allowed discrimination of biological functions related to specific mutations rather than KRAS expression level. While none of the mutations tested induced obvious morphological changes or migratory abilities, clear differences were identified in the induction of EGF-independent proliferation and anchorage-independent growth. Finally, KRAS-G13D induced EGFR expression, senescence, and a distinct pattern of transcriptional changes that may help explain the improved response of patients with G13D-positive cancers to anti-EGFR therapies.  Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally associated with poor response to standard therapies1â€“3. Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP4 and the absence of known allosteric regulatory sites. Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis5,6. With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concentration. This gives GTP an advantage over GDP7 and increases the proportion of active GTP-bound Ras. Here we report the development of small molecules that irreversibly bind to a common oncogenic mutant, K-Ras(G12C). These compounds rely on the mutant cysteine for binding and therefore do not affect the wild-type protein. Crystallographic studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region. Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favour GDP over GTP and impairing binding to Raf. Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.  To target K-Ras(G12C) we took advantage of the unique nucleophilicity of cysteine thiols by exploring cysteine-reactive small molecules. This strategy has the added advantage of allowing selectivity for the mutant over wild-type K-Ras. Notably, the mutant Cys 12 sits in close proximity to both the nucleotide pocket and the switch regions involved in effector interactions (Fig. 1a). To identify a chemical starting point, we used a disulphide-fragment-based screening approach called tethering8. We screened a library of 480 tethering compounds against K-Ras(G12C) in the GDP state using intact protein mass spectrometry9,10 (see Methods and Extended Data Table 1). Fragments 6H05 (94 Â± 1% (mean Â± s.d.)) and 2E07 (84.6Â± 0.3%)gave the greatest degree of modification (Fig. 1b, c). Reaction with wild-type K-Ras, which contains three native cysteine residues, was not detected. Conversely, both compounds modify the oncogenic G12C mutant of the highly homologous protein H-Ras11,12 (Fig. 1b). Binding was not diminished by 1 mM GDP in the presence of EDTA, suggesting that the compounds bind in an allosteric site not overlapping with GDP. Pre-loading of K-Ras with GTP significantly impairs modification by both compounds, indicating incompatibility between compound binding and the active conformation of Ras.  Figure 1 Figure 1 Tethering compounds selectively bind to oncogenic K-Ras(G12C) We chose to pursue the top fragment, 6H05, by investigating structureâ€“activity relationships for several analogues8 (Fig. 1c, see Methods). Some changes such as replacing the thioether with a methylene group reduced binding (1, relative potency <0.1). However, other changes such as modification of the tethering linker enhanced binding (6, relative potency 4.2). Having identified a tractable chemotype, we pursued a co-crystal structure to enable structure-based design. To facilitate uniform labelling on Cys 12 we used a K-Ras construct lacking other cysteines, K-Ras(C51S/C80L/C118S) (known as Cys-light), which showed minimal effects on overall protein structure (G12C versus Cys-light, root mean squared deviation (r.m.s.d.) (CÎ±) = 0.33 Ã…). Using this construct, we obtained a 1.29 Ã… co-crystal structure of 6 bound to K-Ras(G12C) in the GDP state (Fig. 1d). Compound 6 does not bind in the nucleotide pocket but extends from Cys 12 into an adjacent pocket composed largely of switch-II. This fully formed pocket is not apparent in other published structures of Ras, although a groove is visible in some cases13 (Extended Data Fig. 1b), and previous studies have suggested the presence of an allosteric site in this region14. We refer to the compound binding region as the switch-II pocket (S-IIP).  The S-IIP is located between the central Î²-sheet of Ras, and the Î±2-(switch-II) and Î±3-helices. Well-defined electron density shows the location of 6 deep within the S-IIP (432 Ã…2 interface; Extended Data Table 2) and confirms the disulphide linkage between 6 and Cys 12 (Fig. 1e; Fo âˆ’ Fc at 2.5Ïƒ). The hydrophobic dichlorophenyl group of 6 makes several hydrophobic contacts (Fig. 1f). Glu 99 and Gly 60 form direct hydrogen bonds to 6. Whereas switch-II shows significant reordering to form the S-IIP, the conformation of switch-I is unchanged from the GDP-bound state. Structural analysis also suggested the presence of sub-pockets in S-IIP that might enable design of more potent inhibitors (Extended Data Fig. 2a, (o-) and (p-)).  Rather than continue with disulphide-based compounds we turned to carbon-based electrophiles, acrylamides and vinyl sulphonamides, which are still chemoselective yet provide irreversible cysteine bond formation. We synthesized nearly 100 analogues guided by iterative structural evaluation to yield substantial improvements in potency (Fig. 2aâ€“c and Supplementary Table 1). Owing to the irreversible nature of binding, potency was assessed by time-dependent modification of the protein, initially 200 Î¼M compound for 24 h. A shift to 10 Î¼M compound was necessary to differentiate optimized analogues (Extended Data Table 3 and Extended Data Fig. 3a), reaching the detection limit of the assay (4 Î¼M K-Ras). Although vinyl sulphonamides generally performed better by these metrics, probably owing to higher reactivity15, we obtained highly effective acrylamides as well (Fig. 2b).  Figure 2 Figure 2 Electrophilic compounds bind to S-IIP of K-Ras(G12C) and disrupt switch-I and switch-II To evaluate the off-target specificity of our most potent acrylamide 12, we used intact protein mass spectrometry to monitor simultaneously for modification of K-Ras(G12C) and bovine serum albumin (BSA) (one free cysteine) in a single mixture. Treatment with 12 resulted in the modification of K-Ras(G12C) but not BSA (Extended Data Fig. 3b), although both react with Ellmanâ€™s reagent (also known as DTNB) (Extended Data Fig. 3c). Optimized electrophiles show no detectable modification of wild-type K-Ras (Extended Data Fig. 3d). Fragments lacking the electrophile did not impair binding of compound 12 to K-Ras(G12C) (data not shown), suggesting limited binding to S-IIP in K-Ras proteins lacking the G12C mutation.  Overlaying multiple co-crystal structures revealed that the compounds follow a similar trajectory through the pocket and project functional groups into the (o-) and (p-) sub-pockets (Fig. 2c and Extended Data Fig. 2). Despite considerable variation at the terminal phenyl ring, the compounds satisfy similar hydrophobic interactions, supporting the crucial role of this region of the S-IIP (Fig. 2c and Extended Data Fig. 2).  In the active state of Ras, residues from switch-II entirely fill the S-IIP (Fig. 2d). Co-crystal structures with tethering compound 6 and the electrophile 8 exhibit displacement of switch-II relative to the active conformation (Fig. 2e, f). Comparison of these co-crystal structures revealed distinct effects on switch-I and switch-II. Tethering compound 6 induces a small displacement of switch-II with little effect on switch-I. By contrast, electrophile 8 induces a more pronounced displacement of switch-II that results in disordering of switch-I and a lack of density for the metal ion.  Proper metal coordination is crucial for tight nucleotide binding16, with mutation of magnesium-coordinating residues Ser 17 or Asp 57 leading to a preference for GDP over GTP17,18. Many of our structures with carbon-based electrophiles show disordering of switch-I and a lack of density for the metal ion (Extended Data Table 4). On the basis of these observations, we predicted that S-IIP binding compounds might differentially affect nucleotide affinities leading to a preference for GDP over GTP. To test this prediction, we carried out EDTA-catalysed off-exchange reactions with 2â€²-deoxy-3â€²-O-(N-methylanthraniloyl) (mant)-dGDP, while titrating unlabelled GDP or GTP (Fig. 3). In the absence of inhibitor, K-Ras(G12C) shows a slight preference for GTP (relative affinity 0.6 Â± 0.2), as reported previously for H-Ras7. However, in the presence of either 8 or 12, GTP affinity is significantly decreased relative to GDP (relative affinity 3.9 Â± 0.6 (8) and 3.5 Â± 0.8 (12), P = 0.004, t-test) (Fig. 3b, c). The catalysis of nucleotide exchange by EDTA suggests that although our compounds may subtly affect metal binding leading to changes in nucleotide affinity, Mg2+ is not precluded even when the S-IIP is occupied.  Figure 3 Figure 3 Compound binding to S-IIP changes nucleotide preference of K-Ras from GTP to GDP Structural analysis also predicts that the function of the exchange factor SOS would be compromised by compound binding to S-IIP19. Indeed, treatment of K-Ras(G12C) with either 8 or 12 blocks SOS-catalysed nucleotide exchange (Fig. 4aâ€“e). As shown above, these compounds do not impair EDTA-catalysed GDP exchange.  Figure 4 Figure 4 Compounds block K-Ras(G12C) interactions, decrease viability and increase apoptosis of G12C-containing lung cancer cell lines In the active conformation of Ras, Gly 60 and Thr 35 make crucial contacts with the Î³-phosphate20. Conservative mutations of Thr 35 (T35S) or Gly 60 (G60A) markedly impair effector binding21,22. Our compounds occupy the required position for Gly 60 in the active conformation and displace this residue to varying degrees, with larger distances correlating with disordering of switch-I (Extended Data Table 4 and Extended Data Fig. 4). This analysis led to the prediction that our compounds would disrupt the conformation of the GTP state of Ras and impair interactions with effectors such as Raf. We measured Rasâ€“Raf association in two K-Ras(G12C)-mutant lung cancer cell lines, H1792 and H358, using co-immunoprecipitation. As predicted, treatment with 12 decreased the association of B-Raf and C-Raf with Ras (Fig. 4f). Although this effect is evident in both cell lines, it is most pronounced in H1792 cells, which express lower levels of Ras23.  Despite limited compound potency, we speculated that the genotype-specificity might afford a therapeutic window for the targeted inhibition of K-Ras(G12C) in cellular models. Therefore, we compared the effects of 12 in a small collection of genetically annotated lung cancer cell lines. As expected, the group of cell lines containing G12C mutations (H1792, H358, H23 and Calu-1) showed decreased viability (Fig. 4g and Extended Data Fig. 5; P = 0.005, t-test) and increased apoptosis (Fig. 4h; P <0.001, t-test) relative to the group lacking this mutation (H1437, H1299 and A549) after treatment with 12. The highly sensitive H1792 cells show low levels of K-Ras GTP (Extended Data Fig. 5b), consistent with preferential binding of our inhibitors to K-Ras GDP (Fig. 1b), and they are highly K-Ras dependent24 (Extended Data Fig. 5c). Notably, both K-Ras-dependent (A549) and -independent (H1299) cell lines lacking G12C were insensitive to compound 12 (Fig. 4g, h and Extended Data Fig. 5). The half-maximum effective concentration (EC50) for compound 12 in H1792 cells (0.32 Â± 0.01 Î¼M) is tenfold lower than that of compound 10 (3.2 Â± 0.4 Î¼M), consistent with in vitro K-Ras labelling efficiency (Fig. 2b, 100% versus 14% labelling at 24 h). Notably, the highly related electrophile-containing compound 17, which does not modify K-Ras(G12C) in vitro (Extended Data Fig. 2c, 0% at 24 h), shows no effect on H1792 cell viability. Overall, our cellular data provide a proof-of-concept for the genotype-specific use of S-IIP binding compounds in K-Ras(G12C)-driven cancer.  Using a structure-guided approach to target the G12C mutant of K-Ras, we have identified a new allosteric pocket, S-IIP, in this protein, and we have used that pocket to develop irreversible, mutant-specific inhibitors of Ras function. The S-IIP is not visible in other structures of Ras, and thus it is probably highly dynamic when GDP is bound, until initial encounter with our compounds. Compound binding to S-IIP impairs Ras function through two distinct mechanisms. First, by shifting the relative nucleotide affinities of Ras to favour GDP over GTP, the compounds should lead to accumulation of Ras in the inactive state. Notably, the two most effective cellular GTPase inhibitors, the natural products brefeldin A and YM-254890, both bind to and stabilize a GDP-bound state of their respective GTPases25,26. So far, published small molecules that bind Ras have not shown this nucleotide preference14,27â€“29. Second, compounds occupying S-IIP diminish interactions with effectors and regulatory proteins. This should act to diminish signalling by K-Ras further. Despite the need for continued chemical optimization of our compounds for future assessment in vivo, initial evaluation of our compounds in lung cancer cell lines suggests allele-specific impairment of K-Ras function. On the basis of these data and our understanding of the biochemical mechanism of the inhibitors, we are confident that our findings can serve as the starting point for drug-discovery efforts targeting K-Ras(G12C) and eventually other alleles of K-Ras.  Go to: METHODS Protein expression and purification  Hexahistidine-tagged recombinant human K-Ras (isoform 2, residues 1â€“169, based on construct used for PDB accession 3GFT) was transformed into Escherichia coli (BL21 (DE3)). After the bacterial growth to an attenuance (D) at 600 nm of 0.4â€“0.6 in Terrific Broth containing 30 mg lâˆ’1 kanamycin at 37 Â°C, induction was carried out at 18 Â°C using 0.5 mM isopropyl-Î²-D-thiogalactoside (IPTG), and growth was continued at 18 Â°C for about 18 h. The bacteria were collected by centrifugation, and the obtained pellet either stored at âˆ’80 Â°C or used freshly for the subsequent steps.  The pellet was resuspended in lysis buffer (500 mM NaCl, 20 mM Tris, pH 8.0 and 5 mM imidazole) containing protease inhibitor cocktail (Roche complete EDTA free), the bacteria were lysed by microfluidizer, 2 mM Î²-mercaptoethanol (Î²ME) (final) was added and cell debris was removed by ultracentrifugation. The supernatant was incubated for 1 h with Co-affinity beads (Clontech, ~2 ml bed volume per 1 l initial culture), the loaded beads were then washed with lysis buffer containing 2 mM Î²ME and the protein was eluted with buffer containing 125â€“250 mM imidazole. The hexahistidine tag was then cleaved using hexahistidine-tagged TEV-protease (1 mg recombinant TEV per 25 mg crude K-Ras, 1 mg GDP added per 20 mg crude K-Ras) while dialysing against a buffer containing 300 mM NaCl, 20 mM Tris, pH 8.0, 5 mM imidazole, 1 mM dithiothreitol (DTT) and 0.5 mM EDTA. The cleaved protein was then diluted fivefold with low-salt buffer (50 mM NaCl, 20 mM Tris, pH 8.0), incubated with Ni-agarose beads (Qiagen) to remove uncleaved protein and protease, and 5 mM MgCl2 and GDP was added to load the metal and nucleotide site of K-Ras fully.  The crude protein was then purified by ion-exchange chromatography (HiTrap Q HP column, salt gradient from 50 to 500 mM NaCl) to give the partially purified protein, commonly in the following buffer (~230 mM NaCl, 20 mM Tris, pH 8.0, small amounts of GDP). At this point the protein was either fully labelled with the desired compound (incubation overnight with an excess of compound at 4 Â°C, labelling checked by mass spectrometry analysis), frozen down and stored at âˆ’80 Â°C, or used for further purification.  The last purification step for the labelled or unlabelled protein was gel filtration using either a Superdex 75 or 200 column (10/300 GL) with the following buffer: 20 mM HEPES, pH 7.5, 150 mM NaCl and 1 mM DTT (for the unlabelled proteins). The freshly prepared and purified protein was then concentrated to 5â€“20 mg mlâˆ’1 and used for the X-ray crystallography trays.  Sequences for the different K-Ras constructs were generally codon-optimized and synthesized by DNA2.0 using the pJexpress411 vector. For the X-ray structures of compound-labelled K-Ras(G12C), a cysteine-light mutant was used (K-Ras (G12C/C51S/C80L/C118S)) to enable more uniformly labelled species.  Purification and labelling of full-length forms of the protein as well as H-Ras was carried out analogously. Nucleotide exchange for crystallographic samples was carried out following published procedures4,27,30.  Tethering screen  Untagged recombinant K-Ras(G12C) (1â€“169) at 1 Î¼M was reacted with 100 Î¼M fragment and 100 Î¼M Î²ME in 20 mM HEPES, pH 7.5, 150 mM NaCl and 10 mM EDTA for 1 h at ambient temperature. The extent of modification was assessed by electrospray mass spectrometry using a Waters LCT-Premier LC/ESI-MS. By setting a threshold of â‰¥60% modification, we achieved a hit rate of 1.9%.  Determination of DR50 and relative potency of fragments  The DR50 is determined by titrating fragment while maintaining a constant Î²ME concentration, in this case 200 Î¼M Î²ME. DR50 = [Î²ME]/[fragment], at which 50% modification is achieved by total protein mass spectrometry10. Relative potency is reported as: fragment DR50/6H05 DR50.  Mass spectrometric screen for extent of irreversible labelling  Untagged recombinant K-Ras(G12C) (1â€“169) at 4 Î¼M was reacted with inhibitors at 200 Î¼M or 10 Î¼M (2% (v/v) dimethylsulphoxide (DMSO) final) in 20 mM HEPES, pH 7.5, 150 mM NaCl and 1 mM EDTA. After 24 h, 10-Î¼l aliquots were removed and the reactions were stopped by the addition of 1 Î¼l 2% (v/v) formic acid. For the BSA specificity experiment, 16 Î¼M BSA was included in the mixture and the reaction was analysed at 6 h, without acid treatment. A similar mixture of K-Ras(G12C) and BSA was treated instead with 200 Î¼M Ellmanâ€™s reagent and labelling was analysed after 5 min. In all cases, the extent of modification was assessed by electrospray mass spectrometry using a Waters Acquity UPLC/ESI-TQD with a 2.1 Ã— 50 mm Acquity UPLC BEH300 C4 column.  Plate-based assay to determine relative affinity of K-Ras for GDP or GTP  The corresponding recombinantly expressed, full-length K-Ras protein (G12C mutant or G12C mutant labelled fully with either compound 8 or 12) at about 10 Î¼M concentration was incubated with 200 Î¼M mant-dGDP (Jena Biosciences) in the presence of 2.5 M EDTA. After 1 h at room temperature, MgCl2 to a final concentration of 10 mM was added. The protein was then run through a NAP-5 column to remove free nucleotide. The concentration of the obtained protein was determined by Bradford assay and the protein was then used in the described plate-based assay.  For the assay, 10 Î¼l of the prepared protein in reaction buffer (20 mM HEPES, pH 7.5, 150 mM NaCl, 1 mM DTT and 1 mM MgCl2) was added to a well of a low volume black bottom plate (Corning, 3676). The fluorescence intensity was measured on a spectramax M5 plate reader (Molecular Devices, 360 nm excitation, 440 nm emission) to provide a value used in later normalization. Then, 5 Î¼l of an EDTA solution with the indicated nucleotide (GDP or GTP) was added to each well and the reaction mix was allowed to equilibrate for 2 h at room temperature. Measurement of the fluorescent intensity at this time provided the end point. Samples were measured in duplicates for each experiment. In the final mix the concentrations were the following: protein (1 Î¼M), EDTA (5 mM) and nucleotide (as indicated, titrated in 2.5-fold dilution series, 15 points). Curves show results from one representative experiment, the column graph shows the averaged data from three experiments, with errors representing s.d. For the determination of IC50, a sigmoidal curve fit was used for each nucleotide (Prism software).  Nucleotide exchange rates for compound-bound K-Ras(G12C)  The experiment was carried out similarly as described for the nucleotide affinity assay using the same plate set-up and plate reader. In brief, the respective full-length proteins were loaded with mant-dGDP (see above). For the assay, 10 Î¼l of the prepared protein (1 Î¼M final) in reaction buffer was added to the wells. To start the reaction, 5 Î¼l of SOS (1 Î¼M final), EDTA (5 mM final), or buffer was added and the fluorescence monitored for 5 h at 90-s intervals. Half-lives were determined using Prism software (single-exponential decay fit).  Cell culture  H23, H358, H1299, H1437, H1792, Calu-1 and A549 (ATCC) were cultured in DMEM with 10% FBS.  Viability assays  Cells were plated in 96-well plates at 2,000 cells per well in 90 Î¼l DMEM with 10% FBS and allowed to attach for 24 h. Cells were treated by the addition of 10 Î¼l 100 Î¼M compound (or half-log dilutions thereof) or vehicle (0.1% DMSO final). After 72 h, media was exchanged and plates were analysed using CellTiter-Glo Luminescent Cell Viability Assay (Promega).  Apoptosis assays  The Annexin V-FITC Apoptosis Detection Kit I (BD Biosciences) was used to detect apoptotic cells. Cell lines were plated in 6-well plates at ~50% confluence, and after 24 h cells were treated with the given compound for 48 h. Subsequently, cells were washed with PBS, trypsinized and resuspended in 150 Î¼l annexin-V binding buffer. Finally, cells were stained with annexin-V-FITC and propidium iodide and incubated in the dark before analysis on a FACS LSRII Flow Cytometer (Beckman Coulter). Data were collected using FACSDiva analysis software (Beckman Coulter). At least 10,000 cells were measured per individual sample. Results are given as Î”annexin-V/propidium-iodide unstained cells of untreated (DMSO) versus treated samples.  siRNA knockdown  Cells were plated either in 96-well plates or 6-well plates 24 h before transfection with short interfering RNA (siRNA). The siRNA constructs were diluted (10â€“20 nM) in RNAiMax-Lipofectamine (Life Technologies) containing OPTI-MEM media, and after 20 min of incubation the mixture was added drop-wise to the cells. After 72 h of incubation, cells were either lysed for immunoblotting experiments (K-Ras antibody, Sigma 3B10-2F2; actin antibody, Cell Signaling Technology 4970) or subjected to CellTiterGlo assays (Promega) for proliferation analysis. The KRAS siRNA guide and passenger strand sequences used were as follows: guide, 5â€²-ACUGUACUCCUCUUGACCUGCU-3â€²; passenger, 5â€²-CAGGUCAAGAGGAGUACAGUUA-3â€².  Immunoprecipitation  Cells were treated at given conditions in 10-cm plates and lysed in phosphatase and protease inhibitor containing lysis buffer. Lysates were mixed with 4 Î¼l primary Ras antibody (Abcam, EPR3255), and incubated with rocking overnight at 4 Â°C. Protein G agarose beads were added to the mixture, and after 3 h beads were pelleted, washed with lysis buffer, and after resuspension in loading buffer and heating at 95 Â°C for 5 min were analysed by SDSâ€“PAGE followed by immunoblot. B-Raf (Santa Cruz, F-7) and C-Raf (Cell Signaling Technology, 9422) antibodies were used to detect the individual proteins.  Go to: Extended Data Extended Data Figure 1  An external file that holds a picture, illustration, etc. Object name is nihms646437f5.jpg Comparison of co-crystal structure of 6 with K-Ras(G12C) to known structures of Ras a, Compound 6 (cyan) bound in the S-IIP of K-Ras(G12C). b, Compound 6 (aligned and overlayed) with GDP-bound wild-type H-Ras showing groove near S-IIP (PDB accession 4Q21)13. c, Clash of compound 6 (aligned and overlayed) with GTPÎ³S-bound K-Ras(G12D), which shows glycerol molecule adjacent to S-IIP (PDB accession 4DSO)27.  Extended Data Figure 2  An external file that holds a picture, illustration, etc. Object name is nihms646437f6.jpg Additional insights into Ras-compound binding and its biochemical effects a, Compound 6 (cyan) is attached to Cys 12 of K-Ras(G12C) and extends into an allosteric binding pocket beneath switch-II (blue), the S-IIP. The binding pocket in K-Ras (surface representation of the protein shown) fits 6 tightly and includes hydrophobic sub-pockets (dashed lines). An extension of the pocket is occupied by water molecules (red spheres) and might provide space for modified compound analogues. bâ€“d, X-ray crystallographic studies of K-Ras(G12C) bound to several additional electrophilic analogues (14, 15 and 16, respectively) reveal a similar overall binding mode. All compounds follow a similar trajectory from Cys 12 into S-IIP but show some variability in the region of the piperidine/piperazine. The respective switch-I regions of the protein can be disordered. e, Overlay of the two different crystal forms of K-Ras(G12C) bound to 9 (space group C2 (grey) and P212121 (cyan)) is shown. The ligand orientation and conformation shows minimal changes, whereas switch-II of the protein appears disordered in the C2 form and atypical in the P212121 form. f, An overlay for several compounds including the disulphide 6 is shown (16-green, 6-yellow, 7-orange, 9-cyan). Key hydrophobic residues are labelled and hydrophobic interaction between the compounds and the (p-) or (o-) sub-pockets are indicated by dashed lines.  Extended Data Figure 3  An external file that holds a picture, illustration, etc. Object name is nihms646437f7.jpg Analysis of compound labelling rate and in vitro specificity a, Percentage modification of K-Ras(G12C) by compounds 9 and 12 over time (n = 3, error bars denote s. d.). b, Selective single labelling of K-Ras(G12C) by compound 12 in the presence of BSA. c, Quantitative single labelling of BSA and multiple labelling of K-Ras(G12C) by DTNB. d, Comparison of modification of K-Ras(G12C) and wild-type by 12 (n = 3, error bars denote s.d.).  Extended Data Figure 4  An external file that holds a picture, illustration, etc. Object name is nihms646437f8.jpg Comparison of active conformation and compound bound form of Ras a, X-ray crystal structure of the active conformation of H-Ras(G12C) with GMPPNP shows interactions of the Î³-phosphate with key residues (Tyr 32, Thr 35 and Gly 60) that hold switch-I (red) and switch-II (blue) in place. The inactive GDP-bound structure of H-Ras(G12C) reveals the absence of these key interactions and increased distances between these residues and the position of the Î³-phosphate (positions from GMPPNP structure indicated by spheres) coinciding with large conformational changes in both switch regions. In the P212121 crystal form of 9 bound to K-Ras(G12C) GDP switch-I is ordered (often disordered by compounds, see Extended Data Table 4), but the structure shows displacement of the Î³-phosphate-binding residues beyond their positions in the inactive state. b, As indicated by the X-ray structures, removal of the Î³-phosphate leads to relaxation of the â€˜spring-loadedâ€™ Ras-GTP back to the GDP state, with opening of switch-II. Compound binding moves switch-II even further away and interferes with GTP binding itself.  Extended Data Figure 5  An external file that holds a picture, illustration, etc. Object name is nihms646437f9.jpg Inhibitor sensitivity, K-Ras GTP levels and K-Ras dependency of lung cancer cell lines a, Percentage viability after treatment for 72 h with 12 relative to DMSO (n = 3 biological replicates, error bars denote s.e.m.). b, K-Ras GTP levels determined by incubating lysates with glutathione S-transferase (GST)-tagged RBD (Ras-binding domain of C-Raf) immobilized on glutathione beads (n = 3 biological replicates). c, Viability of cell lines evaluated 72 h after transfection with KRAS siRNA (n = 3 biological replicates). d, K-Ras immunoblot showing knockdown after KRAS siRNA (n = 3 biological replicates).  Extended Data Table 1  Hit fragments and percentage modification from the primary tethering screen  Fragment structure\\tFragment number\\tPercent Modification\\t An external file that holds a picture, illustration, etc. Object name is nihms646437t1.jpg 2C10\\t60%\\t An external file that holds a picture, illustration, etc. Object name is nihms646437t2.jpg 2D04\\t60%\\t An external file that holds a picture, illustration, etc. Object name is nihms646437t3.jpg 2D05\\t60%\\t An external file that holds a picture, illustration, etc. Object name is nihms646437t4.jpg 2E07\\t70%\\t An external file that holds a picture, illustration, etc. Object name is nihms646437t5.jpg 3C09\\t60%\\t An external file that holds a picture, illustration, etc. Object name is nihms646437t6.jpg 4C09\\t60%\\t An external file that holds a picture, illustration, etc. Object name is nihms646437t7.jpg 5B03\\t60%\\t An external file that holds a picture, illustration, etc. Object name is nihms646437t8.jpg 5F10\\t65%\\t An external file that holds a picture, illustration, etc. Object name is nihms646437t9.jpg An external file that holds a picture, illustration, etc. Object name is nihms646437t10.jpg 6H05\\t95% Extended Data Table 2  Overview of obtained and previously published co-crystal structures and their respective compoundâ€“protein binding interfaces  Compound\\tPDB-code\\tSpace group Unit cell\\tCompound-protein interface (Ã…2)\\tSolvent Accessible Surface Area/ Buried Surface Area in structure (Ã…2) An external file that holds a picture, illustration, etc. Object name is nihms646437t11.jpg  4\\t4LV6\\tP1 (33,39,63Ã…) (77,81,77Â°)\\t421\\t601/531 (chain A) An external file that holds a picture, illustration, etc. Object name is nihms646437t12.jpg  6\\t4LUC\\tP1 (33,39,62Ã…) (78,82,78Â°)\\t432\\t629/548 (chain A) An external file that holds a picture, illustration, etc. Object name is nihms646437t13.jpg  7\\t4M1O\\tC2 (68,84,86Ã…) (90,111,90Â°)\\t407\\t625/489 (chain B) An external file that holds a picture, illustration, etc. Object name is nihms646437t14.jpg  8\\t4LYF\\tC2 (68,84,87Ã…) (90,111,90Â°)\\t359\\t565/430 (chain B) An external file that holds a picture, illustration, etc. Object name is nihms646437t15.jpg  9\\t4LYH\\tC2 (68,84,87Ã…) (90,111,90Â°)\\t408\\t620/507 (chain B) An external file that holds a picture, illustration, etc. Object name is nihms646437t16.jpg  9\\t4LYJ\\tP212121 (39,43,88Ã…) (90,90,90Â°)\\t412\\t632/517 (chain A) An external file that holds a picture, illustration, etc. Object name is nihms646437t17.jpg  11\\t4M21\\tC2 (68,84,87Ã…) (90,111,90Â°)\\t297\\t574/372 (chain B) An external file that holds a picture, illustration, etc. Object name is nihms646437t18.jpg  13\\t4M1S\\tC2 (68,84,86Ã…) (90,111,90Â°)\\t346\\t580/442 (chain B) An external file that holds a picture, illustration, etc. Object name is nihms646437t19.jpg  14\\t4M1T\\tC2 (68,83,85Ã…) (90,111,90Â°)\\t355\\t579/444 (chain B) An external file that holds a picture, illustration, etc. Object name is nihms646437t20.jpg  15\\t4M1Y\\tC2 (68,84,87Ã…) (90,111,902)\\t387\\t599/480 (chain B) An external file that holds a picture, illustration, etc. Object name is nihms646437t21.jpg  16\\t4M22\\tC2 (71,83,87Ã…) (90,109,90Â°)\\t324\\t535/399 (chain B) An external file that holds a picture, illustration, etc. Object name is nihms646437t22.jpg  DCAI K-Ras(G12D) Ref. 27\\t4DST\\tH3 (79,79,78.6Ã…) (90,90,120Â°)\\t204\\t403/252 An external file that holds a picture, illustration, etc. Object name is nihms646437t23.jpg  â€œ13â€� K-Ras(G12V), Ref. 28\\t4EPY\\tP212121 (39,41,92Ã…) (90,90,90Â°)\\t301\\t606/370 Extended Data Table 3  Extent of labelling after 24h at 10 Î¼M inhibitor  Vinyl sulphonamide inhibitors\\tModification (%)\\tAcrylamide inhibitors\\tModification (%) 7\\t50\\t10\\t14 8\\t87\\t11\\t28 9\\t100\\t12\\t100 13\\t21\\t16\\t5 14\\t32\\t17\\t0 15\\t81\\t\\t Extended Data Table 4  Increased distance (Ã…) between position-12 CÎ± and Gly 60 CÎ± correlates with disordering of switch-I  GDP-bound\\t12 CÎ± to 60 CÎ± distance (Ã…)\\tSwitch-I\\tMetal ion WT\\t8\\t\\tMg H-Ras(S17N)*\\t8.1\\tdisordered\\tCa 16\\t8.3\\t\\tMg 4\\t9\\t\\tCa 6\\t9.1\\t\\tCa 13\\t11.1\\tdisordered\\t- 9 (P212121)â€\\xa0\\t11.2\\tatypical\\tMg 14\\t11.6\\tdisordered\\t- 8\\t11.9\\tdisordered\\t- 15\\t12\\tdisordered\\t- 9 (C2)â€\\xa0\\t12.7\\tdisordered\\t- 7\\t12.8\\tdisordered\\t- 11\\tswitch-II disordered\\tdisordered\\t- GTP-bound\\t12 CÎ± to 60 CÎ± distance (Ã…)\\tSwitch-I\\tMetal ion H-Ras(G12C)\\t3.8\\t\\tMg K-Ras(WT)â€¡\\t3.9\\t\\tMg RaplA with CRAF RBDÂ§\\t3.9\\t\\tMg *PDB accession 3LO5. â€\\xa0Compound 9 co-crystalized in two different space groups, P212121 and C2. â€¡PDB accession 3GFT. Â§PDB accession 1C1Y.  To determine the effects of expression of mutant Ki-ras on lung tumorigenesis, we developed a bitransgenic mouse model that expresses the human Ki-rasG12C allele in alveolar type II and/or Clara cells in a tetracycline-inducible, lung-specific manner. Expression of Ki-rasG12C caused multiple, small lung tumors over a 12-month time period. Although tumor multiplicity increased upon continued Ki-ras expression, most lung lesions were hyperplasias or well-differentiated adenomas. This is in contrast to the more severe phenotypes observed in other transgenic mouse models in which different mutant Ki-ras alleles were expressed in the lung. Expression of Ki-rasG12C was associated with a 2-fold increase in the activation of the Ras and Ral signaling pathways and increased phosphorylation of Ras downstream effectors, including Erk, p90 ribosomal S6 kinase, ribosomal S6 protein, p38 and MAPKAPK-2. In contrast, expression of the transgene had no effect on the activation of the JNK and Akt signaling pathways. Withdrawal of doxycycline for 1 month resulted in almost a complete absence of proliferative pulmonary lesions, suggesting tumor regression in the absence of Ki-ras expression. Mutant Ki-rasG12C expression was sufficient for initial lung tumor transformation, required for maintenance of tumor phenotype, and induced transformation of lung epithelial cells by the activation of multiple effector pathways. These results describe a novel mouse lung tumor model demonstrating benign tumor development in the absence of tumor progression, which will provide a new tool for understanding the early stages of lung tumor pathogenesis.  Go to: Introduction Worldwide, lung cancer is the leading cause of cancer-related deaths in both men and women (1). Lung adenocarcinoma (AC), a type of non-small cell lung cancer, is the most common histological variant of lung cancer in the United States, and its incidence is increasing (1,2). Studies by several groups have documented mutations in the Ki-ras gene in human lung cancer, particularly in ACs found in smokers, ranging in incidence from 30-50% of the tumors examined (3-5). The high incidence of mutant forms of the Ki-ras gene observed in both human and animal lung ACs suggests that damage to this genetic locus may play an important role in the pathogenesis of lung tumors (3,5-12). Several groups have shown an association between the presence of a mutated Ki-ras gene and patient prognosis, demonstrating decreased survival in patients harboring the mutated Ki-ras gene (7,9,12-16).  Numerous mouse lung tumor models have been utilized to try to mimic the effects of oncogenic Ki-ras in human lung tumors with either the use of chemical carcinogens or transgenes. Until recently, alterations in the Ki-ras oncogene and p16Ink4a and Retinoblastoma tumor suppressor genes were the most widely studied genes between human and mouse lung tumors (17). Technological advances have allowed the extension of these studies into the analysis of gene expression across the entire genome in both human and rodent tumors. Recently, Sweet-Cordero et al. (18) demonstrated that mouse lung tumors initiated in mice by a mutant Ki-ras allele could be used to classify human samples as either normal or tumor with 97% accuracy using upregulated or downregulated gene sets obtained from gene-expression profiling. Another group, also using gene-expression analysis, showed a similarity between chemically-induced mouse lung tumors and human lung tumors (19). These results suggest that mouse models are a useful tool in the study of lung tumorigenesis.  A previous mouse model from our laboratory demonstrated that, following in utero exposure to the polycyclic aromatic hydrocarbon, 3-methylcholanthrene, lung tumors induced in the offspring exhibited a high incidence of mutations (84%) in the Ki-ras gene (8,20). A high incidence of Ki-ras mutations were found in hyperplastic lung tissue (60%) as well, suggesting that activation of Ki-ras may be an early event in lung cancer formation. One of the most interesting findings from this (8) and a subsequent study (21) was that the type of mutation present in the lesions appeared to influence tumor development. Lesions harboring VAL12, ASP12 or ARG13 mutations were more likely to progress to ACs than were lesions with CYS12 mutations (8,21), suggesting that different mutant Ras alleles may have different oncogenic potentials. These results and others have demonstrated that mutation to the Ki-ras gene locus is a critical target in the initiation and progression of lung tumor development.  Although mutations of Ki-ras are considered an early event in the pathogenesis of lung tumors, direct evidence for this hypothesis has only recently been obtained. Both human and chemically-induced rodent tumors contain a number of genetic lesions in addition to mutated Ki-ras, making it difficult to identify the role for mutant Ki-ras in lung tumor initiation. Mouse lung tumorigenesis protocols usually involve treatment with chemical carcinogens, which can alter any one of a number of gene loci in addition to Ki-ras. Koera et al. (22) demonstrated that expression of the Ki-ras gene is critical for normal development in mouse embryos, as deletion of the Ki-, but not the Ha- or N-ras genes, resulted in embryonic lethality. A number of investigators have confirmed the importance of Ki-ras for normal lung development and shown that expression of Ki-ras in the lung increases during gestation, reaching peak levels in the adult, suggesting that Ki-ras plays a role in normal lung morphogenesis (23,24).  Recent studies by several laboratories have utilized inducible transgenic mouse systems to express various mutant alleles of Ki-ras in a timed and/or tissue-specific manner to more directly determine the effects of Ki-ras on cancer initiation and progression (6,25-29). In these studies, expression of mutant Ki-ras was shown to be a potent oncogenic stimulus for lung epithelial cells, as expression of a mouse or human Ki-ras transgene resulted in a high tumor burden and formation of progressive ACs within 2 to 10 months.  Therefore, in an effort to understand the effects of activated Ki-rasG12C in lung tumorigenesis, we have utilized a bitransgenic mouse model that expresses the mutant human Ki-rasG12C allele in a lung-specific and tetracycline (tet)-inducible manner. In contrast to previous murine lung tumor models expressing mutant Ki-ras transgenes, expression of the CYS12 mutant allele allowed survival to an age of at least 12 months and resulted primarily in proliferative pulmonary lesions morphologically diagnosed as bronchioalveolar hyperplasias and adenomas (ADs). Activation of Ras downstream effector pathways, including the Raf/Mek/MAPK, p38, and Ral pathways was observed. The present study provides evidence for a novel lung tumor mouse model exhibiting little progression past the AD stage.  Go to: Materials and methods Generation of transgenic mice  The Ki-rasG12C allele was cloned from the H358 human lung bronchioalveolar cell line (American Type Culture Collection, Manassas, VA) by RT-PCR. RNA samples were reverse transcribed with the Superscript Preamplification System for first strand cDNA synthesis (Life Technologies, Gaithersburg, MD). Primers to the entire Ki-ras coding sequence were used to amplify a 567 bp product by PCR, which was then subcloned into the pCR2. 1 vector using the TA Cloning Kit (Invitrogen, San Diego, CA). This plasmid was then sequenced to confirm the identity of the cloned gene by the DNA sequencing core laboratory of the Wake Forest University School of Medicine. In addition, an aliquot of the TA-cloned pCR2.1-ras plasmid was used for an in vitro translation reaction, utilizing the TNT Quick Coupled Transcription/Translation System (Promega, Madison, WI), to confirm that the cloned cDNA produced the correct sized protein.  The Ki-ras transgene was constructed using standard molecular biological methods. Twenty Î¼g of the pCR2. 1-ras plasmid was linearized by digestion with Spe1 and blunt-ended with T4 DNA polymerase. The fragment was gel-purified and cut with Not1 to release the Ki-ras insert. The tetO7/CMV/Zeor expression vector was double digested with PmeI, which created a blunt end, and Not1 and then treated with shrimp alkaline phosphatase for 1 h at 37Â°C to prevent self-annealing. The linearized vector and the Ki-ras insert were then directionally cloned and ligated with 24 U of T4 DNA ligase, transformed into XL Blue cells, streaked onto an agar plate containing ampicillin and X-gal, and grown overnight at 37Â°C. Colonies were picked and expanded, then tested for the presence and orientation of the insertion by restriction enzyme digestion. The orientation of the inserted sequence relative to the CMV promoter was confirmed by gene sequencing.  The Ki-ras monotransgenic mouse was constructed at the Transgenic Mouse Facility at the University of Cincinnati, as described previously (30-32). The tetO-CMV-Ki-ras insert was excised from the plasmid by digestion with SpeI and PvuII and purified by CsCl gradient centrifugation. The DNA was then microinjected into donor eggs obtained from superovulated FVB/N mice. The eggs were implanted into pseudopregnant mice and founder mice containing the transgene were established. These are referred to as monotransgenic Ki-rasG12C mice.  Genotyping  DNA was extracted from mouse tails using the Wizard Genomic DNA Purification Kit (Promega) according to the manufacturerâ€™s protocol. Genotyping was performed by PCR with Amplitaq Gold (Applied Biosystems, Foster City, CA). To identify Ki-rasG12C mice, an upstream primer in the CMV minimal promoter (5â€²-CCATCCACGCTGTTTTGACCTC-3â€²) and a downstream primer in the human Ki-ras coding sequence (5â€²-TACTCCTCTTGACCTGCTGTGTCG-3â€²) were used. Primers for genotyping mice for the presence of the CCSP-rtTA construct were 5â€²-ACTGCCCATTGCCCAAACAC-3â€² (forward) and 5â€²-AAAATCTTGCCAGCTTTCCCC-3â€² (reverse), and for the SPC-rtTA construct 5â€²-GACACATATAAGACCCTGGTCA-3â€² (forward) and 5â€²-AAAATCTTGCCAGCTTTCCCC-3â€² (reverse). Samples were amplified following denaturation at 94Â°C for 2 min by 40 cycles of denaturation at 94Â°C for 30 s, annealing at 57Â°C (Ras) or 61Â°C (CCSP and SP-C) for 1 min, and extension at 72Â°C for 1 min, with a final extension for 7 min at 72Â°C (Bio-Rad iCycler, Hercules, VA).  Tumor studies  Eight week old CCSP/Ki-ras and SP-C/Ki-ras bitransgenic mice, as well as wild-type FVB/N and monotransgenic Ki-ras mice, were either untreated or given 500 Î¼g/ml doxycycline (DOX) (Sigma, St Louis, MO) in the drinking water for 12 days, 1.5, 3, 6, 9 or 12 months. Mice were killed by CO2 asphyxiation/exsanguination, and macroscopic lung lesions were counted and measured prior to fixation. The lungs were then inflated and fixed in 10% phosphate-buffered formalin. At the 12-month time point, liver, spleen, kidney, pancreas, heart and either testis and prostate or ovaries were examined macroscopically and then removed and placed in 10% phosphate-buffered formalin. After 24 h, the formalin solution was replaced with 70% ethanol to avoid excessive aldehyde cross-linking for immunohistochemistry. Tissues were embedded in paraffin and 5 mm sections prepared by microtomy. The cut slides were stained with hematoxylin and eosin and examined by veterinary pathologists (J.E. and N.D.K.). Proliferative lesions were evaluated using established morphological criteria for mice (33).  Ras and Ral activation  CCSP/Ki-ras and SP-C/Ki-ras bitransgenic mice were either untreated or given 500 Î¼g/ml DOX in the drinking water for 72 h. Lungs were removed and protein was extracted using modified RIPA buffer (50 mM Tris-HCl (pH 7.4), 150 mM NaCl (pH 7.4), 1 mM EDTA, 1 mM PMSF, 1 Î¼g/ml of aprotinin, leupeptin, pepstatin and 1 mM NaF) without detergents. Whole lung was homogenized using a Polytron homogenizer (Kinematica, Crystal Lake, IL) on setting 6 for 2 pulses of 30 s duration. Protein concentration was determined with the Bio-Rad protein assay kit according to the manufacturerâ€™s instructions (Bio-Rad, Hercules, CA). Activation of Ras and Ral was determined from the protein lysates using kits from Upstate Technologies (Charlottesville, VA). For each assay, 15 Î¼l of agarose beads conjugated to either the Ras binding domain (RBD) of Raf or Ral binding protein (RBP) were added to 10 mg of lysate. The samples were rocked for 1 h at 4Â°C. The beads were washed and denatured with 50 Î¼l loading dye containing 2-mercaptoethanol and boiled for 5 min. An aliquot of 20 Î¼l of each sample was loaded onto a 10% SDS-PAGE gel and transferred onto a nitrocellulose membrane. The membranes were incubated with either 1 Î¼g/ml anti-Ras (clone Ras10) monoclonal antibody diluted in phosphate buffered saline containing 5% dry milk and 0.05% Tween-20 or 1 Î¼g/ml anti-Ral A monoclonal antibody diluted in Tris buffered saline containing 5% dry milk and 0.05% Tween-20. Membranes were then incubated with a goat anti-mouse horseradish peroxidase-conjugated secondary antibody diluted 1:1000 in either phosphate-buffered saline or Tris buffered saline containing 5% dry milk and 0.05% Tween-20. Protein bands were visualized with Bio-Radâ€™s Immun-Star Substrate detection kit by autoradiography.  Immunohistochemisry (Ras effectors)  Formalin fixed, paraffin embedded mouse lungs sectioned 5 Î¼m thick were placed on Super Frost slides and air-dried. Slides were baked for 30 min at 58Â°C prior to deparaffinization. Baked slides were deparaffinized in xylene 3Ã— 5 min, rehydrated in 100% ethanol 2Ã— 10 min, 95% ethanol 2Ã— 10 min, and water for 2Ã— 5 min. Antigen retrieval was performed by heating slides to 100Â°C for 15 min in 10 mM sodium citrate (pH 6.0), slides were then cooled to room temperature before 2Ã— water rinse. Endogenous peroxidase activity was quenched by a 10 min incubation in 3% peroxide followed by 2 water rinses of 5 min each. Slides were blocked with TBST [25 Î¼m Tris-HCl (pH 7.2), 100 mM NaCl and 0.1% Tween 20] containing 5% goat serum (Sigma) for 1 h. Slides were then incubated overnight at 4Â°C with the following primary antibodies (Cell Signaling Technology, Beverly, MA) diluted in blocking buffer: anti-phospho Erk (THR202/TYR204) rabbit monoclonal 1:300; anti-phospho p38 (THR180/TYR182) rabbit monoclonal 12F8 IHC specific 1:100; anti-phospho MAPKAPK-2 (THR319) rabbit polyclonal 1:100; anti-phospho p90Rsk (THR359/SER363) rabbit polyclonal 1:100; anti-phospho S6 (SER235/236) polyclonal 1:100; anti-phospho JNK (THR183/TYR185) rabbit polyclonal 1:50; anti-phospho Akt (SER473) rabbit polyclonal 1:50. Slides were washed 3Ã— in TBST for 5 min then incubated with biotinylated goat anti-rabbit antibody (Vector Laboratories, Burlingame, CA) 1:1000 in TBST for 30 min, washed 3 Ã— 5 min with TBST, then incubated with avidin-biotin complex (Vector Laboratories) for 1 h. Negative controls consisting of tissue and secondary antibody only were performed to account for non-specific binding. Nova Red (Vector Laboratories) was used as per the manufacturerâ€™s instructions for 1 min before quenching in water. Slides were dehydrated in 2 Ã— 95% ethanol followed by 2 Ã— 100% ethanol for 10 s, then rinsed 3x for 10 s with xylene prior to mounting with Vectamount (Vector Laboratories). Slides were imaged with a Nikon Eclipse TE300 microscope fitted with an AxioCam digital camera.  Immunohistochemistry (CCSP, SP-C, Ki-67 and cleaved caspase 3)  Slides were deparaffinized with 3 Ã— 5 min washes with xylene. The slides were hydrated through graded alcohols 3 Ã— 1 min each. They were then placed in distilled water for 5 min. Antigen retrieval was performed using 10 mM citrate buffer (pH 6.0) for 35 min using a steamer. The slides were cooled for 20 min and placed in distilled water for 5 min. Primary antibodies were added: anti-CCSP 1:1000 (34); anti-SP-C 1:2000 (35-38); anti-Ki-67 1:25 (Abcam, Cambridge, MA), and anti-cleaved caspase 3 1:50 (Cell Signaling Technology). Samples as well as negative controls were incubated overnight at 4Â°C. Slides were then washed 5Ã— with 1Ã— Tris buffer (Biomeda, Foster City, CA) containing 0.5% casein. A secondary biotinylated anti-rabbit antibody (BioGenex, San Ramon, CA) was added at a dilution of 1:20 for 30 min at 32Â°C. Slides were washed 5Ã— in Tris buffer followed by a 1:20 dilution of Streptavidin-alkaline phosphatase conjugate (BioGenex) for 30 min at 32Â°C. Slides were washed 5Ã— in Tris buffer and Vector Red substrate (Vector Laboratories) was added for 5 min. Slides were then washed 2x in 0.1 M Tris-HCl buffer (pH 8.2-8.4) and 5Ã— with distilled water. Slides were counterstained with Mayerâ€™s hematoxylin for 5 min, then washed under running water for 5 min. Slides were dehydrated through graded alcohol and cleared through several changes of p-xylene.  Expression of transgenic and endogenous Ki-ras RNA by real-time fluorescent PCR  Eight week old CCSP/Ki-ras and SP-C/Ki-ras bitransgenic mice, as well as monotransgenic Ki-ras mice, were either untreated or given 500 Î¼g/ml DOX in the drinking water for 24 h, 48 h, 7 or 14 days. Thirty mg of whole lung tissue was homogenized with a Polytron homogenizer in RLT lysis buffer supplied in the RNeasy Mini Kit at speed 6. Total RNA was extracted using the RNeasy Mini Kit (Qiagen, Valencia, CA). In addition, RNA was also isolated from the liver, spleen, thymus, pancreas, kidneys, heart, intestines and either testis and prostate or ovaries at 14 days of treatment. For RT-PCR, cDNA was initially generated from 1 Î¼g of RNA using the iScript cDNA Synthesis Kit (Bio-Rad). One tenth of the cDNA (2 Î¼l) was used to amplify the Ki-ras transgene, the endogenous murine Ki-ras gene, and GAPDH using the iCycler (Bio-Rad) for 40 cycles with SYBR Green Supermix (Bio-Rad).  To generate a standard curve, each PCR product (Ki-ras transgene, endogenous Ki-ras, and GAPDH) was first cloned into a pCR2.1 TA cloning vector. The plasmids were then utilized to generate the standard curves for each of the three genes using the same primers to be used for the unknown samples in order to quantify the copy number of each gene by the relative standard curve method (39-41). Both the Ki-ras transgene copy number and endogenous Ki-ras copy number were normalized to GAPDH copy number. PCR efficiencies and R2 values for the standard curves ranged from 99 to 103% and 0.991-0.998, respectively.  The primers used to amplify the Ki-rasG12C transgene were 5â€²-CTGCAGAATTCGCCCTTATGACTGA-3â€² (forward) and 5â€²-TAGCTGTATCGTCAAGGCACTCTTGC-3â€² (reverse); for the endogenous murine Ki-ras gene were 5â€²-GCAGGGTTGGGCCTTACAT-3â€² (forward) and 5â€²-ATGCGTCGCCACATTGAAT-3â€² (reverse); and for GAPDH were 5â€²-TCTCCCTCACAATTTCCATCCCAG-3â€² (forward) and 5â€²-GGGTGCAGCGAACTTTATTGATGG-3â€² (reverse). Cycling conditions included an initial denaturation step at 94Â°C for 3 min followed by 40 cycles of denaturation at 94Â°C for 30 s, 66.5Â°C (transgene and GAPDH) or 57Â°C (endogenous Ki-ras) for 1 min, and 72Â°C for 1 min. A melt curve was determined following each real-time PCR run to ensure the presence of a single product.  For values reported in Table II, the iCycler software determined a copy number for the Ki-ras transgene, endogenous Ki-ras, and GAPDH for each unknown sample based on the gene-specific standard curve. Both transgene and endogenous Ki-ras copy numbers were normalized to GAPDH by dividing each sample by their respective GAPDH copy numbers. These normalized values were averaged and standard deviations calculated. The numbers in parentheses were determined by dividing all of the normalized averages by the monotransgenic normalized endogenous Ki-ras value.  Table II Table II Ki-rasG12C transgene and endogenous Ki-ras expression normalized to GAPDH in CCSP/Ki-ras mice To compare transgene expression of Ki-rasG12C to Ki-rasG12D, RNA samples from the CCSP/Ki-rasG12D mice were obtained from the University of Cincinnati Transgenic Mouse Facility for each of the following treatment groups, no DOX treatment, 24 h DOX treatment, and 7 days DOX treatment. Each treatment group was provided in triplicate. A 2 months DOX-treated lung sample was also provided by Dr Harold Varmus from the Memorial Sloan-Kettering Cancer Center. An additional time point of 2 months was included for CCSP/Ki-rasG12C mice as a comparison with this CCSP/Ki-rasG12D sample. Expression of the murine Ki-rasG12D transgene, endogenous Ki-ras, and GAPDH genes were determined by real-time PCR as described above, except that the primers for the mouse transgene used in their construct (26) were: 5â€²-CAAGGACAAGGTGTACAGTTATGTGACT-3â€² (forward) and 5â€²-GGCATCTGCTCCTGCTTTTG-3â€² (reverse).  Statistical methods  Continuous variables were compared among groups with t-tests. Logistic regression adjusting for sex, time and type of experiment was used to estimate adjusted means. Ï‡2-test was used to evaluate adjusted means and generated P-values.  Go to: Results Construction and characterization of the bitransgenic Ki-rasG12C mice  Previous studies have shown that expression of the normal Ki-ras gene is critical for normal mouse development and survival (22-24). To overcome the problems associated with expression of a strong mitogenic signal during lung development, we have utilized the inducible â€˜tet-onâ€™ system. This gene construct uses a reverse tet trans-activator (rtTA) protein that requires the presence of the ligand, DOX, in order for the rtTA gene product (consisting of the mutant tet repressor linked to the VP16 activation domain) to recognize the tetO sequence and thus stimulate gene transcription. In this approach, the gene of interest (in this case, the cDNA of mutated Ki-ras) is cloned into the tetO7-CMV plasmid, which places the Ki-ras gene downstream of a tet-inducible promoter. Founder mice established with this construct will be unable to express the Ki-ras gene because they lack the tet rtTA protein. These mice are then crossed with a second transgenic mouse line that contains the tetracycline rtTA protein linked to either the surfactant protein C (SP-C) or Clara cell secretory protein (CCSP) promoters, directing lung-specific expression of the rtTA protein. The SP-C promoter specifically targets expression to the alveolar type II cells of the lung (30,31,42,43), which has been suggested as the putative cell of origin for lung ACs in both humans and rodents (17), while the CCSP promoter appears to target expression to both the Clara and alveolar type II cells (42,44,45). Construction, generation and characterization of SP-C-rtTA and CCSP-rtTA mice have been described previously (46). The SP-C-rtTA (line 74) mice used in this study are a distinct line from the SP-C-rtTA line previously used although it contains an identical transgene construct (43). This line has very low background levels of transgene expression in the absence of DOX compared with the previously described line (unpublished data). Previous studies have shown that treatment with DOX results in very high levels of induction within 12-24 h whereas after withdrawal of DOX, expression of the transgene returns to untreated levels within 24-48 h (42,46,47).  To generate mice containing the Ki-rasG12C transgene, the insert was prepared as described in the Materials and methods section and microinjected into donor eggs obtained from superovulated FVB/N mice. The eggs were implanted into pseudopregnant mice and founder mice containing the transgene were established. To screen the pups for the presence of the transgene, genomic DNA was isolated from a small piece of the tail and then probed for the presence of the tetO and Ki-ras sequence using PCR, producing a 264 bp product that extended from the 3â€² region of the tetO7-CMV expression vector through to exon 2 of the Ki-ras gene.  Initially, five Ki-rasG12C founders were identified by PCR and bred to non-transgenic littermates to establish germ line transmission. One founder did not transmit the transgene to offspring. Animals from each of the remaining four lines were bred to CCSP-rtTA or SP-C-rtTA activator mice to obtain bitransgenic mice that were then treated with DOX for 7 days to induce transgene expression. By RT-PCR analysis, one line had robust transgene expression, two had low expression, and one was not induced; however, expression was not quantified. The uninduced line was not characterized further. Bitransgenic mice from the two lines expressing low levels of Ki-ras RNA were treated with DOX from 6 weeks to 6 months. Upon killing, analysis of lung histology from these animals revealed no proliferative lesions and these lines were terminated. The remaining line developed lung adenomas upon induction of oncogenic Ki-ras expression and was used for our studies. Twenty five Î¼g of liver DNA from the Ki-rasG12C mouse line was restricted with EcoRI and analysed by Southern blotting with a 73 bp exon 1 probe to determine the copy number of the inserted transgene. By densitometric analysis, the ratio of the 430 bp transgenic fragment to the 700 bp endogenous fragment demonstrated that there were ~22 copies of the transgene present in the mouse genome.  Treatment with DOX induces morphological changes in the lung  Bitransgenic CCSP/Ki-ras and SP-C/Ki-ras, monotransgenic Ki-ras, and control wild-type FVB/N mice were either untreated or treated with 500 Î¼g/ml of DOX in the drinking water for 12 days, 1.5, 3, 6, 9 and 12 months. At each time point, mice were killed, macroscopic lesions counted, and the lung tissues were evaluated microscopically for the presence of proliferative changes. DOX-treated bitransgenic mice on either the CCSP or SP-C background exhibited small, hyperplastic lung foci after only 12 days of DOX treatment (Figure 1B). By 5 weeks of treatment, extensive epithelial hyperplasia of the alveolar region of the lung tissue could be seen. Lung morphology in untreated bitransgenic mice(Figure 1A) or DOX-treated single transgenic or control mice was normal.  Fig. 1 Fig. 1 CCSP/Ki-ras lung morphology following DOX treatment. Hematoxylin and eosin stained lung tissue following (A) no DOX treatment, 10; (B) 12 days of DOX treatment, 20x, focal hyperplasia; (C) 3 months of DOX treatment, 10Ã—, focal hyperplasia; (D) ... Macroscopically visible lesions were first detected following 3 months of DOX treatment. CCSP/Ki-ras mice had a tumor incidence of 73% with a multiplicity of 3.5 Â± 4.7 tumors per mouse while SP-C/Ki-ras mice had a tumor incidence of 55% with a multiplicity of 1.7 Â± 2.0 tumors per mouse (Table I). At 3 months, all lesions were <1 mm in size. With continued exposure to DOX, the macroscopic lesions increased in number, with CCSP/Ki-ras mice displaying a higher tumor multiplicity than SP-C/Ki-ras mice at 6 and 9 months (P < 0.0001). By 6 and 9 months of DOX treatment, both CCSP/Ki-ras mice and SP-C/Ki-ras mice exhibited a high incidence of lung tumors with a tumor multiplicity of 28.8 Â± 8.7 in CCSP/Ki-ras mice and 10.2 Â± 7.2 in SP-C/Ki-ras mice 9 months after the initiation of DOX treatment. Tumor multiplicity increased in both lines of mice as the time of DOX treatment increased (P < 0.0001), resulting in a tumor multiplicity of 34.0 Â± 8.4 in CCSP/Ki-ras mice and 12.5 Â± 8.2 in SP-C/Ki-ras mice by 12 months. There was a low tumor incidence found in monotransgenic mice. These tumors were all 1 mm in size and could be attributed to either the low basal expression (Table II) of the transgene or spontaneous tumor formation often found in the FVB/N strain.  Table I Table I Lung tumor incidence and tumor multiplicity The majority of visible lesions detected at 3 months were morphologically diagnosed as hyperplastic lesions characterized by regular cuboidal cells lining alveolar septa (Figure 1C and D). A single adenoma was identified in one CCSP/Ki-ras mouse at this time. With continued DOX treatment for 6, 9 and 12 months, the incidence of adenomas increased (Figure 1E-J shows lesions after various time points and at different magnifications). Interestingly, by 12 months time, only two lesions were identified as low-grade carcinomas characterized by regular, but closely aligned pneumocytes arranged in a ribbon pattern. These small clusters had small, condensed nuclei and a hypereosinophilic cytoplasm (necrosis) (Figure 1K); otherwise, none of the tumors had progressed past the adenoma stage. Although there was a significant difference between SP-C/Ki-ras and CCSP/Ki-ras bitransgenic mice in tumor formation (Table I), the morphology of the proliferative lesions in both lines of bitransgenic mice were similar in being diagnosed as hyperplasias and adenomas. Immunohistochemical staining with Ki-67 demonstrated scattered nuclear staining throughout the lung lesions (Figure 2A). Comparison of the lesion morphology on the two activator backgrounds revealed that the CCSP/Ki-ras mice developed a higher incidence of proliferative changes with diminished latency. The early lesions in these animals appear to rise with characteristics of bronchiolar origin, whereas the early hyperplastic lesions in SP-C/Ki-ras mice look to be of alveologenic origin. Alveolar lipoproteinosis is found regularly in the SP-C/Ki-ras mice, indicative of abnormal surfactant metabolism. These results are particularly interesting since Fisher et al. (26), using the identical tet-on bitransgenic system but with a mutant murine Ki-rasG12D allele, found that Ki-rasG12D mice developed aggressive ACs after only 2 months of DOX. Taken together, these results suggest the development of a novel lung tumor mouse model exhibiting a less aggressive tumor type.  Fig. 2 Fig. 2 Ki-67 (A) and secretory protein (B and C) expression in lung lesions from CCSP/Ki-ras mice. Immunohistochemical analysis of lung tissue following 9 months of DOX treatment with (A) anti-SP-C, 20x, solid AD, (B) anti-CCSP, 20x, solid AD and (C) Ki-67, ... Identification of the progenitor cells of lung tumors as either Clara cells or alveolar type II cells has been an ongoing controversy, and has recently been addressed in mouse models (26,27,29,48). Several of these studies have suggested that the majority of murine lung lesions appear to arise in alveolar type II cells, while a few groups have identified cells positive for both CCSP and SP-C. Recently, cells have been identified in the bronchioalveolar junction that stain for both proteins and have been suggested to be the putative bronchioalveolar stem cells that may serve both as the stem cell population for alveolar type II and Clara cells, as well as the cell of origin for peripheral lung tumors (49). Therefore, to determine in which cell type the lung tumors in the Ki-rasG12C expressing mice arise, immunohistochemical analysis was performed using polyclonal antibodies to both the SP-C and CCSP proteins. All of the lung lesions showed moderate staining for SP-C but only very faint or no staining for CCSP in proliferative cells, which is shown for CCSP/Ki-ras mice in Figure 2B and C. While these results could be interpreted as implicating alveolar type II cells as the origin of these lesions, they would also be consistent with the presence of a common progenitor cell line as suggested by Kim et al. (49).  Ki-ras is required for tumor maintenance  Recent studies by Fisher et al. (26), utilizing a similar bitransgenic mouse system, reported that, following induction of mutant murine Ki-rasG12D expression, removal of DOX resulted in regression of the lung tumors. In order to demonstrate that the Ki-rasG12C allele is similarly important for tumor maintenance, bitransgenic CCSP/Ki-ras and SP-C/Ki-ras mice were treated with 500 Î¼g/ml of DOX for 9 months and then withdrawn from DOX treatment for up to 1 month. Two weeks after DOX withdrawal, there were only four lung tumors still visible on the surface of the lungs in both CCSP/Ki-ras and SP-C/Ki-ras mice. By 1 month of withdrawal, 4 tumors were still visible on the surface of the lung of CCSP/Ki-ras mice, whereas no lesions were seen in SP-C/Ki-ras mice. Unfortunately, the macroscopic lesions were too small for further analysis; however, the persistent lesions could possibly be attributed to a second genetic alteration inhibiting tumor regression. Microscopically, both SP-C/Ki-ras and CCSP/Ki-ras mice exhibited few lesions after 1 month of DOX withdrawal, with only minimal hyperplastic foci generally present (Figure 1L). The remaining foci following 2 weeks of DOX withdrawal showed scattered positive nuclear staining for Ki-67, while the foci remaining following 1 month of DOX withdrawal were negative. None of the lesions were positive for cleaved caspase 3, indicating that these lesions were not regressing through an apoptotic mechanism. These data indicate that expression of Ki-rasG12C is necessary for maintenance of lung lesions and regression occurs through a mechanism alternative to apoptosis.  Expression of Ki-ras transgene  Real-time PCR was used to quantitate expression of the human Ki-rasG12C transgene in the lungs following 24 and 48 h, and 7 and 14 days of DOX treatment. Liver, spleen, pancreas, intestines, kidneys, heart, thymus and either testis and prostate or ovaries were screened for transgene expression after 14 days of antibiotic treatment. In the lung, transgene expression was shown to increase by at least 3 logs following 24 h of DOX treatment, and increased an additional 2-fold by 7 days in both lines of bitransgenic mice (Table II). Expression of the transgene in CCSP/Ki-ras mice was ~2-fold higher at every time point than in SP-C/Ki-ras mice, although this difference was not statistically significant (P < 0.9). This difference could be due to the fact that in CCSP/Ki-ras mice, both Clara cells and alveolar type II cells express the Ki-ras transgene, whereas in SP-C/Ki-ras mice only alveolar type II cells express the transgene. Except in testis and prostate, there was no detectable level of transgene expression in any of the other organs examined (Figure 3A). Since the rtTA protein should not be expressed in these two tissues, we tested the monotransgenic Ki-ras mice for transgene expression and found that the level of expression in the testis was ~6.3-fold higher than in the DOX-treated, bitransgenic lungs of SP-C/Ki-ras mice (Figure 3B). Interestingly, the testis exhibited no morphological changes even after 12 months of DOX treatment, consistent with the fact that testicular cancers do not have a high incidence of mutations in any of the Ras genes (50). The expression of the transgene in the prostate was ~8-fold higher than the lung in the same monotransgenic mice and, similar to the testis, displayed no alterations in normal histology. These results suggest that the transgene may have integrated downstream of a region of the chromosome that is regulated by male-specific hormones.  Fig. 3 Fig. 3 Ki-ras transgene expression in multiple organs of bitransgenic mice. (A) Total RNA was extracted from kidney, heart, lung, testis, liver, thymus, intestines and spleen tissue of CCSP/Ki-ras mice Â± treatment with 500 Î¼g/ml of DOX. RT-PCR ... We also quantified endogenous murine Ki-ras expression following 24 and 48 h, and 7 and 14 days, and 2 months of DOX treatment for both the CCSP/Ki-ras and SP-C/Ki-ras mice. The level of endogenous Ki-ras expression was identical between both lines of bitransgenic mice (data not shown). Similar to the results obtained by Fisher et al. (26), the expression levels of endogenous murine Ki-ras declined upon extended exposure to elevated levels of transgene expression (shown for CCSP/Ki-ras mice in Table II). The levels of endogenous Ki-ras expression decreased 1.5-fold after 7 days of DOX, and remained at reduced levels at 14 days (P < 0.0001). At 7 and 14 days, the level of expression of the endogenous gene was ~20-fold higher than the transgene; however, following 2 months of DOX treatment, expression of the endogenous gene was comparable with that of the transgene (Table II). These results suggest that the total levels of Ki-ras expression are strictly regulated in pulmonary epithelial cells.  The Ki-rasG12C mice used in this study demonstrated a much less severe phenotype than reported by several other laboratories utilizing transgenic mouse lines (6,26-29). In particular, the mice described by Fisher et al. (26) utilized the same vector constructs, tetO7-CMV and CCSP-rtTA, and expressed the murine Ki-rasG12D mutant allele in the same genetic background (FVB/N). Our previous data had suggested that the Ki-rasG12D mutation would be highly oncogenic, while the Ki-rasG12C mutation used in this study would be less oncogenic (8,21). To compare the levels of transgene expression, we obtained RNA from Ki-rasG12D mice either not treated or treated for 24 h and 7 days with DOX. In addition, we also isolated RNA from a sample of lung tumor tissue from Ki-rasG12D mice (a generous gift from Dr Varmusâ€™ laboratory). In an individual experiment with newly generated standard curves, we compared these samples with RNA isolated from lung tissue in Ki-rasG12C mice following exposure to DOX for each time point. The results showed that murine Ki-rasG12D transgene expression for 24 h and 7 days was 4.31 Ã— 10-4 Â± 2.11 Ã— 10-4 and 1.46 Ã— 10-3 Â± 1.24 Ã— 10-3 compared with 6.83 Ã— 10-4 Â± 6.83 Ã— 10-4 and 8.90 Ã— 10-4 Â± 1.25 Ã— 10-3, respectively, in Ki-rasG12C mice, neither time point displaying any statistically significant difference between the two lines of mice (P = 0.5758 and 0.6022). There was a large difference in transgene expression between mice, accounting for the large standard deviations. However, this variation was not seen in our housekeeping gene, GAPDH, indicating that induction of transgene expression varied between mice. At 2 months, expression of the Ki-rasG12D transgene was ~2.5 times higher than the Ki-rasG12C allele in CCSP/Ki-ras mice. The tumor mass and lung weights were also two times higher in the Ki-rasG12D mice (26), which accounts for this difference in the RNA levels observed at 2 months between the two transgenic mouse lines. These results suggest that differences in phenotype between Ki-rasG12D and Ki-rasG12C mice were not due to differences in expression of their respective transgenes.  Activation of downstream signaling pathways  Mutations in Ki-ras result in constitutive activation of the Ras protein and increased signaling to downstream effectors. Therefore, the activation of Ras and one of its downstream effectors, Ral, was assayed. Ras and Ral activation were detected with pull down assays that detected binding of the activated protein to either the RBD of Raf or RBP, respectively. The activated proteins were then detected by western blot using antibodies for either the Ras or Ral proteins. In both bitransgenic mouse lines, Ras and Ral activation increased following treatment of the mice with DOX for 72 h. Ras activation was shown to be elevated 1.4- and 2.2-fold in DOX-treated SP-C/Ki-ras and CCSP/Ki-ras mice, respectively, compared with untreated mice (Figure 4A). Similarly, Ral activation was increased 1.4- and 1.8-fold in DOX-treated SP-C/Ki-ras and CCSP/Ki-ras mice, respectively, compared with untreated mice (Figure 4B). Thus, expression of the Ki-rasG12C allele increased signaling to downstream effectors of Ras. It should be emphasized that lysates used in the binding assays were tissue preparations from whole lung after 72 h of DOX treatmentâ€”given the large number of cell types in the lung and the restricted expression of the transgene to Clara and/or alveolar type II cells, it is likely that the activation assays have probably underestimated the actual amount of enhanced activation.  Fig. 4 Fig. 4 Ras and Ral activation in the lungs of CCSP/Ki-rasG12C bitransgenic mice. Total protein was extracted from whole lungs of bitransgenic mice Â± treatment with 500 Î¼g/ml of DOX. (A) Active Ras was pulled down from cell lysates with agarose ... Immunohistochemical analyses of lung tissues from CCSP/Ki-ras mice demonstrated increased phosphorylation of downstream Ras effectors after exposure to DOX. Increased phosphorylation was detected in Erk, with the highest level of activation in the nucleus and intermediate activation in the cytoplasm of hyperplastic tissue, adenomatous tissue, and normal bronchiolar epithelium (Figure 5A). Staining in lung tumors showed a relatively homogenous pattern, with scattered staining throughout the lung lesions. Downstream targets of activated Erk were also identified, including p90 Rsk which showed scattered, highly active nuclear and intermediately active cytoplasmic protein (Figure 5B) and activated ribosomal protein S6, identified by scattered cytoplasmic activation (Figure 5C). These effectors were active in the same cell types showing Erk activation. In addition, elevated phosphorylation of p38 was seen in the cytoplasm of proliferative cells (Figure 5D) and its downstream target, MAPKAPK-2, showed activation in both the nucleus and cytoplasm of proliferative cells (Figure 5E). Interestingly, phosphorylation of Akt and Jun kinase (JNK) was not increased (Figure 5F and G). These results suggest that mutant Ras-mediated activation of the Raf-Mek-Erk and Ral pathways may play a role in the early stages of lung tumorigenesis.  Fig. 5 Fig. 5 Phosphorylation of Ras downstream effectors in CCSP/Ki-rasG12C mice Â± treatment with 500 Î¼g/ml of DOX for 3, 6 or 9 months. Lung tissue was removed, embedded in paraffin, and 5 Î¼m sections were cut for immunohistochemistry with ... Go to: Discussion Several laboratories, utilizing transgenic mouse lines, have demonstrated that expression of a mutant Ki-ras transgene results in the formation of lung tumors that rapidly progressed to ACs, with mice succumbing to their lung tumor burden in 3-10 months (6,26-29). The stark contrast in the phenotype induced in the mice reported here as compared with these other studies is particularly relevant to the results reported by Fisher et al. (26), who utilized the same tet-inducible system in the same mouse strain and the same line of CCSP-rtTA mice as employed in these studies. Fisher et al. (26) reported that expression of murine Ki-rasG12D resulted in lung ACs after only 2 months of DOX treatment. In comparison, although expression of the human Ki-rasG12C allele initially caused the appearance of visible macroscopic lesions after 3 months of DOX treatment, mice expressing the Ki-rasG12C allele for 12 months never displayed enough of a tumor burden to affect survival. The overwhelming majority of lung lesions were hyperplasias and ADs. Thus, mice with the Ki-rasG12D allele exhibited more advanced tumors than mice with the Ki-rasG12C allele; however, transgene expression in both of these mice was similar at 24 h and 7 days.  There are three possible explanations for the phenotypic differences seen between the Ki-rasG12C and Ki-rasG12D mice. First, previous studies from this laboratory provided evidence that different Ki-ras mutations influenced tumor progression, with lesions harboring the more potent VAL12 and ARG12 mutations more likely to progress to a malignant phenotype than lesions containing wild-type sequences or a CYS12 mutation (8,21). Recent studies have suggested that the correlation of mutated Ki-ras with decreased patient survival may depend on the actual base substitution, with the VAL12 and ARG12 mutations showing a trend with poor patient outcome (7,13,16). Thus, the results presented in this study, combined with those cited above, may explain the severe phenotypic differences and could be the first to demonstrate in vivo that the type of mutation induced in Ki-ras may determine the rate of lung tumor progression and the severity of the lesion produced.  Second, the Ki-rasG12C allele is a human allele while the Ki-rasG12D is a murine allele, both expressed in identical murine backgrounds. While the Ki-ras transcript and protein sequences are highly conserved between mouse and human, demonstrating 93% mRNA and 98% protein homology, there is the possibility that the human protein is signaling differently than the murine protein. Anytime there is cross-species expression, this must be considered.  Third, these transgenic mice were made through a random insertion event. Since the insertion site of neither transgene is known, there is the possibility that the phenotype is an effect of differential insertion. This problem can generally be addressed with multiple founder mice; however, we were unable to obtain a second founder line with comparable transgene expression. Further research needs to be done to distinguish between these potential mechanisms.  The benign phenotype of this model prompted us to further characterize the effects of the Ki-rasG12C transgene at the molecular level, thus we examined the activation of downstream Ras effectors. Activated Ras is thought to induce tumorigenicity by turning on multiple downstream effector pathways that increase cell proliferation and inhibit the induction of apoptosis (51). To determine effector activation, Ras and Ral activation assays as well as immunohistochemistry with phospho-specific antibodies to detect activation of downstream effectors were used. Both Ras and Ral activity were elevated ~2-fold in Ki-rasG12C expressing mice relative to untreated controls. Immunohistochemical analysis revealed that Erk and its downstream effectors, p90Rsk and pS6, were highly phosphorylated in lung tumor cells expressing mutant Ki-ras compared with non-tumor tissue or untreated, DOX naive mice. However, while we detected no increase in the activity of the JNK or Akt pathways, increased phosphorylation of p38 and MAPKAPK-2 were detected. Interestingly, Tuveson et al. (25) failed to detect phosphorylated Erk1/2 by immunohistochemistry in murine lung hyperplasias induced by expression of the Ki-rasG12D allele, although they detected elevated levels of cyclin D1. The reason for these contrasting observations is not known, but could be due to differences in the Ras alleles used in the transgene constructs, differences in levels of expression of the different Ki-ras transgenes, or differences in the antibodies or staining methods used.  Several studies have shown that stimulation of Ras activity in primary cells induces both an increase in cell proliferation and also activates a defensive mechanism to enhance apoptosis or cell cycle arrest in response to inappropriate mitogenic signals (52,53). The tumors in this model demonstrated scattered positive staining for Ki-67 but were negative when stained with an antibody specific for the cleaved fragment of caspase 3. The lack of apoptosis in both the proliferative lung lesions and in the tissue undergoing regression following DOX withdrawal suggests that there may be another regulatory mechanism accounting for the benign phenotype in this tumor model. Whereas signaling through JNK and Akt have been suggested to inhibit apoptosis (54,55), the role of p38 in the regulation of cell growth, differentiation, and cell death remains somewhat controversial (56). While an increase in Ras-mediated p38 signaling has been shown by some groups to inhibit proliferation or increase apoptosis in a number of different cell types (57-59), elevated levels of p38 have been observed in human lung tumor samples (60), suggesting that activation of p38 in some cell contexts could have a mitogenic effect. It is likely that the Ki-rasG12C allele activates a specific subset of downstream effector pathways relative to other Ki-ras mutants, and that this could determine how quickly tumors progress to more severe phenotypes.  Until recently, there has been much controversy regarding the putative progenitor cells of peripheral lung tumors. Many late-stage lung tumors express both SP-C and CCSP, making it difficult to distinguish the cell of origin. Recent transgenic mouse models have demonstrated that the majority of peripheral lung tumors stain positively for SP-C only whereas none of the lesions demonstrated staining for CCSP alone, and only a few lung tumors stain positively for both SP-C and CCSP (26,27,29). Recent studies by Kim et al. (49) have identified a potential common stem cell in the lung, which they have termed bronchioalveolar stem cells, that was identified in the bronchioalveolar junction. The authors provided evidence that these cell types may be the progenitor cells of both Clara and alveolar type II cells, and in addition may be the cells of origin for peripheral lung tumors. In the mouse model described in the present study, the Ki-rasG12C transgene is expressed in both Clara and alveolar type II cells when expressed in the CCSP-rtTA activator line, but immunohistochemical analysis demonstrated positive staining for SP-C and very little or no staining for CCSP in the proliferative cells. These data provide evidence that alveolar type II cells may serve as progenitors of murine lung tumors, but are also consistent with the possibility that tumors arise from a common progenitor cell for both Clara and alveolar type II cells.  One of the advantages of the Tet-on system is the ability to turn off transgene expression and thus determine the role of activated Ki-ras in tumor maintenance and progression. Results obtained in this study, as well as those reported by Fisher et al. (26), demonstrated that, following removal of DOX for 1 month, the number of proliferative lesions was reduced, consistent with regression of the tumors and the requirement for oncogenic Ki-ras expression for maintenance of the tumor phenotype. Using a nude mouse assay, Rothâ€™s group transfected antisense RNA constructs of Ki-ras into human tumor cell lines and demonstrated a strong inhibition of tumor cell growth in mice, demonstrating the importance of the mutated ras allele in stimulating tumor cell proliferation and maintenance of the transformed phenotype in human cells as well (61,62). Interestingly, although the tumors that developed in Ki-rasG12C mice were not as severe, 1 month following DOX withdrawal a small number of lesions were still present when transgene expression was driven from the CCSP promoter. The remaining lesions stained positive for Ki-67 following 2 weeks of DOX withdrawal and negative following 1 month of withdrawal. However, all of the lesions were negative for cleaved caspase 3, suggesting that once the proliferative stimulus of elevated mutant Ki-rasG12C expression is removed, either tissue remodeling or the initiation of a non-apoptotic cell death pathway (63,64) may mediate the reversion of the benign lesions to normal-looking pulmonary tissue. It is possible that the longer latency of tumor development in Ki-rasG12C mice allowed a second genetic mutation to occur in some lesions that interfered with the ability of some lesions to regress. However, we also cannot rule out the possibility that due to the leakiness of transgene expression (Table II), low levels of transgene expression may prevent tumor regression in a small number of lesions.  Our results support the development of a novel mouse model for lung tumorigenesis where the tumors do not progress past the adenoma stage, in contrast to other models using Ki-ras transgenes. This long latency will allow the tumors to be monitored for an extended time period without the complications encountered by a high tumor burden or early death in the mice. With an understanding of specific activation pathways of Ras effectors, more effective targeting of novel anti-cancer agents to the specific molecules activated by specific-mutant Ras alleles could be developed, allowing therapies to be tailored to the specific molecular lesions of the tumor. Thus, these results strongly confirm the current strategy of targeting Ras proteins or members of the Ras signaling pathway in the development of novel anti-neoplastic and chemopreventive agents.   Abstract It is thought that KRAS oncoproteins are constitutively active because their guanosine triphosphatase (GTPase) activity is disabled. Consequently, drugs targeting the inactive or guanosine 5â€²-diphosphateâ€“bound conformation are not expected to be effective. We describe a mechanism that enables such drugs to inhibit KRASG12C signaling and cancer cell growth. Inhibition requires intact GTPase activity and occurs because drug-bound KRASG12C is insusceptible to nucleotide exchange factors and thus trapped in its inactive state. Indeed, mutants completely lacking GTPase activity and those promoting exchange reduced the potency of the drug. Suppressing nucleotide exchange activity downstream of various tyrosine kinases enhanced KRASG12C inhibition, whereas its potentiation had the opposite effect. These findings reveal that KRASG12C undergoes nucleotide cycling in cancer cells and provide a basis for developing effective therapies to treat KRASG12C-driven cancers.  Wild-type RAS guanosine triphosphatases (GTPases) cycle between an active, guanosine 5â€²-triphosphate (GTP)â€“bound, and an inactive, guanosine 5â€²-diphosphate (GDP)â€“bound, state (1, 2). This is mediated by nucleotide exchange factors, which catalyze the exchange of GDP for GTP, and GTPase-activating proteins, which potentiate a weak intrinsic GTPase activity (3). Cancer-causing mutations impair the GTPase activity of RAS, causing it to accumulate in the activated state (4â€“6). Despite the prevalence of these mutations, no therapies that directly target this oncoprotein are currently available in the clinic (7â€“9). A recently identified binding pocket in KRASG12C (10) now enables the discovery of compounds that potently inhibit KRAS-GTP or effector signaling by this mutant.  Here we characterize a novel compound, ARS853, designed to bind KRASG12C with high affinity (11). The structures of ARS853 and previously reported (10) compounds (cmpds) 6 and 12 are shown in fig. S1A. Treatment of KRASG12C-mutant lung cancer cells with ARS853 reduced the level of GTP-bound KRAS by more than 95% (Fig. 1A, 10 Î¼M). This caused decreased phosphorylation of CRAF, ERK (extracellular signalâ€“regulated kinase), and AKT. In contrast, even at the highest concentration tested, cmpd 6 or 12 had only a minimal effect on pCRAF and pERK, without affecting KRAS-GTP levels (Fig. 1A and fig. S1B). ARS853 inhibited proliferation with an inhibitory concentration 50% (IC50) of 2.5 Î¼M, which was similar to its IC50 for target inhibition (Fig. 1, A and B). ARS853 (10 Î¼M) inhibited effector signaling (Fig. 1C and fig. S1C) and cell proliferation (Fig. 1D and fig. S2) to varying degrees in six KRASG12C mutant lung cancer cell lines, but not in non-KRASG12C models (Fig. 1E and fig. S1, C and D). Similarly, it completely suppressed the effects of exogenous KRASG12C expression on KRAS-GTP levels, KRAS-BRAF interaction, and ERK signaling (fig. S1E). Inhibitor treatment also induced apoptosis in four KRASG12C mutant cell lines (Fig. 1, F to H). Thus, ARS853 selectively reduces KRAS-GTP levels and RAS-effector signaling in KRASG12C-mutant cells, while inhibiting their proliferation and inducing cell death.  Fig. 1 Fig. 1 Selective inhibition of KRASG12C signaling and cancer cell growth by ARS853 In contrast to the rapid inhibition of signaling by kinase inhibitors, inhibition of KRASG12C by ARS853 occurred slowly (Fig. 2A and fig. S3). In some cell lines, maximal inhibition of KRAS-GTP occurred in 6 hours; in others, in 48 to 72 hours. To understand this phenomenon, we examined the mechanism of KRASG12C inhibition in more detail. To determine whether ARS853 binds to the active or the inactive conformation of KRASG12C, we used differential scanning fluorimetry, which assays ligand-induced changes in protein thermal stability (12). Recombinant KRASG12C was loaded with either GTPÎ³S or GDP (fig. S4A) and then incubated with ARS853. Samples were incubated at increasing temperatures in the presence of a fluorescent dye that binds to hydrophobic surfaces exposed during thermal denaturation. ARS853 increased the amplitude of the thermal denaturation curve of KRASG12C loaded with GDP but not with GTPÎ³S (Fig. 2B and fig. S4B). ARS853 did not alter the denaturation curve of GDP-loaded KRASWT (fig. S4C). These data suggest that ARS853 preferentially interacts with inactive, or GDP-bound, KRASG12C.  Fig. 2 Fig. 2 Inhibition of active KRAS levels despite an interaction with GDP-bound KRASG12C KRAS mutants are thought to exist in a â€œconstitutivelyâ€� active (GTP-bound) state in cancer cells (13). Thus, inhibition of KRAS-GTP levels by a drug that preferentially interacts with GDP-bound mutant KRAS is puzzling. Codon 12 mutations disable the activation of RAS GTPase by GTPase-activating proteins (14â€“16). It is possible, however, that the basal GTPase activity of KRASG12C is sufficient to enable nucleotide cycling in cancer cells. Consequently, we hypothesized that binding of the inhibitor to KRASG12C traps it in an inactive (GDP-bound) conformation by reducing its susceptibility to exchange, which then results in the observed time-dependent reduction in cellular KRAS-GTP levels. For this to be the case, (i) inhibition by the drug should require KRASG12C GTPase activity. (ii) If KRASG12C GTPase activity is constant, the rate of RAS inhibition by the drug should depend on exchange factor activity. (iii) Regulating exchange factor activity should affect drug potency and the kinetics of inhibition.  We tested this hypothesis by using mutations that occur in cancer patients and affect the guanine nucleotide cycle of RAS (Fig. 3A and fig. S5A). A59G is a â€œtransition stateâ€� mutant that abrogates GTPase activity (17, 18). When assayed for their GTPase activity (Fig. 3B), KRASWT and KRASG12C generated approximately three times as much hydrolysis product as the control reactions, whereas KRASA59G or KRASG12C/A59G had little activity. Thus, KRASG12C has basal GTPase activity that is suppressed by the A59G mutation (see also Fig. 3C, lane 5 versus lane 1). ARS853 reduced KRAS-GTP levels and ERK phosphorylation in human embryonic kidney 293 (HEK293) or H358 cells engineered to express KRASG12C but not in those expressing KRASG12C/A59G (Fig. 3C and fig. S5B). The crystal structure of HRASA59G shows that Q61, a critical residue for GTP hydrolysis, is displaced in this mutant (fig. S5C). Q61L impairs in vitro GTP hydrolysis by HRAS (19), and, when expressed in HEK293 cells, KRASG12C/Q61L was less sensitive to ARS853 than KRASG12C (fig. S5D). Y64A both reduces RAS GTPase activity and impairs the interaction of HRAS with the active site of the guanine nucleotide exchange factor (GEF) SOS (18, 20, 21) (fig. S6, A to C). As expected, ARS853 caused only minimal inhibition of GTP-bound KRASG12C/Y64A or signaling driven by this mutant (fig. S6D). Thus, KRASG12C GTPase activity is required for its inhibition by ARS853.  Fig. 3 Fig. 3 Inhibition of KRASG12C requires GTPase activity and is attenuated by nucleotide exchange The percentage of KRASG12C bound to ARS853 in cells was evaluated by mass spectrometry (Fig. 3D, dotted line). GTPase-impairing mutations A59G or Y64A diminished the cellular interaction of ARS853 with KRASG12C by ~75 and ~50%, respectively. This effect was inversely proportional to their effect on KRASG12C-GTP inhibition (Fig. 3D, solid line), supporting the conclusions that ARS853 interacts preferentially with GDP-bound KRASG12C in cells and that GTPase activity is required for the drug to bind and inhibit KRASG12C.  ARS853 reduced the interaction of KRASG12C with the catalytic subunit of SOS (Fig. 3E) and attenuated SOS-mediated nucleotide exchange by KRASG12C (fig. S6E). These findings suggest that ARS853 traps KRASG12C in a GDP-bound conformation by lowering its affinity for nucleotide exchange factors. It is therefore possible that nucleotide exchange activity modulates the effect of ARS853. We tested whether RAS mutations that affect nucleotide exchange alter the sensitivity of KRASG12C to ARS853 (Fig. 3F). Y40A, N116H, and A146V increase, whereas Y32S decreases, nucleotide exchange (22â€“24). Mutations that increase exchange attenuated the effect of ARS853 on KRASG12C-GTP, whereas Y32S caused a small augmentation of its effect (Fig. 3, G and H, and fig. S6, F and G). These data imply that potentiation of nucleotide exchange reduces sensitivity to ARS853, by lowering the levels of GDP-bound KRASG12C available for drug binding.  If the inhibitor works by the trapping mechanism suggested by these data (Fig. 4A), then cellular parameters regulating nucleotide exchange ought to modulate the inhibition of KRASG12C. Exposure to epidermal growth factor (EGF), an activator of SOS (25), delayed the inhibition of KRASG12C by ARS853 by several hours compared to inhibition in serum-free medium (Fig. 4B and fig. S7A). Treatment with an EGF receptor (EGFR) inhibitor (fig. S7, B and C) or SOS-specific small interfering RNAs (fig. S7D) had the opposite effect. We determined if EGFR signaling affects the inhibition of the mutant KRASG12C allele, by expressing hemagglutinin (HA)â€“tagged KRASG12C in EGFR-mutant PC9 cells, in which nucleotide exchange depends on EGFR (26, 27). The specific effect on the mutant allele was determined by pulling down active KRAS and then immunoblotting for HA-tagged KRASG12C. ARS853 inhibited GTP-bound KRASG12C more potently in the presence of gefitinib, as compared to its absence (fig. S7E). Furthermore, EGFR inhibition enhanced the suppression of KRAS-GTP levels by ARS853 in both KRASG12C homozygous and heterozygous cells (fig. S7F). Receptor tyrosine kinase (RTK) signaling and nucleotide exchange are also subject to feedback inhibition by ERK or AKT (28). Mitogen-activated protein kinase kinase (MEK) or AKT inhibitors relieve this feedback and activated RTK signaling (29, 30). Inhibiting these pathways also attenuated the inhibitory effect of ARS853 (fig. S7, G and H). Taken together, these findings show that RTK-mediated nucleotide exchange regulates the kinetics and potency of KRASG12C inhibition in cancer cells.  Fig. 4 Fig. 4 Tyrosine kinase activation of nucleotide exchange modulates KRASG12C inhibition in cancer cells The above results also suggest that pharmacologic inhibition of the exchange reaction, as achieved, for example, with inhibitors of RTK signaling, ought to potentiate the antiproliferative effect of KRASG12C inhibitors. To test this hypothesis, we evaluated the effects of 11 inhibitors targeting RTKs, or their downstream signaling pathways, in KRASG12C-mutant cell lines. In agreement with our model, inhibiting the tyrosine kinase activities of MET, SRC, or FGFR (fibroblast growth factor receptor) enhanced growth inhibition by ARS853 in H1972 and H2030 cells, whereas MEK, ERK, phosphatidylinositol 3-kinase (PI3K), or AKT inhibitor treatment did not have a significant effect (Fig. 4, C and D). In H2122 and Calu-1 cells, the effect of ARS853 was potentiated by concurrent inhibition of EGFR (Fig. 4D). Inhibition of EGFR also enhanced the induction of caspase activity by ARS853 (fig. S7I). These findings support another report showing that EGFR is required for KRAS-driven cancer formation (31) and suggest that the RTKs responsible for modulating the timing of KRASG12C inhibition in lung cancer vary from tumor to tumor. They also provide a rationale for selecting optimal combination treatments to achieve maximal KRASG12C inhibition in patients.  We thus report an inhibitor that suppresses KRASG12C signaling and tumor cell growth by binding to the GDP-bound form of the protein. This reduces the levels of KRASG12C-GTP, because the protein retains GTPase activity and undergoes nucleotide cycling in cells. Some RAS mutants have low levels of basal GTPase activity in vitro; yet, perhaps owing to an inability to directly test such activity in vivo, these oncoproteins are assumed to be constitutively GTP-bound in cancer cells. The data presented here, however, indicate that inhibition by ARS853 requires GTPase activity and that nucleotide exchange activity is inversely related to the kinetics and/or magnitude of inhibition. The model suggests that the RAS inhibitor and exchange factors compete for binding to KRASG12C-GDP. Drug-bound KRASG12C-GDP is not susceptible to exchange and exits the cycle, thus lowering the levels of KRASG12C-GTP.  This model also predicts that genetic alterations that completely disable KRASG12C GTPase activity, or those that activate nucleotide exchange function, will confer resistance to this class of drugs. Careful selection of combination therapies may in turn prevent or delay the emergence of resistance by enabling a more potent inhibition of KRASG12C. The mechanism of action elucidated here establishes a basis for such combination therapies. Inhibitors of receptors dominantly responsible for activation of RAS nucleotide exchange are predicted to enhance activity, whereas inhibitors of downstream pathways that reactivate these receptors are likely to blunt inhibition of mutant KRAS.  KRASG12C mutations occur in 16% of lung adenocarcinomas (32), making them one of the most frequent activating genetic alterations in lung cancer, a disease responsible for approximately 1.6 million annual deaths worldwide. No effective inhibitor of mutant KRAS has yet been developed for use in the clinic. Our results suggest that KRASG12C alleleâ€“specific inhibitors, such as the one described here, merit investigation as potential therapeutics for KRASG12C-lung cancers.'"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['Text'][3154]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\"Mutations of the sequence-specific master regulator p53 that alter transactivation function from promoter response elements (REs) could result in changes in the strength of gene activation or spectra of genes regulated. Such mutations in this tumor suppressor might lead to dramatic phenotypic changes and diversification of cell responses to stress. We have determined â€œfunctional fingerprintsâ€� of sporadic breast cancer-related p53 mutants many of which are also associated with familial cancer proneness such as the Li-Fraumeni Syndrome and germline BRCA1/2 mutant-associated cancers. The ability of p53, wild type and mutants, to transactivate from 11 human target REs has been assessed at variable expression levels using a cellular, isogenomic yeast model system that allows for the rapid analysis of p53 function using a qualitative and a quantitative reporter. Among 50 missense mutants, 29 were classified as loss-of-function. The remaining 21 retained transactivation towards at least one RE. At high levels of galactose induced p53 expression, 12/21 mutants that retain transactivation appeared similar to WT. When the level of galactose was reduced, transactivation defects could be revealed suggesting that some breast cancer related mutants can have subtle changes in transcription. These findings have been compared with clinical data from an ongoing neoadjuvant chemotherapy treatment trial for locally advanced breast tumors. The functional and nonfunctional missense mutations may distinguish tumors in terms of demographics, appearance and relapse, implying that heterogeneity in the functionality of specific p53 mutations could impact clinical behavior and outcome.  Keywords: p53, breast cancer, altered-function mutations Go to: Introduction The tumor suppressor p53 is a master regulatory gene that regulates the differential expression of target genes in a sequence-specific manner in response to cellular and environmental insults (1-3). Cell cycle regulation, apoptosis, angiogenesis, replication and repair are processes interconnected within the p53 transcriptional network. p53 exerts itself as a transcription factor by binding as a homotetramer, or dimer of dimers, to a consensus response element (RE) sequence comprised of two decamer half-sites [RRRCWWGYYY]2 (where R= purine; W= A/T; Y=pyrimidine) which vary between target genes. While the canonical consensus sequence allows for spacing between the half-sites of up to 13 nucleotides, we recently showed that transactivation was greatly reduced as the separation between decamers is increased beyond 2 nucleotides (4). Activation of a p53 target gene is dependent on a matrix of factors including cell type, stimuli, post-translational modifications and transcriptional co-factors (5).  The importance of p53 as a tumor suppressor and sequence-specific transcription factor in human cells is highlighted by the occurrence of p53 mutations in the majority of cancers (6). Interestingly, p53 is unique in comparison to other transcription factors in that over 75% of mutations that occur in this tumor suppressor are single amino acid changes which result in missense mutations (7). These missense mutations predominantly occur in the DNA binding domain (DBD) of the protein (>80%) (7). At the molecular level, p53 mutations found in cancers, including breast cancer, are usually associated with loss of the ability to maintain proper cell cycle checkpoints, suppress transformation caused by oncogenes, induce apoptosis and maintain the integrity of the genome (1, 8).  Specific mutations in p53 are known to denature the native protein or abrogate its ability to bind DNA thus completely abolishing its function. Such mutations are thought to provide a selective advantage within cancerous cells by forming a hetero-tetramer with WT p53 and functioning in a dominant-negative fashion (9, 10). Alternatively, gain-of-function mutations can potentiate tumorigenesis through oncogenic mechanisms including aberrant transcriptional regulation of either known or novel target genes--presumably through structure-selective DNA binding or protein-protein interactions (9-11). Many p53 missense mutations have been described that retain function as sequence-specific transcription factors such that the ability to regulate cellular responses is altered, but not completely lost (2, 12).  Functional mutations that alter the transcriptional capacity of the p53 master gene have been identified as super-transactivating, change-in-spectrum or overall downward modulation of transactivation. For example, change-in-spectrum mutants may be capable of regulating genes containing a strong RE, such as p21, but unable to regulate those with a weak RE, such as Bax. This is consistent with the observation of mutant p53s that still induce cell cycle arrest, yet lose the ability to activate apoptosis (13-15). In addition, p53 mutations may alter the active binding sites of potential transcriptional cofactors, thus diminishing the potential maximal level of transcriptional response. Modifications in the transcriptional network due to altered-function mutations may result in cellular responses that impact genome stability, repair, replication, and programmed cell death. Varying patterns of cellular responses, including apoptosis and survival have been elicited in human cells as a consequence of distinct altered-function p53 mutations (16). Furthermore, the aberrant biological consequences of specific altered-function mutations can be influenced by specific cell type and activating stimuli.  Mutations in p53 are associated with approximately 25% of sporadic cases of breast cancer, a frequency lower than that in other sporadic cancers, such as lung and colorectal carcinomas. However, sporadic p53 mutations occur at much higher frequencies in BRCA1/2 germline-associated breast cancers possibly due to a decreased efficiency to repair damage (17, 18). BRCA1/2 and p53 are involved in maintaining genome stability by controlling aspects of homologous recombination and repair, centrosome regulation, cell cycle checkpoints and transcription (19), where loss of either increases the likelihood of cancer (20). Interestingly, BRCA1-associated cancers have an altered spectrum of p53 mutations which may reflect changes in mutagenesis and/or selection for the acquired mutations (17). While BRCA1 mutations are largely absent in somatic breast tumors, silencing of the gene through hypermethylation has been reported in sporadic cases (21). Such epigenetic changes have been reported to associate with estrogen receptor negative (ER-) tumors and occur concomitantly with p53 mutations (21).  At the clinical level, p53 mutations in breast cancer have been associated with poor prognosis, earlier on-set, increased aggressiveness of tumors, aneuploidy, and adverse responses to chemotherapeutic treatments (22). Studies that classify breast cancers based on gene expression profiling have shown p53 mutations are more frequent in the hormone receptor-negative subtypes such as the HER2+/ERâˆ’ [human epidermal growth factor-2 positive/estrogen receptor negative] and the basal-like subtypes [ERâˆ’, PRâˆ’ (progesterone receptor), HER2âˆ’, cytokeratin 5/6+, and/or HER1+] (23-25). Based on a recent population-based study, these subtypes were prevalent among African American and/or premenopausal women and correlated with a more aggressive disease and shortened survival, irrespective of lymph node status (25). Regardless of subtype, p53 status (WT or mutant) also displays a signature expression profile in breast tumors which is a prognostic indicator of patient survival, where WT p53 associates with a more favorable outcome (23, 26, 27).  We have employed a newly developed model system in diploid yeast (4) to analyze the functional consequences of p53 missense mutations found in breast cancers on gene activation endpoints in the p53 transcriptional network at various levels of p53 expression. Transactivation capacities of WT and mutant p53 have been determined using a qualitative and a quantitative reporter, and a â€œfunctional fingerprintâ€� was established for each p53 variant towards a subset of human REs that are representative of p53-dependent cellular responses. We have determined that p53 missense mutations found in sporadic and familial breast cancers can retain function and the alterations in transactivation are often subtle where differences can be exaggerated by changes in p53 levels. While patient numbers are limited, the separation of missense p53-associated breast cancer mutations into functional (which include fully functional and altered function) versus nonfunctional classes appears to associate with prognostic factors and outcome in a largely locally-advanced patient population treated with chemotherapy prior to surgery. Functional fingerprinting of cancer-associated p53 mutants may thus be a useful tool for understanding tumor biology and behavior.  Go to: Materials and Methods Isogenomic diploid yeast strains  Two panels of isogenic haploid yeast strains, a â€œp53-hostâ€� and â€œresponse element (RE) reporterâ€� strains, were developed in the budding yeast, S. cerevisiae with the delitto perfetto site-directed mutagenesis system as previously described (Figure 1) (4, 79). Each â€œp53-hostâ€� strain, yAT-iGAL::p53 (MATa leu2-3,112 trpl-1 his3-11,15 can 1-100 ura3-1, trp5::pGAL1:p53:cyc1-Ter, lys2::HygroR), contains the wild type or mutant p53 cDNA controlled by the inducible, â€œrheostatableâ€� GAL1 promoter (2) integrated at the TRP5 locus on chromosome VII. p53 mutations were constructed using a derivative of the previously described p53 host strain containing a CORE cassette (CO, counterselectable, KLURA3; RE, reporter, KanMX4 resistance gene) integrated at various nucleotide positions spanning the p53 cDNA (4). Modifications of the p53 cDNA were performed using the delitto perfetto approach so that CORE cassettes were replaced with an oligonucleotide containing the mutation of interest to generate a full-length mutant p53 cDNA. (Oligonucleotide sequences are available upon request.) Replacement of the CORE was confirmed by selection on 5-FOA and kanamycin sensitivity. Specific p53 alterations were confirmed by colony PCR and sequencing (Big dye, Applied Biosystems, Foster City, CA). The second panel of isogenic strains, constructed previously, contain human target p53 REs upstream of the CYC1 minimal promoter and either the ADE2 or firefly luciferase reporter (2, 4). The RE reporter strains are also isogenic with the p53 host strains, but LYS2 and Hygros. Mating of the reporter and p53-host strains followed by selection for diploid cells on Lysâˆ’ Hygro+ plates, results in isogenomic yeast that enable the assessment of the transactivation potential for WT or mutant p53 proteins towards individual REs in the p53 transcriptional network.  Figure 1 Figure 1 Evaluation of transactivation potential towards REs by WT and mutant p53s at variable levels of expression Qualitative ADE2 color assay  Single colony isolates of the p53-inducible RE-ADE2 reporter strains were streaked onto an YPDA control plate containing glucose and high levels of adenine and grown to equivalent amounts at 30Â°C. The plates were then replica plated onto a series of 9 plates containing selective media with low levels of adenine [5 mg/L], 2% raffinose and increasing galactose (0, 0.001, 0.002, 0.004, 0.008, 0.016, 0.032, 0.064 and 0.128%). Transactivation capacities for the p53 mutants were determined after three days of growth at 30Â°C by the ability of the mutant to produce a change in colony pigmentation. Transactivation of the ADE2 gene, which is a direct readout of p53 interaction with the specific RE, results in white colonies where decreased or loss-of-transactivation of the ADE2 results in pink and red colonies, respectively (32). Transactivation capacities for the p53 WT and mutants were determined after three days of growth at 30Â°C by the ability of the mutant to produce a change in colony pigmentation. Colony pigmentation was manually scored on a scale of 1 to 5, where 1 is no apparent transactivation (red colonies) and 5 is strong transactivation (white colonies) (Supplemental figure 2).  Quantitative luciferase assay  Diploid yeast strains containing GAL1::p53 (WT or mutant) crossed with a specified RE-luciferase reporter were grown overnight in 5ml YPDA plus adenine [200 mg/L] rich media. Overnight cultures were diluted 1:50 in H2O. For each measurement, 1 ml of the diluted culture was spun down, washed of residual glucose with H2O and re-suspended in 2mL synthetic complete -LYS media plus 2% raffinose supplemented with increasing amounts of galactose (0, 0.002, 0.004, 0.008, 0.010, 0.012, 0.016, 0.020, 0.024, 0.028 or 0.032%). These cultures were grown overnight (~ 18 hr) at 30Â°C to ~2 âˆ’ 4 Ã— 107 per ml (late log early stationary). The 2 ml cultures were spun down and the supernatant was aspirated. The remaining pellet was resuspended in 100 Î¼l reporter lysis buffer (Promega, Madison, WI) and an equivalent amount of 425-600 micron acid-washed, glass beads was added (Sigma, St. Louis, MO). Samples were homogenized for 30 seconds in the Biospec Products, Inc. mini-bead beater (Bartlesville, OK), briefly incubated on ice and spun for 20 minutes at 16k relative centrifugal force (rcf) in an Eppendorf 5415R centrifuge (Batavia, IL) to separate the soluble protein fraction. The standard protocol recommended by the manufacturer (Promega; Madison, WI) was performed for the luciferase assay system starting with 10 Î¼l of protein extract. Luciferase activity was measured from 96-well, white optiplates (Perkin Elmer, Waltham, MA) in a Wallac Victor2 multilabel counter (Perkin Elmer, Waltham, MA). Light units were standardized per Î¼g protein as determined by a Bio-Rad protein assay (Bio-Rad; Hercules, CA).  Western analysis  Diploid yeast strains containing GAL1::p53 (WT or mutant) crossed with the p21-5â€² RE-luciferase reporter were grown as described above. Overnight cultures containing 0.024% galactose were harvested, lysed in 35 Î¼l reporter lysis buffer (Promega, Madison, WI) plus 2% protease inhibitors (cocktail for use with fungal and yeast extracts; Sigma, St. Louis, MO) and processed in the same fashion as those used in the luciferase assay. Protein concentrations were measured with the Bio-Rad protein assay according to the standard protocol (Bio-Rad; Hercules, CA). 50 Î¼g of total protein was run on 4-12% BisTris NuPAGE and transferred as previously described (32). The p53 protein was detected with a mix of DO7 (BD BioSciences Pharmenigen) and pAb1801 (Santa Cruz) antibodies unless otherwise specified according to the manufacturerâ€™s protocol. Bands were detected using horseradish peroxide-conjugated secondary antibodies (Santa Cruz) and the enhanced chemiluminescence (ECL) detection system (Amersham, Cleveland, OH, USA). Membranes were stained with Ponceau S to determine efficiency of protein loading.  Mutation analysis from patients participating in a neoadjuvant trial  p53 mutational analysis was performed on untreated breast cancer tissues obtained from women participating in two clinical-translational trials, a single institution study from the University of North Carolinaâ€“Lineberger Comprehensive Cancer Center (UNC-LCCC 9819), and a multiinstitutional cooperative group trial sponsored by the National Cancer Institute (CALGB 150007). In these correlative studies, women with locally advanced breast cancer were treated first with an anthracycline generally followed by taxane-based chemotherapy (with or without trastuzumab, depending on Her2 status). Both trials involved acquisition of breast cancer core biopsy tissue before treatment with any chemotherapy and were designed to examine molecular markers predicting response to cytotoxic chemotherapy (29). These studies were approved by the institutional review boards of the University of North Carolina at Chapel Hill and participating institutions through the Cancer and Leukemia Group B (CALGB). All study subjects gave written informed consent to participate in the clinical trial and the correlative science studies. p53 gene mutations were assessed using a two-tiered screening strategy. A pre-release version of the p53 AmpliChip array (Roche Molecular Systems) was first used to detect point mutations and 1 base pair deletions. Samples identified as potentially positive by the AmpliChip was sequenced to confirm the mutation. Specimens that were mutation-negative by AmpliChip were evaluated by single strand conformational polymorphism (SSCP) analysis in p53 exons 2-11 to detect deletions larger than 1 base pair and insertions (80). Mutations identified as potentially positive by SSCP were sequenced to identify the mutation. To rule out the possibility that mutations occurred due to PCR errors, we re-amplified and re-sequenced all mutation-positive specimens.  Go to: Results Functional fingerprinting of p53 missense mutations associated with breast cancers  Budding yeast lack endogenous p53 and have been used as an in vivo test tube to analyze directly interactions between p53 and REs in a cellular environment (2, 3). Recently, we developed a diploid yeast model system to address the contribution of RE sequence, organization and level of human p53, as well as the consequences of mutations upon p53-mediated transactivation (Figure 1) (4).  We sought to define how specific p53 missense mutations found in breast cancers interfere with p53 function by assessing the ability of mutant p53 to transactivate from a panel of REs associated with p53-dependent downstream target genes (Table 1). Fifty missense mutations were chosen for examination if they were identified in cases of sporadic breast cancers. Importantly, 20 were identified in patients undergoing neoadjuvant treatment for locally advanced breast tumors and participating in clinical trials examining biomarkers predicting response to sequential anthracycline- and taxane-based chemotherapy prior to surgery (28, 29). Furthermore, 18 also associate with familial BRCA1/2 cancers and/or are found as germline mutations in Li-Fraumeni Syndrome (LFS), Li-Fraumeni-like Syndrome (LFL), and/or familial history (FH) cancer patients (Table 1). Of particular interest were mutations present in the L2 loop, L3 loop or zinc binding regions of the protein which have been correlated with breast cancers that are often nonresponsive to chemotherapeutic treatments including doxorubicin, tamoxifen, and/or combined therapies of 5-flurouracil and mitomycin (19, 30, 31).  Table 1 Table 1 p53 missense mutations associated with breast cancers: functional status, frequency, and features. The yeast ADE2 plate color assay (32) was used to determine functional fingerprints for WT and p53 missense mutations based on their ability to transactivate from 11 different human REs at variable levels of protein expression (Figure 2 and Supplemental Figures 1 and 2). Briefly, single colony isolates of yeast strains containing the mutation and RE of interest were replicated onto plates containing increasing concentrations of galactose where the ability of the p53 variant to drive transactivation from a specific sequence could be assessed based on colony pigmentation (see Materials and Methods and Supplemental Figure 2). The REs analyzed are associated with known human p53 target genes involved in cell cycle, DNA repair, apoptosis, angiogenesis, and p53 regulation (Supplementary Table 1). Mutations were categorized as fully functional if they were indistinguishable from WT p53 in transactivation capacity or altered function if the allele retained the ability to function from at least one RE(s), but deviated from WT p53 in transactivation capacity from the REs examined at any of the levels of p53 expression examined.  Figure 2 Figure 2 Functional fingerprints of p53 mutants reveal subtle transactivational differences Among the 50 missense mutations, 29 were classified as loss-of-function due to their inability to transactivate from any RE (Table 1). The remaining 21 (42%) mutations were able to function from at least one RE, where the transcriptional capacities varied from different levels of functionality to fully functional. Among the 21 functional mutations, 9 were clearly altered in transactivation capacity at high levels of galactose (0.128% galactose) (Supplemental Figure 1) and displayed a change-in-spectrum for REs transactivated, as exemplified by reduced or complete lack of transactivation from the 14-3-3Ïƒ and PCNA REs by P151A and R283P, respectively, in Figure 2. Of these 9 mutations, 3 (Y220C, M237I and P278A) retained the ability to transactivate from only the strongest REs, p21-5â€² and P53R2 when p53 expression was induced with high levels of galactose (and, in the case of P278A, MDM2 which contains two full-site REs) (Supplemental Figure 1).  At high levels of galactose, the remaining 12 of the 21 mutants (L130V, A138V, C141W, P151H, R174K, R174W, P190L, H214R, R267Q, V272L, E285K, and R337H) looked similar to WT in their transactivation capacity (i.e., L130V in Figure 2, Supplemental Figure 1). Transactivation from three biological replicates was either indistinguishable from WT p53 or nearly identical with the exception of 1 or 2 REs at 0.128% galactose. However, when the levels of galactose were reduced, subtle transactivation defects were revealed that further differentiated 6 of the 12 mutants (underlined) from WT p53 at 3 or more REs. Similar to previous studies (2), the deviation from WT p53 at lower levels of induction consisted of both reduced and enhanced transactivation capacities from specific REs, such as the ability of L130V to transactivate from the Cyclin G and AIP1 REs at 0.008% galactose, respectively (Figure 2). The remaining 6 mutants remained similar to WT p53 in transactivation capacity even at low levels of galactose with the exception of subtle variation at 1 or 2 REs. For example, A138V was only observed to be slightly reduced in transactivation compared to WT p53 towards PCNA at 0.008% galactose (Supplemental Figure 1).  Luciferase assays confirm transcriptional anomalies  The functional status of the 21 missense mutants that retained function and several loss-of-function mutants was examined with a luciferase reporter assay that provides the opportunity to quantitate transactivation from REs in late log phase growing cells. The transactivation capacities from the p21-5â€², GADD45 and 14-3-3Ïƒ REs were comparable to those with the color plate assay in terms of classifying functional status; however, the assay provides greater discrimination between mutant and WT p53 transactivation. Assessment of the altered-function mutants with the luciferase assay showed varying degrees of functionality from the p21-5â€² RE, where the maximal level of transactivation was dependent on the specific mutation (Figure 3). Similar to the results with the plate assay, several mutants (i.e., R267Q) differed from WT p53 in their ability to transactivate from the p21-5â€² RE only at low levels of galactose (low p53 expression), whereas other mutations (i.e., L194P) displayed a decreased ability to transactivate from the p21-5â€² RE at all levels of induction and corresponding p53 expression. Of the 6 fully functional mutations examined that were similar to WT p53 in terms of transactivation from the strong p21-5â€² RE in both the plate and luciferase assays, one (H214R) showed an altered transactivation potential when assessed for transactivation from the weaker GADD45 RE in the luciferase assay (Figure 4). H214R had an increased ability to transactivate from the GADD45 RE in comparison to WT p53, whereas the remaining fully functional mutations, including A138V and R174W remained indistinguishable from WT p53. Change-in-spectrum mutations which retained transactivation function from some REs, but were devoid of function from others, were also verified with the luciferase assays. As shown in Figure 5, Y220C was able to transactivate from the strong p21-5â€² RE, but to reduced levels; the maximal level of transactivation was comparable to that for WT p53 transactivating from the weaker 14-3-3Ïƒ RE. The Y220C mutant was actually unable to transactivate from the 14-3-3Ïƒ RE.  Figure 3 Figure 3 Assessment of WT and mutant p53 transactivation towards the p21-5â€² RE using a luciferase assay Figure 4 Figure 4 Transactivation from the GADD45 RE distinguishes altered-function mutants from WT p53 Figure 5 Figure 5 Change-in-spectrum p53 missense mutations can eliminate REs from the p53 transcriptional network The transactivation profiles with increasing levels of galactose (i.e., increased p53 expression) were similar between the DBD mutants and WT p53 where initial induction occurred between 0.004 â€“ 0.008% galactose and maximal levels of transactivation were between 0.016% - 0.024% galactose (Figures (Figures33--5).5). This was not observed for the tetramerization domain mutant R337H (Figure 6). Although maximal levels of transactivation appeared similar to WT p53, the R337H mutation clearly altered transactivation from the p21-5â€², 14-3-3Ïƒ and GADD45 REs, requiring higher levels of p53 expression than WT to initiate transactivation (Figure 6). The requirement for increased p53 levels necessary for initial transactivation by the tetrameric mutants appeared dependent on the strength of the RE. The R337C mutation resulted in overall reduction in transactivation.  Figure 6 Figure 6 Altered-function mutations at the same codon in the tetramerization domain display different transactivation capacities Altered-function mutants can maintain protein levels comparable to WT p53  Protein levels were analyzed at 0.024% galactose (within the range of expression where transactivation was shown to plateau in the luciferase assays) by Western analysis for the 21 mutations that retained function and a representative loss-of-function mutant (Supplemental Figure 3). Of the functional mutants, 13 displayed similar levels of protein to WT p53, as did the loss-of-function missense mutation. However, seven mutants (C141W, L194P, Y220C, M237I, P278A, E285K and R337C) had reduced expression compared to WT p53. Surprisingly, several of these mutants were shown to efficiently function from multiple REs in the plate and luciferase assays. Detection of the p53 protein with additional antibodies that recognize different epitopes was consistent with most of these mutants having reduced levels of protein in comparison to WT p53 (Supplemental Figure 4).  Functional status and clinical response  To address how different p53 mutations might influence response to chemotherapeutics, we examined transcriptional functional status of p53 missense mutations found in breast cancers in relation to clinical manifestations. Twenty-nine unique p53 missense mutations analyzed for functionality in this study (20 mutations) or in a related haploid yeast system (14, 33) have been identified in 46 patients with locally advanced breast tumors (primarily ductal carcinomas). The patients were participants in clinical trials that monitor clinical and pathologic responses to neoadjuvant treatment prior to surgery (Supplemental Table 3). As summarized in Table 2 and Supplemental Table 3, among the 46 patients, 10 of the missense mutations (11 patients) resulted in p53s that retained function (9 altered and 1 fully functional mutant) and 19 were loss-of-function (35 patients). This corresponded to ~24% of patients (11 among 46) having functional mutations, suggesting a significant group that might be approached differently regarding treatments.  Table 2 Table 2 Clinical characteristics and response to therapy in patients with p53 functional or nonfunctional missense mutants. There were 29 unique breast cancer-associated p53 mutations identified among 46 patients. While the number of p53 missense mutations examined is limited, there are emerging trends that may differentiate patients with functional (altered or fully functional) versus nonfunctional p53 missense mutants (Table 2 and Supplemental Table 3). Among the tumors with somatic p53 missense mutations where HER2, ER and PR status could be assessed, functional missense mutations were more common in HER2-negative tumors (7 of 25, 28%) compared with HER2-positive (2 of 15, 13%). Among triple negative (ER, PR, and HER2) tumors with p53 missense mutations, 5 of 16 (31%) carried functional mutations. There appeared to be a higher frequency of functional/total missense mutations among Caucasians and Asians (7/29 and 2/3, respectively) than among African American patients (2/14, 14%). Functional mutations appear to be associated with good prognostic factors such as low incidence of nodal involvement (2 of 11, 18%). Conversely, compared with functional mutations, patients carrying nonfunctional mutations were more likely to be stage III at diagnosis (53 vs 27%), have high grade tumors (59 vs 36%) and to relapse in distant sites (34 vs 10%), as well as local sites (12 vs 0%) although the numbers are small. In terms of responsiveness to chemotherapy, the pathologic complete response (pCR; i.e., eradication of tumor) was similar between patients with functional p53 mutations and nonfunctional missense mutations (21% and 27%, respectively). However, women with nonfunctional mutations were more likely to die (36% vs 20%) at 3 years post-treatment than those with functional mutations. While suggestive of trends, none of these differences reached statistical significance.  Go to: Discussion Single amino acid changes in the p53 master regulator protein that differentially impact transactivation may result in the selective advantage of specific mutations in certain tissue types or stages of neoplastic transformation, as well as alter the responsiveness to or the efficacy of chemotherapeutic agents. We have used a diploid yeast system to analyze the potential transactivation capacity for a set of p53 missense mutations associated with breast cancers. The 50 missense mutants examined represent approximately 18% of all somatic p53 mutations reported in the International Agency for Research on Cancer (IARC) TP53 mutation database and 20% of all p53 missense mutations documented in breast tumors (7). Among the mutations analyzed, one (K305M) occurs at an acetylation site in a non-structured portion of the protein, 2 (R337H and R337C) are in the tetramerization domain and the remaining 47 are distributed across the sequence-specific DNA binding domain.  Functional fingerprinting established that 21 out of the 50 missense mutants associated with breast cancer can retain p53 function. The effect on transactivation appears dependent upon the specific amino acid alteration such that even different missense mutations at the same residue can vary in the impact on p53 functionality. This is exemplified at codon 194 where changing the leucine residue to an arginine (L194R) results in loss-of-function, whereas a proline (L194P) results in altered function.  The majority of the altered-function mutations analyzed in this study do not seem to be the result of a general reduction in transactivation from all REs. Rather, they appear to be change-in-spectrum mutants that impact the REs differentially. Since each functional p53 missense mutation had a unique functional fingerprint (Supplemental Figure 1) ranging from severely altered to fully functional, there may be diverse cellular effects. The functional consequences of mutations appear not to be predictable simply by assessments of conservation, topology, or structural models, emphasizing the need to address the function of p53 mutants in vivo (Supplemental Table 2). For example, a recent structure-based analysis of mutations in the DBD does not appear indicative of the in vivo transactivation capacity. Over 45% of the predictions based on computational geometry (which assessed residual score profiles derived from Delaunay tessellations) (7, 34) differed with results from our transactivation assays (Supplemental Table 2).  Importantly, the differences in transactivation for all of the mutations could not be attributed simply to protein stability since most of the p53 mutants were expressed at levels comparable to that for WT protein. Of the mutations that had a reduced level of protein expression, the C141W and E285K mutants were only modestly compromised for transactivation capacity in comparison to WT p53, whereas others such as M237I and Y220C were severely compromised. It is possible that reduced levels of protein may be due to an increased level of degradation within the cell due to conformation changes. This could also explain associated temperature sensitivity of some alleles, specifically Y220C, M237I, and E285K (35-37).  However, regarding these low expressing mutants, the issue arises as to whether cellular p53 protein expression is comparable between yeast and tumor cells. There are examples where expression in yeast is matched by low or undetected levels in breast cancer cell lines (i.e., E285K in BT474 and MDA-MD-134VI cells (38, 39) and Y220C in HCC1419 cells (40, 41)) or alternatively being accumulated in the cells (i.e., M237I in SUM149PT cells (38)). Regardless of the amount of p53 protein expressed in yeast, the results are informative for those cases where the mutant proteins retain transactivation ability as this indicates the potential for mutant protein to function in mammalian cells.  Breast cancer associated mutations in the DBD can modulate p53 transactivation  The DBD of p53 consists of a Î²-sandwich which provides a scaffold for two large Î²-loops, L2 and L3 that are stabilized by a zinc ion and a loop-sheet-helix motif (42, 43). Mutations in the DBD have been postulated to affect the binding affinity of p53 towards REs by abolishing DNA contacts, decreasing the thermodynamic stability of the protein, causing local distortions in the DNA binding surface, or enhancing the loss of the Zn ion (44, 45). Mutations in the L2/L3 loops are predicted to be highly destabilizing to the tertiary structure of p53, can cause chemical shifts which alter the DNA binding surface and/or can alter the response to chemotherapeutics (7, 19, 30, 31, 46).  Of the DBD mutations analyzed for transactivation potential, 16 correspond to residues in the L3 loop and 10 are within the L2 loop; among these 26 mutants, 6 are also zinc-binding residues. All 6 mutations (C176F, H179R, C238F, C242F, C242S, C242Y) that interfere with the histidine or cysteine side chains involved in coordination of the Zn ion rendered the protein nonfunctional in terms of transactivation, emphasizing the vital role of Zn in sequence-specific DNA binding and stabilization of the p53 protein (44, 47). Similarly, all the missense mutations analyzed in the L3 loop, which binds the minor groove of DNA and partakes in the dimerization interface between core domains (45), were loss-of-function mutations with the exception of M237I which was very weak for transactivation. However, 4 mutants (R174K, R174W, P190L and L194P) in the L2 loop retained function of which several (underlined) displayed subtle alterations in transactivation capacity. These results suggest that mutations in the L2 loop, which functions as a support for the L3 loop, may be less detrimental to transactivation potential than those of zinc binding or in the L3 loop.  Tetramerization mutations may alter the level of p53 required for transactivation  Although the majority of p53 missense mutations occur in the DBD, several missense mutations have been found in the tetramerization domain that are associated with germline syndromes and are found in sporadic breast tumors. R337C is a partial function mutation associated with LFS (48). R337H has been associated with pediatric cases of adrenocortical carcinoma (ACC); however, it may be a low penetrant LFS or LFL allele as well (49-52). The functional fingerprints varied between these two altered-function mutations in that R337C had a greater impact on p53 transactivation displaying an overall dampening effect from the various REs. This reduced transactivation may reflect a greater instability of the protein as depicted in the protein analysis (Supplemental Figures 1-4). Contrary to previous reports, the p53 missense mutation R337H was not a silent mutation, but displayed altered function when examined in the ADE2 phenotypic assay (Supplemental Figure 1). At high levels of galactose induction (i.e., high levels of p53 expression from the GAL1 promoter), R337H looked identical to WT p53 in the color assay. At low levels of galactose, R337H had a reduced ability to transactivate from REs in comparison to WT p53, presumably due to its reduced ability to form tetramers. Interestingly, the luciferase assay revealed a pattern of transactivation that was unique to R337H where higher levels of protein expression were required to detect transactivation. This altered pattern of transactivation may be a manifestation of the novel features of R337H which include a pH-dependent instability and formation of amyloid-like fibrils (53, 54).  Subtle variation in transactivation capacity of altered-function missense mutations  Especially interesting is the observation that many of the mutants looked similar to WT p53 when examined at high levels of galactose (i.e., high p53 expression) yet were altered function at lower levels of galactose, consistent with our earlier findings (32). Importantly, these mutations would not have been distinguished from WT p53 in typical functional assays where p53 is expressed at high levels from a constitutive promoter. Thus, by reducing the level of transcription with the rheostatable promoter, it was possible to unmask subtle transcriptional discrepancies. These mutants might have unique properties in terms of biological consequences. Possibly, the mutants function similarly to WT p53 for gene expression from target genes under high stress and chemotherapeutic conditions, but differently under conditions of low p53 expression. Also, other transcription factors could further modify the response (3).  Although mutations in p53 are often associated with nuclear accumulation of the protein, recent evidence indicates this may not always apply in vivo. For example, Tsuda and Hirohashi (55) showed with immunohistochemical analysis of over 50 human breast-cancer tissue specimens that nuclear accumulation of the p53 protein was dependent on the type and position of the mutation, where missense mutations did not always result in stabilization of the protein. Similarly, in a study comparing immunohistochemical analysis with cDNA-based sequencing using ~300 primary breast tumor samples (56), in over 30% of the cases where a mutation was observed through sequencing there was not a corresponding accumulation of p53 protein.  Recent results with knock-in mouse models indicate levels of mutant p53 can be regulated in both normal and some tumor cells (57, 58). The accumulation of mutant protein does not occur until additional mutations are acquired in genes which may disrupt the p53 degradation pathway such as MDM2 or p16INK4a (59). Although the subtle altered-function mutations identified in the present study retain the ability to function from the MDM2 RE, Lukashchuck and Vousden (60) have shown that the ability of p53 to transactivate MDM2 is not essential for degradation of the mutant p53 protein. Rather, additional E3 ubiquitin ligases, such as CHIP (C terminus of Hsc70-interacting protein), can target mutant p53 for degradation independent of ubiquitination by Mdm2, where Mdm2 plays a role in delivering the ubiquitinated proteins to proteasomes.  The functional relevance of p53 at low expression levels is beginning to be elucidated and highlights the need to understand mutants under such conditions. Espinosa et al. (61) observed that p53 occupies some target REs, including p21, prior to overall p53 stabilization. There is transcriptional initiation from these REs, but the transcriptional machinery stalls prior to elongation. Such regulation may be required for a rapid response to cellular stress. In addition, there is a transcriptional-dependent role for p53 in promoting cell survival, as well as modulating glucose metabolism and reactive oxygen species at basal levels of p53 expression (62, 63). For example, p53 has been found to target the sestrins (i.e., SESN1 and SENS2) and TIGAR (TP53-induced glycolysis and apoptosis regulator) to stimulate antioxidant and pro-survival signals (62, 63). The loss of transactivation function at low levels, as observed for some of the subtle, altered-function mutants, might lead to alternative modes of promoter selectivity, and/or provide the opportunity for competing transcription factors to bind promiscuously to p53 target elements. Furthermore, p53 has also been found to promote, presumably in a transcriptional-independent fashion, global chromatin relaxation (64). This appears to influence genomic repair in response to UV stress at exposures that are lower than those required for its activation as a transcription factor.  It is possible that mutations that subtly affect transactivation are acquired early in tumor development and when combined with mutations in other genes contribute in an additive fashion to the complex cancer disease. For example, the altered-function P190L and H214R mutants that were indistinguishable from WT at higher levels of galactose-induced expression are associated with germline BRCA1 and BRCA2 mutations, respectively. Such mutations in p53 may be an underlying contributor to the genomic instability observed in BRCA1-associated breast cancer cases and the functional status of individual p53 missense mutations may impact the degree of genetic imbalance. Interestingly, a recent hierarchical clustering analysis using immunohistochemistry profiling to determine the relatedness of tumors has established that the extent of genomic instability correlates with specific breast cancer subtypes, where the basal subtype (the subtype in which p53 mutations are frequently observed) had the highest number of genomic aberrations in both sporadic and familial BRCA-associated cases (65-67).  In addition, inherited p53 mutations may influence tumor type and penetrance of the disease (57, 58, 68, 69). LFS and LFL germline disorders--which often harbor a p53 mutation--display an array of early onset, tissue specific tumors of which breast tumors are among the most frequent observed (7, 68). Recent studies which have assessed the functional status of p53 germline mutations using yeast-based assays have related severity of inherited p53 missense mutations in terms of transcription functionality with clinical manifestations (7, 70). Partial deficiency alleles, defined by the ability to transactivate from at least one RE to 25% of the levels obtained by WT p53, are associated with a less severe family history, lower number of tumors, later onset of disease in comparison to severe deficiency (loss-of-function) alleles and a higher risk of breast tumors.  In another recent study (71), the frequency and average size (bp deletion or duplication) of DNA copy number variation is enriched in carriers of germline TP53 mutations within LFS families in comparison to those with WT p53 or in a healthy population. The clinical phenotypes which arise in later generations may correlate with greater genomic instability, as well as specific germline p53 mutation. Thus, the wide spectrum of transcription potentials of the 12 p53 missense mutations associated with germline disorders in the current study (ranging from nonfunctional to altered and subtle, or fully functional) can be expected to result in varying phenotypes where the severity of the disease is likely influenced by the extent of p53 functionality.  Functional status and clinical response  We examined transcriptional functional status of p53 missense mutations found in breast cancers in relation to clinical manifestations (Table 2 and Supplemental Table 3). While the number of breast cancer-associated functional p53 mutants was small, there were trends described in the Results that suggest differences in presentation and outcome between functional versus nonfunctional missense mutations. The nonfunctional missense mutations are associated with clinical responses similar to those in our studies with null mutations (data not shown) which are known to have poorer prognosis and reduced survival as compared to patients with tumors that are WT for p53 (72). We have found that functional p53 mutations appear to have better disease-free survival compared with loss-of-function mutations (Table 2).  In terms of clinical response to chemotherapy, we observed in the present study a higher response rate for tumors expressing loss-of-function p53 mutations which may seem counterintuitive. However, tumors with p53 mutations in general are more chemo-sensitive, possibly because the breast cancer subtypes which usually have a higher proportion of p53 mutations (e.g. the basal-like) are more highly proliferative or have aberrant DNA repair functions (23, 25). Also, the cumulative response to anthracycline/cyclophosphamide followed by taxane therapy was determined in this study. While there is some indication from the literature that p53 mutant status could confer different responses to these single agents (73-75), it is difficult to predict responses to the sequential treatment. While previous studies have investigated correlations between p53 mutations and pathologic variables in breast cancers, few studies have attempted to correlate the transcriptional activity of specific mutations with clinical phenotypes. In a large scale study of a cohort of approximately 1,800 woman (72), the presence of a p53 mutation was associated with high grade, positive node status, loss of hormone receptors and greater risk of death due to breast cancer within a 10 year follow-up. However, when the functional status of the missense mutations was taken into consideration, no correlation was found between p53 transactivation activity and patient survival. Importantly, the functional status of the missense mutations was determined with another yeast-based functional assay developed by Kato et al. (12) that uses a high p53 expression plasmid along with a high copy RE reporter plasmid system.  Functional analysis based on the Kato et al. system is available at the IARC p53 mutation database for 49 of the 50 mutations examined in the present study (7). Interestingly, a comparison between the two data sets shows less than 65% agreement in terms of overall transactivation functionality (Supplemental Table 2). Only 8/49 or 16% of the missense mutations were previously identified as functional as compared to 21 mutations in the present study system. The large discrepancy between the two systems may be in part due to the method of classifying a mutation as retaining function (Supplemental Table 2) (7, 12). For example, we concluded that the R337C was an altered-function missense mutation whereas within the IARC database it is considered to be a nonfunctional mutation although it did display transactivation for 3 of the 8 REs analyzed (WAF1, MDM2 and P53R2) (7). Furthermore, in a separate analysis, the Kato et al. group (37) found that greater than 140 missense mutations were temperature sensitive for transactivation capacity towards at least one RE; this indicated there may be cellular conditions under which these mutations may become functional. Interestingly, 5 of these temperature sensitive mutations which were reported as loss-of-function in the IARC database were classified as altered-function in the present study (P151A, H214R, V272L, R283P and E285K). Another mutation Y220C had also been categorized as loss-of-function; however, we have diagnosed this as a weak altered-function protein and previously it was reported as temperature sensitive in mammalian cell assays (35).  Go to: Conclusions Given the heterogeneity of breast cancers, understanding the consequences of altered-function mutations on the p53 transcriptional network will help elucidate how specific mutations predispose and/or contribute to the development, penetrance and phenotype of breast cancers. Among fifty p53 mutations identified in breast cancers, 21 had altered functionâ€”not simply complete lossâ€”towards at least one RE. Although the transcriptional effects associated with these mutations are often subtle, we found that by reducing the level of transcription with the rheostatable promoter, it is possible to address the retained functions of mutant p53 and novel features in transcriptional networks. It is important to emphasize that the yeast-based results indicate the potential for transactivation and that many factors can come into play in p53 mediated transactivation in human cells. Because of the wide range of p53 expression, the present study also provides greater opportunity to identify change-of-spectrum mutants, as well as subtle changes in transactivation.  While beyond the scope of the present study, this system may also be used to address the biological activity of the multiple p53 isoforms from specific response elements. Given that p53 isoforms are known to be differentially expressed in breast tumors in comparison to normal breast tissue (76), the ratios at which these isoforms are expressed, which can change with the presence of a mutation, may alter the transactivation profiles of wild type and/or mutant p53. For example, similar to the p53 family members p63 and p73, specific p53 isoforms can influence promoter selection or functionality from specific response elements through either synergistic or antagonistic mechanisms (76, 77).  Results obtained with the yeast functional assay appear predictive of whether a mutation will also have a biological impact in mammalian cells (16). While the yeast-based assay can predict the potential for specific p53 mutations to display altered function, assays in mammalian cells can ascertain the full impact of the functional mutations in the presence of p53 transcriptional cofactors and in the endogenous chromatin context of the RE. However, we propose that assessments of functional fingerprints for p53 missense mutations associated with breast cancer in yeast provide diagnostic value and with further study may also be used as a prognostic tool for implementing chemotherapeutic treatment. This would be particularly relevant to the tailoring of individual therapies, especially when the treatment agents impact p53-dependent biological responses. Furthermore, the technique of functional fingerprinting will be useful in determining if p53 dysfunction as a transcription factor can also be corrected by agents that reverse its structural stability, such as the carbazole derivative PhiKan083 acting on Y220C (78).  Mutational hotspots indicate selective pressure across a population of tumor samples, but their prevalence within and across cancer types is incompletely characterized. An approach to detect significantly mutated residues, rather than methods that identify recurrently mutated genes, may uncover new biologically and therapeutically relevant driver mutations. Here we developed a statistical algorithm to identify recurrently mutated residues in tumour samples. We applied the algorithm to 11,119 human tumors, spanning 41 cancer types, and identified 470 hotspot somatic substitutions in 275 genes. We find that half of all human tumors possess one or more mutational hotspots with widespread lineage-, position-, and mutant allele-specific differences, many of which are likely functional. In total, 243 hotspots were novel and appeared to affect a broad spectrum of molecular function, including hotspots at paralogous residues of Ras-related small GTPases RAC1 and RRAS2. Redefining hotspots at mutant amino acid resolution will help elucidate the allele-specific differences in their function and could have important therapeutic implications.  Among the best-studied therapeutic targets in human cancers are proteins encoded by genes with tumor-specific mutational hotspots, such as KRAS, NRAS, BRAF, KIT, and EGFR. The acquisition of somatic mutations is one of the major mechanisms responsible for the dysregulation of proliferation, invasion, and apoptosis that is required for oncogenesis. Comprehensive genomic characterization of tumors has produced significant insights into the somatic aberrations that define individual cancer types1,2, broadening our understanding of the dysfunctional molecular pathways that govern tumor initiation, progression, and maintenance. These data have spurred the development of computational algorithms to identify cancer driver genes, defined as those in which molecular abnormalities lead to a fitness advantage for the affected cancer cells (Methods).  These computational approaches develop either gene-level statistical models that exploit different mutational patterns3â€“6 to identify significantly mutated genes or use weight-of-evidence-based methods1,7 that are heuristic and ratiometric in approach. Together, these methods focus on identifying cancer genes from a multitude of diverse molecular abnormalities affecting the gene. However, not all genomic alterations in cancer genes are driver alterations. Furthermore, not all driver alterations in a cancer gene have the same functional impact, and are therefore likely to have varying clinical significance. The potentially diverse functional effects of different lesions in the same gene are not captured and reported by gene-level models, but are rather assumed to be equivalent. However, emerging data indicate that different hotspot mutations in the same cancer gene can be functionally distinct in vitro and in vivo and display different clinical phenotypes and drug sensitivity8â€“11. Moreover, it is unknown how widespread such hotspot-specific functional differences may be.  To date, studies of hotspot mutations in cancer have been limited to within individual tumor types12â€“14 or have focused on individual cancer genes across tumor types15. A systematic population-scale, cross-cancer, genome-wide analysis of mutational hotspots has not been performed and the extent to which mutant allele and lineage-specific effects exist remains unknown. As broad-based clinical sequencing has begun to inform the care of individual cancer patients, this would begin to address one of the greatest challenges in the practice of genomically driven cancer medicine: interpreting the biological and clinical significance of mutations in even presumed actionable cancer genes as they arise in oncology clinics.  To address this challenge, we develop a computational algorithm to identify driver mutations, rather than driver genes. We assembled and rigorously curated a large repository of cancer genome data consisting of the sequenced tumor exomes and whole genomes of 11,119 human tumors representing 41 tumor types. We developed a biologically aware, statistically principled computational model by combining observed biological phenomena such as nucleotide mutability and varying gene-specific mutation rates into coefficients that we incorporate into binomial statistics. From this, we systematically identify individual recurrent mutations and associate these with related temporal and transcriptional data to investigate lineage-specific variation in mutations, and identify novel hotspots with likely clinical implications.  Go to: Results Landscape of hotspots mutations in primary human cancer  We collected the mutational data from the sequenced exomes and genomes of 11,119 human tumors in 41 tumor types (Supplementary Table 1). These originate from diverse sources including large international consortia and various published studies (Methods). This cohort represents a broad range of primary human malignancies with three or more tumor types in each of nine major organ systems (Fig. 1a). The repository consists of 2,007,694 somatic substitutions in protein-coding regions with a median of 57 mutations (25 and 125 mutations; 25th and 75th percentile respectively) per tumor-normal pair with significant variability in mutation rates among and between tumors and types4,16. In total, 19,223 human genes harbor at least one somatic mutation in this dataset.  Figure 1 Figure 1 Mutational data and hotspot detection Here, we define a mutational hotspot as an amino acid position in a protein-coding gene that is mutated (by substitutions) more frequently than would be expected in the absence of selection (Methods). In this analysis, we focus exclusively on individual substitutions rather than other somatic abnormalities such as translocations, amplifications, deletions, or epigenetic modifications. To identify mutational hotspots, including low-incidence mutations, we developed a binomial statistical model that incorporates several aspects of underlying mutational processes including nucleotide context mutability, gene-specific mutation rates, and major expected patterns of hotspot mutation emergence (Supplementary Fig. 1a and 2, Supplementary Code; Methods). As considerable variability exists in the methods and standards for mutation calling used by individual studies and centers, we also developed several evidence-based criteria for eliminating probable false positive hotspots (Methods, Supplementary Fig. 1b). In total, we identified 470 statistically significant hotspots (q-value < 0.01) affecting 275 protein-coding genes (Supplementary Tables 2 and 3). Overall, more than half of all hotspots were determined to be novel (Fig. 1b, Table 1, Supplementary Table 2, and Methods) and 54.8% of all tumors assessed here possessed one or more hotspot mutations.  Table 1 Table 1 Select new hotspots in cancer genes Most affected genes possessed only a single hotspot (Supplementary Fig. 3a). A subset of genes, however, possessed many hotspots of varying frequency. In total, 49 genes possessed two or more hotspots (Fig. 1c), with many of these also arising in the greatest number of tumor types (Supplementary Fig. 3b). TP53 R248 was the most disseminated hotspot, observed in 25 tumor types. Among a subset of even well characterized oncogenes, a pattern of both known and novel hotspots emerge (Fig. 1d). Moreover, the number of observed mutant amino acids at a given hotspot generally increases with its mutational frequency across tumors types (Supplementary Fig. 3c), though 35% (n=164) of hotspots mutate to only a single variant amino acid. In most genes, hotspots bear only a fraction of the total mutational burden across the gene, whereas in a subset of cancer genes, the dominant mutational hotspot constitutes the vast majority of mutations independent of total mutational burden (Fig. 1d and Supplementary Fig. 3d). Overall, we identified considerable variability in the patterns of mRNA expression of individual hotspots in even canonical oncogenes (Supplementary Fig. 4), indicating that levels of expression are often not correlated with the biologic significance of known activating mutations.  The patterns by which some hotspots emerge support new clinical paradigms for testing targeted agents. Some hotspots that dominate the mutational landscape in one or a few cancer types also arise as uncommon subsets of many others. For instance, IDH1 R132 is most common in low-grade gliomas, glioblastomas, acute myeloid leukemias (AMLs), and cutaneous melanomas; but it is also present in 1 to 6 tumors in each of 11 additional cancer types. AKT1 E17K arises in greatest numbers in breast cancer, but also in 1 to 3 tumors of 10 additional cancer types. The distribution of CREBBP R1446 mutations is qualitatively different. They were originally identified in relapsed acute lymphoblastic leukemias17, but in this cohort of mostly primary disease, we find that they arise in only a small minority (1â€“3; 0.17â€“1.7%) of many (11) cancer types. Such patterns reaffirm the value of basket study designs that test mutant-specific inhibitors in early phase clinical trials, where enrollment is based on specific mutations in patients instead of tissue of origin.  A lineage map of all hotspots in genes with at least one common hotspot (Fig. 2a and Supplementary Fig. 5) indicates most hotspots are defined more by the tissue types rather than the organ systems in which they arise. Of all hotspots, 81% arise in two or more tumor types, suggesting that many hotspot mutations may confer a growth advantage across diverse lineages. Indeed, of hotspots present in multiple tumor types, only 7.6% (n = 36) are confined to a single organ system (Table 2). Thus, hotspot mutations that arise in a single tumor type may reflect organ-specific growth advantages but they represent only a small minority of all hotspot mutations in cancer. Likewise, a subset of hotspots arises in a cell-type specific manner. Twenty-seven hotspots (5.7%) were more frequently mutated in tumors of a squamous cell lineage (Supplementary Fig. 6), the most significant of which were MAPK1 E322 and EP300 D1399 (q-value = 6Ã—10âˆ’13 and 1Ã—10âˆ’11 respectively, Ï‡2) and may potentially confer a squamous cell-type specific growth advantage.  Figure 2 Figure 2 Lineage landscape of hotspot mutations Table 2 Table 2 Organ system-specific hotspots Overall, the presence, type, and frequency of hotspots by tumor type vary widely (Fig. 2b). In some tumor types, a large proportion of tumors possess one or more hotspot mutations including a significant fraction of tumors with a hotspot in a candidate oncogene (Fig. 2b, top). Conversely, other tumor types never or rarely possess a tumor defined by a hotspot identified here. Some of these differences are certainly attributable to the fact that hotspots are only one of many possible driver genomic aberrations, including specific gene fusions or focal amplifications and deletions. These other aberrations may define tumors of a given type, but they are not mutually exclusive with hotspots in many cancers. Other differences could not, alone, be explained by the overall mutational burden in these tumor types. For instance, uterine carcinosarcomas and prostate cancers have a similar mutation rate while there is 3-fold greater frequency of hotspot-bearing tumors among the former. Likewise, while papillary thyroid and high-grade pontine gliomas have mutations rates similar to nasopharyngeal tumors and neuroblastomas, the former far more commonly bear hotspot mutations (Fig. 2b).  Unconventional hotspots  In addition to missense mutations, we identified a variety of unconventional hotspot mutations with varied impact. Among these were 13 splice site hotspots. For each of these hotspots, an associated transcript abnormality was identified from RNA sequencing of affected tumors (exon skipping, intron retention, in-frame deletions; Supplementary Fig. 7a), including two previously characterized in-frame activating mutations (MET D1010_splice and PIK3R1 M582_splice, both exon 14 skipping events). We also identified 70 hotspots in 34 genes for which a nonsense mutation was among a diversity of changes at the affected residue, including 28 hotspots in which only a nonsense mutation was present (Supplementary Fig. 7b). While nonsense mutations scattered throughout a gene may reflect a pattern of loss-of-function consistent with tumor suppressor activity, a nonsense hotspot would appear to indicate the selection for the selective truncation of specific functional domains. Such events are consistent with the loss of some functions and the retention of others, as has been observed previously in genes such as PIK3R1, NOTCH1, and MET18,19. These hotspots aside, there was a depletion of nonsense mutations in hotspots in constitutively essential genes (p-value<10âˆ’16, those genes predicted or experimentally verified to be essential across all cell and tissue types and developmental states20). Otherwise, the specific impact of nonsense hotspots is generally unknown and belies the disseminated pattern of truncating mutations in likely or proven tumor suppressors (Supplementary Fig. 7c).  Lineage diversity and mutant allele-specificity  The majority of hotspot mutations arose in diverse tumor types and organ systems, yet widespread differences exist among individual residues and mutant amino acids in hotspots, genes, and tumor types (Fig. 3a). Examining the spectrum of KRAS mutations, which includes the most frequently mutated hotspot overall in our study (KRAS G12; n=736 mutant tumors, Fig. 1d and and2a),2a), clarified patterns only incidentally observed in the past. We found that gastric cancers were more similar to multiple myeloma in the preponderance of non-G12 mutations compared to endometrial, lung, colorectal, and pancreas tumors (p-value = 5.3Ã—10âˆ’18, Supplementary Table 4). Only colorectal tumors had KRAS A146 mutations whereas pancreas tumors lacked G13 mutations (p-values = 4Ã—10âˆ’7 and 2.8Ã—10âˆ’15 respectively). Many of these lineage-specific patterns were present at finer resolution as well. Among KRAS G12 mutations, the abundance of G12C mutations are highest in lung adenocarcinomas (p-value = 4Ã—10âˆ’42), an event that may be associated with prognostic differences compared with non-G12C KRAS mutations21â€“23. Such mutant amino acid specificity was also apparent in pancreas tumors, where KRAS G12R was more common than in any other tumor type (21% versus between 0 and 2.6%; Ï‡2 p-value = 4.8Ã—10âˆ’19). Gastric cancers, on the other hand, had the fewest G12V mutations among all KRAS G12-mutant tumor types, but the highest proportion of G12S (p-value = 0.007, Fig. 3b). There is a different balance among hotspots in the other Ras genes. While papillary thyroid cancers nearly exclusively possessed codon Q61 mutations in HRAS and NRAS (p-value = 4Ã—10âˆ’7), there was a higher prevalence of G12 and 13 codon mutations in these genes in AMLs, colorectal, bladder, and head and neck cancers, which together share few mutational processes in common (p-value = 4Ã—10âˆ’10, Fig. 3a).  Figure 3 Figure 3 Lineage diversity and mutant allele specificity Similar differences emerged in other driver cancer genes with multiple hotspots. V600E mutations describe nearly all BRAF hotspot mutations in melanoma, papillary thyroid, and colorectal carcinomas, whereas multiple myelomas are similar to lung adenocarcinoma in which non-V600E hotspots predominate (p-value = 1.9Ã—-10âˆ’32). The balance between extracellular and kinase domain mutations in EGFR between brain tumors and lung adenocarcinoma (p-value = 3.3Ã—10âˆ’12) respectively have been documented previously and affect their biological impact and the efficacy of genotype-directed therapy10. ERBB2 followed a similar pattern, where extracellular domain mutations typified by S310F are far more common than are kinase domain mutations in bladder cancers compared to breast cancers (p-value = 0.006, Fig. 3a). Another notable gene was PIK3CA. While bladder and cervical cancers are similar in their distribution of PIK3CA hotspot mutations, they vary significantly from breast cancers in the overall balance of helical to kinase domain mutations, possessing far fewer H1047R mutations among PIK3CA-mutated cases (p-value = 4.8Ã—10âˆ’19). Endometrial and colorectal cancers also have a similar pattern of PIK3CA hotspots, but both have a higher prevalence of R88Q mutations than any other tumor type (p-value = 1.3Ã—10âˆ’11, Fig. 3a). Such patterns extend beyond essential MAPK or PI3K signaling components, such as with SF3B1 K700 mutations that predominate in breast cancers and chronic lymphocytic leukemias whereas melanomas more frequently possess R625 mutations (p-value = 0.0001). Finally, mutant amino acid specificity was not limited to hotspots in Ras genes. The IDH1 R132H hotspot mutation predominated in multiple brain tumor types, but cysteine was the most common IDH1 R132 mutant amino acid in melanoma, which is unlikely to be exclusively related to UV light exposure, as this is also true in AMLs that lack a UV-driven etiology (p-value = 3.9Ã—10âˆ’21). Together, these results indicate that substantial mutant amino acid specificity exists among hotspot mutations across highly diverse tumor lineages. Two related conclusions may be drawn from these data. First, different hotspots in the same gene may possess in many cases different function, much of which may be lineage-dependent, while not excluding the possibility that some may still arise as a function of differing underlying mutational mechanisms. Second, that perhaps different mutant amino acids within the same hotspot can be functionally different, support for which is growing8,11.  Timing of individual hotspots  We next sought to determine if hotspot mutations, many of which are likely driver mutations and in some cases may serve as the initiating lesion, typically arise earlier than do non-recurrent mutations in the same genes and are therefore more often clonal. Overall, mutations at hotspot residues more often resided in a greater fraction of tumor cells (see Methods) and were therefore earlier arising (presumptive clonal), than were non-hotspot mutations in the same genes (Fig. 3c). So, while prior work has shown that driver genes in lung adenocarcinomas were enriched for clonal mutations24, we found that this was true of hotspot mutations across a broad class of cancer genes and tumor types. However, there was considerable variability among hotspots. While colorectal and endometrial cancers have a similar pattern of PIK3CA hotspot mutations (Fig. 3a) and share hypermutated subtypes of tumors driven by MSI and POLE exonuclease domain mutations25,26, colorectal tumors were unique in the clonality of the E545 and H1047 mutations. The majority of PIK3CA E545 helical domain mutations in colorectal cancers were subclonal, whereas H1047 kinase domain mutations were clonal, a difference that was not apparent in endometrial tumors, in which both are early clonal mutations (Fig. 3d). This may be a function of the pattern of oncogenic co-mutation in these tumors as PIK3CA E545, but not H1047, mutations were significantly associated with KRAS mutations in these colorectal cancers (Ï‡2 p-value = 0.0004) and in previous cohorts27. Overall, these differences in the molecular timing of specific hotspots augurs potentially important differences in their function in tumor initiation versus progression that requires further study.  Population-level hotspots in the long tail  Consistent with the so-called long tail of the frequency distribution of somatically mutated genes across cancer2, we found that 85% of all hotspots identified here were mutated in less than 5% of tumors of all cancer types in which they were found (Fig. 4a). Such findings have led to calls for sequencing up to many thousands of additional specimens from every tumor type28. However, many hotspots present at low frequency across cancers are not mutated commonly or significantly in even a single cancer type. Indeed, 23% of all hotspots identified here were present in only one or two samples in the tumor types in which it was observed. This included 19 hotspots arising in only one sample of each affected cancer type such as U2AF1 I24, MYC T58, the hyperactivating MTOR I250029, PIK3CB D1067, EP300 H1451, and ERBB3 M60. Therefore, many driver mutations (rather than genes) may never be found mutated at even the minimal frequencies (2â€“3%) proposed by previous studies as a goal in each cancer type. Conversely, population-level analysis, rather than by individual cancer type or organ system, allows identification of hotspots that arise as even private mutations in rare malignancies, for which additional broad-scale sequencing is most challenging. While rare, such recurrent alleles are evidence of selection and may be associated with specific phenotypes, such as exceptional responses30,31, de novo resistance to cancer therapy, or reveal specific facets of pathway biology. Consequently, we found that notable long-tail hotspots affect a broad spectrum of abnormal molecular function including macromolecular transport and transcriptional regulation (Table 1, Supplementary Note, and Supplementary Fig. 8), as well as essential components of key signaling pathways.  Figure 4 Figure 4 Candidate Ras-related small GTPase driver mutations in the long tail Long-tail hotspots in Ras superfamily members  Mutations in the Ras family of small GTPases occur widely in human cancers. As expected, these were among the most significant hotspots detected here (Supplementary Table 2), affecting 1,335 tumors (12% of all cases). Whereas G12, G13, and Q61 codon hotspots predominate in KRAS, NRAS, and HRAS, albeit at varying frequencies in different tumor types (Fig. 2a and and3a),3a), we also identified GQ60GK, K117, and A146 hotspots in KRAS. Both K117 and A146 are known activating hotspots in the long tail, but we also identified a previously occult GQ60GK dinucleotide substitution (q-value = 2.3Ã—10âˆ’6) in 11 tumors. This dinucleotide substitution results in a Q61K mutation accompanied by a G60 synonymous mutation that are present in cis (in concomitant RNA sequencing, Supplementary Fig. 9). Although Q>K mutations at codon 61 can result from 3â€™G>T single-nucleotide mutations in KRAS, 100% of these tumors harbored the dinucleotide substitution, a rare spontaneous event in human genomes. Overall, the distribution of codon 61 mutations in KRAS, NRAS, and HRAS are very different, with Q>K mutations occurring significantly less frequently in KRAS (p-value=0.016; Fig. 4b). GA>TT mutations were the most common dinucleotide substitution producing GQ60GK (Fig. 4c) and converts the ACC codon at KRAS G60 to TCC, which is the sequence of the G60 codon in NRAS, in which Q61K mutations are far more common and arise nearly exclusively from single-nucleotide mutations. It remains to be determined whether KRAS GQ60GK is therefore driven by a pattern of codon usage at the âˆ’1 position. Notably, only one tumor had evidence of a non-KRAS GQ60GK mutation, an NRAS-mutant cutaneous melanoma (Fig. 4c and Supplementary Table 5).  We next explored whether KRAS GQ60GK may serve as a driver of Ras pathway activity as do conventional KRAS hotspots. GQ60GK is indeed present in diverse tumor types that all have well-established Ras-driven subsets (Supplementary Table 5). Reasoning that if GQ60GK were a passenger mutation in Ras-driven tumors, alternative MAPK activating mutations may be present in these tumors. Instead, we found that in every GQ60GK-mutant sample where another putative driver of MAPK signaling was present, that lesion was either 1) subclonal, defining a different clone than did GQ60GK; 2) low activity; or 3) a passenger mutation (Supplementary Table 5). Also, despite the frequency of GA>TT, there was no evidence that a common underlying mutational process or exogenous mutagen was the source of GQ60GK. There was no evidence of UV light exposure in the clinical histories or nucleotide contexts of most affected cases, only one of which was a cutaneous melanoma. Moreover, GQ60GK arose in both hypermutated (MSI-H colon lacking BRAF V600E) and non-hypermutated tumors. Finally, rare G60 missense mutations were evident in K- and HRAS in this dataset and in the literature (Supplementary Table 5)32. So, while we cannot exclude the possibility that the GQ60GK dinucleotide substitution is simply an alternative mechanism to achieve Q61K, the accompanying KRAS-specific G60 synonymous mutation may potentiate a different class of Q61-mutant tumors or cause signaling differences among Q61K-mutant tumors between K-N- or HRAS. Although further studies will need to explore the molecular properties of KRAS GQ60GK, this allele represents the most common dinucleotide substitution spanning two codons in human cancer and a mutation more common than other known hotspots in KRAS.  Novel long-tail hotspots were also identified in two other genes that encode members of the Ras superfamily of small GTPases. RAC1, in which we identified two hotspots, is a Rho subfamily member that plays a vital role in various cellular functions. RAC1 P29S is an oncogenic hotspot in melanomas12,33 that we also identified in head and neck and endometrial cancers (Fig. 4d). This mutation can confer resistance to RAF inhibitor treatment in vitro34, and may underlie early resistance in patients35. We also identified a novel RAC1 A159V hotspot present in 10 tumors (q-value = 2.27Ã—10âˆ’6; Fig. 4d). Notably, RAC1 A159V is paralogous to KRAS A146, a known activating mutation36. Whereas activating KRAS A146T mutations arise predominantly in colorectal carcinomas (Supplementary Table 2), RAC1 A159V mutations are most common in head and neck cancers and were not present in any melanomas, despite the frequency of RAC1 P29S in this cancer type. Moreover, similar to P29S mutations, we observed RAC1 A159V mutations in tumors that are both Ras/Raf wildtype and mutant (Fig. 4e). To determine whether RAC1 A159V is an activating mutation, we assessed its effect in vitro. Active RAC1 is GTP-bound, interacting with PAK1 to activate downstream effectors. Therefore, to quantify RAC1 activation in vitro, we utilized a PAK1 pull-down assay. In HEK293T cells expressing RAC1 A159V, there was significant RAC1 activation to levels equal to or exceeding positive control RAC1 GTPÎ³S cells and greater than even those levels induced by the known RAC1 P29S oncogenic mutation (Fig. 4f). Moreover, cells expressing RAC1 Q61R, a mutation we identified in a primary prostate cancer that is paralogous to KRAS Q61, also potently induced RAC1 activation (Fig. 4d,f).  RRAS2 is a Ras-related small GTPase37. RRAS2 is overexpressed or mutated in a small number of cancer cell lines of various origins38â€“40, and is oncogenic in vitro with transforming ability similar to established Ras oncoproteins41. However, it has not been documented as somatically mutated in human tumor specimens. Here, we identified a RRAS2 Q72 hotspot present in nine tumors (q-value = 8Ã—10âˆ’15). Similar to RAC1 A159V, the RRAS2 Q72 hotspot is paralogous to KRAS Q61 (Fig. 4d). However, unlike RAC1, RRAS2 Q72 does not predominate in any individual tumor type. Also unlike RAC1, the RRAS2 Q72 mutation was present in Ras/Raf wildtype tumors among the affected types (Fig. 4e). This result suggests that RRAS2 activation may be an alternative avenue for tumors to acquire Ras-like activation as previous studies have shown that RRAS2 shares many Ras downstream signaling elements including phosphatidylinositol-3 kinase (PI3K)42,43, the Ral GDP dissociation pathway42, and Raf kinases44. Beyond these hotspots, several less common RAC1 and RRAS2 mutations affect paralogous residues of highly recurrent alleles in KRAS (Fig. 4d), some of which we validated were also activating in vitro (Fig. 4f), indicating that the landscape of potentially functional mutations in these genes extends beyond even these less common long-tail hotspots to private mutations as well.  Go to: Discussion Although we focus only on recurrent substitutions, we did find that while a subset of hotspots were prevalent in individual cancer types, most hotspots are present infrequently across many cancer types. This indicates that studies of any individual cancer type may have limited power to identify novel alleles. We have also begun to detail best practices for the use of diverse public cancer sequencing data in the translational setting. Our approach for hotspot detection incorporates features such as the variable background mutational burden of individual codons and genes, thereby avoiding passenger mutations whose recurrence is due only to their presence in highly mutable amino acids. While the identification of private driver mutations remains challenging, our approach did uncover low-incidence hotspots in highly mutated genes. Though less common, these hotspots are under selection and may confer important clinical phenotypes in cancer patients, such as exceptional responses to cancer therapy30,31.  New mutant alleles in established genes are likely to emerge faster than new cancer genes are identified, extending the long tail of the frequency distribution of somatic mutations. This is especially true as clinical sequencing focuses on profiling advanced and metastatic disease for clinical trial enrollment. Such pre-treated, late-stage cases have been historically under-represented among such population-scale resources, including the one studied here. Moreover, at present there are fewer actionable mutations in cancer than there are cancer genes. Yet the near-term clinical utility of expanding the former is far greater. Our results suggest this will require an understanding of the function of different hotspot mutants in the same gene by lineage, as their function and response to therapy may be mutant amino acid specific. While positive selective pressure may produce the same hotspot mutation, or different variant amino acid changes within the same hotspot residue, it does not imply that they will confer similar selective advantages across lineages. Underlying functional distinctions may explain the differences observed here in the emergence and frequencies of hotspots across lineages. While this remains speculative or unknown for most hotspots, early evidence suggest that this will be true for even some of the most important alleles in human cancer8,9,11. Understanding this landscape of distinct molecular function is the necessary translational prerequisite for effective clinical implementation. This focus on mutations rather than genes will spur studies of the biochemical, biological, signaling impact, and drug sensitivity of candidate individual alleles. Collectively, the complementary study of both significantly mutated individual alleles as well as genes will prove indispensable in enabling precision oncology through clinical decision support for patients sequenced at the point of care.  Go to: Methods Mutational dataset and pre-processing  Mutational data were obtained from three publically available sources: 1) The Cancer Genome Atlas (TCGA); 2) the data portal of the International Cancer Genome Consortium (ICGC); 3) various published studies in peer-reviewed journals in which mutational data was made available45,46. Mutation calling algorithms and mutation reporting practices varied from study to study in these curated data, so mutation data review and correction were undertaken where possible. Genomic coordinates of variants from alignments to human reference assembly NCBI36 (hg18) were converted to GRCh37 using LiftOver47 with an Ensembl chain file (see URLs). After standardization to GRCh37, the mutation calls were annotated to gene transcripts in Ensembl release 75 (Gencode release 19), and a single canonical effect per mutation was reported using Variant Effect Predictor (VEP) version 7748 and vcf2maf version 1.5. All possible pairs of any two samples with at least 10 somatic mutations were interrogated for sample duplication. For any pair of tumors that shared greater than 80% mutational identity and identical or near-identical clinic-pathological characteristics (upon review of data from the source site/publication), a single tumor in the pair was chosen at random and removed from further analysis as a presumptive duplicate specimen. Furthermore, we excluded small insertions and deletions (indels), despite their presence as true oncogenic hotspots in some genes, due to their greater variability in call quality across datasets. In total, the final dataset included 1,348,424 missense; 524,827 synonymous; 100,866 nonsense; 30,346 splice-site; and 3231 mutations affecting translational start or stop codons. There are also 21,130 oligo-nucleotide variants the majority of which are di-nucleotide mutations along with 71 tri-nucleotide mutations and 13 substitutions of 4bp or longer. Individual mutations and hotspots of interest (detected as described below) were inspected in individual BAM files from tumor and matched normal specimens of DNA and available RNA sequencing data downloaded from CGHub. When available, expression analyses were based on level-3 RNASeqV2 RSEM normalized gene expression counts from RNA sequencing available via the TCGA Data Coordinating Center. These values were log-transformed and scaled across all samples within each cancer type to facilitate comparisons between cancer types.  Definition of a mutational hotspot  For the purposes of this analysis, we first define a driver cancer gene as one in which a molecular abnormality leads to a fitness advantage for the affected cancer cell. This is the broadest definition that encompasses both initiating lesions on which tumor growth depends as well as lesions arising later in tumor progression that perhaps confer a more modest fitness advantage. We then define a hotspot as an amino acid position in protein-coding gene mutated more frequently than would be expected in the absence of selection. Therefore, all of the following mutation types result in the same hotspot: 1) mutations in different nucleotide positions in the same codon of a gene, 2) different nucleotide substitutions at the same site in the same codon that result in different amino acid changes, and 3) mutations where the amino acid substitution is identical but the nucleotide change are different. At present, this analysis is limited to recurrent somatic substitutions, but can be expanded to other classes of somatic alterations such as small insertions and deletions, DNA copy number alterations, and structural rearrangements.  Determining significant mutational hotspots  To determine the statistical significance of individual mutational hotspots, we developed a truncated binomial probability model by incorporating not only underlying features of mutation rates in cancer but also anticipating the gene-specific pattern with which hotspots may arise in different classes of possible cancer genes. In its most general form, if X represents the count of mutations in n samples, the probability of observing k mutations is:  Pr(X=k)=(nk)pk(1âˆ’p)nâˆ’k (1) where p is the probability of a mutation in any sample. However, differences exist in the mutability of specific nucleotide contexts in cancer genomes. These vary as a function of the underlying mutational process, potential molecular abnormality in normal DNA maintenance pathways, and possible exposure to exogenous mutagens16. Moreover, individual genes have highly variably nucleotide composition and background mutation rates. To address these fundamental characteristics, we integrated a coefficient into a position-specific probability that incorporates both the mutability of the trinucleotide context in which the mutation arose and the trinucleotide composition of the affected gene. For each of the 32 possible trinucleotides, we estimate the mutability of a given trinucleotide t as:  mt=CtFt (2) where Ct is the number of mutations affecting the central position of trinucleotide t across all samples and Ft is the number of occurrences of the trinucleotide t in the coding genome. Too little data existed to compute tumor type- or underlying mutational process-specific mutability. Because a mutated codon in a given gene is comprised of mutations in any one of three trinucleotides that encode that codon, we estimate the mutability of a codon c in gene g as:  mc,g=âˆ‘tâˆˆcmtnt,cnc (3) where nt,c is the number of mutations in the central position of trinucleotide t in codon c and nc is the number of mutations in codon c overall. We estimate the mutability of gene g as Î¼g = Cg/(nLg), where Cg is the number of mutations affecting the gene across the n samples and Lg is the length of the gene in amino acids. We then estimated the expected mutability of a given gene g as:  mg=âˆ‘tNt,gmtLG (4) where Nt,g is the number of occurrences of trinucleotide t in gene g. The relative mutability of a codon within a gene is then rc,g = mc,g/mg. This leads to a binomial parameter for hotspot detection of:  pc,g=rc,gÎ¼g (5) We sought to avoid overestimating the background mutation rate for a gene with several hotspots. This would limit the detection of lower frequency hotspots (warmspots) due to the rate of recurrence of one or a few dominant hotspots in the same gene. We therefore developed a truncated form by removing positions in gene g bearing greater than or equal to the 99th percentile of all mutations in the gene. The new background rate is therefore Î¼â€²g, calculated as before where the prime signifies the mutation counts and lengths modified using the above threshold. Then pâ€²c,g=rc,gÎ¼â€²g. Finally, in rarely mutated genes where the probability p is exceedingly small (relative to the size of the cohort N and the length of the protein L), we limited the number of false positive hotspots by allowing pâ€²c,g to get no smaller than the 20th percentile of all pâ€² dataset-wide. Therefore, the final binomial probability is:  pâ€²â€²c,g=max{pâ€²c,g20%ile of allp' (6) Accordingly, we calculate one-sided p-values for all unique amino acids in every annotated gene per the binomial form given in eq. (1) with probability from eq. (6) and test whether more mutations are observed than would be expected by chance given the pattern of all mutations in the gene; its composition and length; the pattern of its mutability; and the number and type of samples assessed.  Multiple hypothesis correction  To correct for multiple hypotheses, we employed a method for false discovery rate correction that assumes dependence among tests. This correction was performed on the gene level in the following manner. P-values were aggregated per gene on the basis of their codon position. For codons that were not mutated in a given gene and therefore not formally assessed, we padded this with a vector of p-values equal to 1 such that the final set of p-values equaled the amino acid length of the given gene. For all resulting p-values in each gene, they were corrected with the Benjamini and Yekutieli49 method (implemented in p.adjust in the stats package in R) and significant hotspots were those sites with q-values < 0.01.  Mutation call quality filtering  Considerable variability exists in the processing and generation of mutational data in individual cohorts by originating centers. To address this variability, we developed several weight-of-evidence based criteria for eliminating presumptive false positives and sequencing artifacts from individual mutation calls as well as from hotspots across the dataset (Supplementary Fig. 1b). Initially, to exclude likely germline variants misattributed as somatic mutations we exclude any mutation identified by both 1000genomes and the NHLBI or those identified only by 1000genomes in two or more samples. We then reasoned that hotspots arising in genes not expressed in a given tumor type are less likely to exert biological impact. We therefore removed from consideration hotspot mutations in genes whose expression was <0.1 transcripts per million (TPM) in 90% or more of the tumors of that type, or for tumors that lacked RNA sequencing data, if more than 95% of all tumors independent of organ of origin had expression of TPM < 0.1. After determining statistically significant hotspots (described above), hotspots were removed from consideration based on a decision tree model as follows. First, a presumptive true positive (pTPs) list of hotspots was predetermined as coding positions harboring substitutions in five or more tumor samples (from the August 2013 release of the cBioPortal45,46) in one of 341 key cancer-associated genes sequenced as part of routine CLIA-certified sequencing of matched tumor and normal specimens at Memorial Sloan Kettering Cancer Center50. Initially, for all samples in which a hotspot was observed and for which the fraction of tumor cells mutated could be calculated from corresponding variant allele frequency and DNA copy number data, we calculated the fraction of tumors in which that site was mutated subclonally (in fewer than 90% of tumor cells). If the fraction of samples in which the hotspot arose subclonally exceeded the maximum such value among pTPs, it was excluded. For remaining sites, we excluded potential hotspots that arose from mutation calling bias from a single source center. We identified cohorts in which subsets of samples were called by different centers and excluded hotspots in which greater than 85% of contributing mutation calls originated from a single mutation-calling center. Next, as local sequence complexity can affect alignment accuracy in various ways based on the read lengths and chemistry of source studies in our dataset, we sought to exclude hotspots on the basis of sequence context. We excluded hotspots where the minimum of Shannon entropy calculated from both 12bp or 24bp of flanking sequence on either the 5â€™ or 3â€™ side of the mutated site was less than the minimum such value among pTPs. We then excluded hotspots that were positioned at either the 5â€™ or 3â€™ end of a mono-, di-, or tri-nucleotide homopolymer runs of 10bp or longer. Remaining hotspots were then excluded if either the sum of their ranked weighted 100 and 24bp alignability (determined by CRG Alignability; UCSC Genome Browser) was less than the minimum value of pTPs or their weighted 24bp alignability was lower than the 12.5 percentile of all sites. We also excluded any hotspot that while passing these criteria affected a gene that was 1) already rich in presumptive false positives by these criteria [the number of retained hotspots was less than two times the count of hotspots in the gene excluded by one or more of these criteria] or 2) one of 20 well-characterized presumptive â€œred-herringâ€� cancer genes due to high mutation rates that co-vary with underlying features independent of selection4. Finally, we manually inspected the sequencing data contributing to the mutation call for select hotspots in a sampling of affected tumor and matched normal samples. The significant hotspots (q-value < 0.01) that were excluded from consideration on the basis of this model are available in Supplementary Table 3.  Hotspot mutation data and literature review  In addition to the mutation call filtering described above, we independently validated the accuracy of a subset of mutation calls contributing to the novel hotspots discussed in the text. We downloaded BAM files of the aligned and unaligned sequencing reads corresponding to the tumor and matched normal exomes from each patient harboring one of the novel hotspots we discuss in detail in the manuscript (RAC1 A159; RRAS2 Q72; NUP93 E14 and Q15; MAX R60; and MAX H28) and reprocessed these from raw FASTQ to mutation calls with an independent sequence analysis pipeline30,31. We re-identified the hotspot mutation of interest in 34 of 35 affected specimens. Only a single sample failed to reproduce the published mutation (NUP93 E14 in a hepatocellular carcinoma), but NUP93 E14K remains statistically significant even after excluding the tumor in which the mutation was not called by the independent pipeline. To determine novel hotspots among the 470 identified here, we classified each mutation into one of three levels of evidence from an extensive literature review (Supplementary Table 2). Level-1 mutations are those not previously identified in human tumors or have been identified in an individual sample, but never described as a hotspot of recurrent mutation. This also includes mutations previously documented in the germline of patients, but never identified previously as recurrently somatically mutated. Level-2 hotspots are those mutations that have been reported previously in one tumor type that we also identified in the same, but also find mutated in one or more additional tumor types not previously described. Level-3 hotspots are those mutations that have been previously identified in one or more tumor types and have been assessed functionally in vitro or in vivo. Using this categorization, we identified 243, 46, and 181 levels-1, -2, and -3 mutations from the 470 hotspots described here. These were classified as being present or not in established candidate cancer genes (n=880, Fig. 1b) compiled from the Sanger Cancer Gene Census and four additional published sources1,4,5,50.  Cell type specificity analysis  We determined the enrichment of individual hotspots in different cell types (squamous versus non-squamous). For this analysis, squamous cell cancers included head and neck squamous cell carcinoma, lung squamous cell carcinoma, cervical squamous cell carcinoma, and esophageal squamous cell carcinoma. Hotspots mutated in at least one squamous tumor were examined and their statistical significance was assed with Fisherâ€™s exact test assuming the null hypothesis that squamous and non-squamous tumor samples are equally likely to possess a given hotspot. Resulting p-values were corrected for multiple hypothesis testing with Benjamini and Yekutieli49 method and squamous cell-type specific hotspots were those with q-values < 0.01.  Analysis of the fraction of cancer cells mutated  Level-3 segmentation of DNA copy number data was acquired when available without restriction for 15 cohorts (primarily TCGA). Utilizing this data, we estimated tumor purity using absCN-seq51, due to the ease of its automation. We calculated the fraction of tumor cells bearing each mutation (tumor/cancer cell fraction) in each tumor based on this purity estimate, local copy number, and mutant and reference allele sequencing coverage all as previously described52.  Functional validation  DNA coding sequences for wildtype RAC1 as well as RAC1P29S, RAC1Q61R, and RAC1A159V were generated via site-directed mutagenesis (Genewiz, NJ) to include an N-terminal 3xFLAG epitope tag and were subcloned into a pcDNA3 mammalian expression vector (Life Technologies, NY). The expression constructs were transfected into HEK293T cells using Lipofectamine 2000 (Life Technologies), and cells were harvested after 72 hours. GTP-bound Rac1 (active Rac1) was isolated via immunoprecipitation using recombinant p21-binding domain (PBD) of PAK1 (PAK1-PBD; Active Rac1 Detection Kit, Cat#8815, Cell Signaling, MA), according to the manufacturer's instructions. The Rac1 was detected using kit provided Rac1 primary antibody.\""
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['Text'][414]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "\" Abstract Background  Non-small cell lung cancer (NSCLC) is a heterogeneous group of disorders with a number of genetic and proteomic alterations. c-CBL is an E3 ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer. We determined the genetic variations of c-CBL, relationship to receptor tyrosine kinases (EGFR and MET), and functionality in NSCLC.  Methods and Findings  Using archival formalin-fixed paraffin embedded (FFPE) extracted genomic DNA, we show that c-CBL mutations occur in somatic fashion for lung cancers. c-CBL mutations were not mutually exclusive of MET or EGFR mutations; however they were independent of p53 and KRAS mutations. In normal/tumor pairwise analysis, there was significant loss of heterozygosity (LOH) for the c-CBL locus (22%, nâ€Š=â€Š8/37) and none of these samples revealed any mutation in the remaining copy of c-CBL. The c-CBL LOH also positively correlated with EGFR and MET mutations observed in the same samples. Using select c-CBL somatic mutations such as S80N/H94Y, Q249E and W802* (obtained from Caucasian, Taiwanese and African-American samples, respectively) transfected in NSCLC cell lines, there was increased cell viability and cell motility.  Conclusions  Taking the overall mutation rate of c-CBL to be a combination as somatic missense mutation and LOH, it is clear that c-CBL is highly mutated in lung cancers and may play an essential role in lung tumorigenesis and metastasis.  Go to: Introduction In the US alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. This number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. In addition the prognosis is usually poor and the five-year survival rate is less than 15%. There are also significant ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. Gender differences are also striking with women having significantly better prognosis as compared to men. There are a number of genetic alterations that can occur in lung cancer. As an example, in NSCLC, mutations in KRAS, p53, EGFR and MET have been identified. Many of these pathways, especially Receptor Tyrosine Kinases (RTKs) are controlled by c-CBL.  CBL (Casitas B-lineage lymphoma) is a mammalian gene located on human chromosome 11q23.3 [2] and is involved in cell signaling and protein ubiquitination [3]. CBL proteins belong to the RING finger class of ubiquitin ligases (E3) and there are three homologues c-CBL, CBL-b, CBL-3 [4]. The c-CBL and CBL-b genes are ubiquitously expressed with the highest levels in hematopoietic tissues [5]. c-CBL consists of four regions encoding for functionally distinct protein domains: the N-terminal tyrosine kinase binding (TKB) domain, the linker region, the catalytic RING finger domain, the proline-rich region and the c-terminal ubiquitin-associated (UBA) domain that also overlaps with a leucine-zipper (LZ) domain [3]. Both TKB and RING finger domains are essential for ligand-induced ubiquitination of RTKs [6], [7], [8], [9]. The RING finger domain is required for the recruitment of E2 ubiquitin-conjugating enzymes. The TKB domain includes four-helix bundle (4H), a calcium-biding EF hand, and a modified SH2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. In addition, the proline-rich region of c-CBL can associate with the SH3 domain of Grb2, which can indirectly recruit c-CBL to RTKs via the GRB2 adaptor protein [7], [13], [14].  c-CBL also binds to EGFR and acts as the E3 that targets EGFR for ubiquitination and degradation. Furthermore, CBL desensitizes EGF signaling and opposes cellular proliferation induced by EGF [15]. EGF activation also appears to activate the tyrosine kinase SRC, which phosphorylates c-CBL and in turn activates the ubiquitination and degradation of EGFR [16], [17], [18]. A recent study shows that defective endocytosis of EGFR is characterized by a deletion mutant and the point mutation L858R, whereby its association with c-CBL and subsequent ubiquitination are impaired [19]. Recently, the first human c-CBL mutations were reported in acute myeloid leukemia (AML) patients [20]. The mutation R420Q inhibits FMS-like tyrosine kinase 3 (FLT3) internalization and ubiquitination [20].  Not only can E3 activity be important in oncogenesis, c-CBL has a dual but separate function as a signal transduction molecule. We have previously shown that c-CBL is important in binding CRKL and BCR/ABL in hematopoietic cells. Also, it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin. The TKB domain is important in binding to a number of molecules, and they then function in signal transduction.  Given the critical role of CBL in normal homeostasis and cancer, we hypothesized that it might be mutated in lung cancers. In this study, we report novel c-CBL somatic mutations S80N/H94Y, Q249E and W802* in Caucasian, Taiwanese and African-American lung cancer patients, respectively. Expressing these mutations in NSCLC cell lines lead to increased proliferation and cell motility. We show that c-CBL mutations occur with or without MET or EGFR mutations but are mutually exclusive of a LOH at the c-CBL locus. Additionally, c-CBL LOH is associated with either MET or EGFR mutations. We thus hypothesize that c-CBL mutations might contribute to the oncogenic potential of MET and EGFR in lung cancer.  Go to: Methods Ethics Statement  Written consent on all research on human subjects has been obtained from the Institutional Review Board, University of Chicago and covers all research performed in the laboratory. The following is their contact information: Institutional Review Board, The University of Chicago, McGiffert Hall, 5751 S. Woodlawn Ave., 2nd floor, Chicago, IL 60637. Written informed consents were received from all patients whose tissue samples were used for this study.  Tissue Samples  Lung cancer tissue and paired adjacent normal lung tissues were obtained from 50 Caucasian, 29 African-Americans and 40 Taiwanese NSCLC patients who were recruited at the University of Chicago Hospital (Chicago, USA) (Caucasian and African-American patients) and Taipei Veterans General Hospital of Taiwan (Taiwanese patients) after obtaining appropriate Institutional Review Board permission and informed consent from the patients. Out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. In terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage I, 14 were stage II, 34 were stage III, and 13 were stage IV (Table S1).  Cell Culture  Human non-small cell lung carcinoma cells A549 and H358 were maintained in DMEM and RPMI-1640, respectively. Human embryonic kidney 293T cells were cultured in DMEM. Media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 Âµg/ml of streptomycin (Invitrogen, Carlsbad, CA). Cells were cultured at 37Â°C in a humidified incubator containing 5% CO2.  c-CBL Gene Mutational Analysis  Exons 2 to 16 of c-CBL gene were individually amplified by polymerase chain reaction (PCR). Primers are listed in Table S2. PCR conditions were 1 cycle of 95Â°C for 5 minutes; 35 cycles of 94Â°C for 30 seconds, 58Â°C for 30 seconds and 72Â°C for 2 minutes; and one cycle of 72Â°C for 10 minutes. PCR products were treated with ExoSAP-IT (USB Corporation, Cleveland, OH) and sequenced by Big-Dye Terminator Chemistry (Applied Biosystems, Foster City, CA). Sequencing was performed on the forward coding strand with confirmation of c-CBL alterations performed by sequencing the reverse strand as well. Chromatograms were analyzed for mutations using Mutation Surveyor v2.61 (Softgenetics, State College, PA).  Plasmid Constructs and Site-Directed Mutagenesis  The wild-type c-CBL cDNA insert was subcloned into the pAlterMax expression vector using XhoI and SalI restriction enzyme sites (Promega, Madison, WI). Using this parental plasmid pAlterMax-c-CBL, the TKB domain double mutation (S80N/H94Y), the point mutation (Q249E), and the C-terminal point mutation W802* of c-CBL were created using the following primers: 5â€²-GCTGGCGCTAAAGAATAACCCACCTTATATCTTAGAC-3â€² and 5â€²-CTACCAGATACCTACCAGTATCTCCGTACTATCTTGTC-3â€² for the double mutation S80N/H94Y; 5â€²-CTTTACCCGACTCTTTGAGCCCTGGTCCTCTTTGC-3â€² for Q249E, and 5â€²-CAGCTCCTCCTTTGGCTGATTGTCTCTGGATGGTGATC-3â€² for W802* along with their complementary primers using the QuickChange Site-Directed Mutagenesis XL kit (Stratagene, La Jolla, CA) according to the manufacturer's instructions. The constructs were confirmed for the point mutations by standard DNA sequencing of both strands.  Loss of Heterozygosity (LOH) Analysis  Five microsatellites on chromosome 11 (3 on 11q at or within 200 kb up or downstream of the c-CBL gene and 2 control markers on 11p) were selected for analysis (Table S3). Established microsatellite markers and respective primer sequences were selected from the GeneLoc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, Weizmann Institute of Science, Rehovot, Israel). Primers were custom designed and each forward primer was fluorescently labeled at the 5â€² end with FAM, PET, NED, or VIC (Applied Biosystems). Primer annealing temperatures and duplex scores were evaluated with NIST Primer Tools (http://yellow.nist.gov:8444/dnaAnalysis/primerToolsPage.do; National Institute of Standards and Technology, Gaithersburg, MD). Primers were verified by performing PCR with control DNA (isolated from TK6 cells) and resolving the products on agarose gels. Bands were visualized with an UV transilluminator. Genomic DNA was extracted from tumor samples and paired normal lung tissue. Primers were grouped into multiplex combinations shown in Table S4. Marker D11S929 served as an internal control to check for consistency in PCRs and of peaks from capillary electrophoresis. Multiplex PCRs were carried out in a volume of 10 ÂµL that contained 1 ÂµL genomic DNA (20â€“50 ng), 0.5 ÂµM of each primer (1.0 ÂµM total for each primer pair), 400 ÂµM dNTPs, 1X PCR buffer containing MgCl2, and 0.2 U Taq DNA polymerase. PCR was performed on the ABI GeneAmp 9700 PCR System under the following conditions: 5 min at 94Â°C; 30 cycles of 30 sec at 94Â°C, 1 min at 60Â°C, 1 min at 72Â°C; and 5 min at 72Â°C. The PCR products were separated by capillary electrophoresis on an ABI 3130XL DNA Analyzer. Chromatograms were analyzed with Peak Scanner 1.0 and GeneMapper 3.7 software (Applied Biosystems) for allelic alterations. The area of the peaks produced by the DNA PCR products was quantified for each allele. The ratio of the allelic areas was calculated for each tumor and paired normal DNA sample. When the qLOH (allelic ratio for the tumor peaks divided by the allelic ratio of paired normal sample) was â‰¤0.5 or â‰¥2.0 for c-CBL and at least one other 11q marker in at least two separate experiments, the sample was considered as having an allelic imbalance and interpreted as LOH. Samples were evaluated in at least two separate experiments and samples showing prospective LOH at c-CBL repeated a third time which included a new control marker at the BAX locus (data not shown) on chromosome 19 to verify integrity of sample DNA.  Transfection of c-Cbl Constructs  The A549 cell line was transfected using the Fugene HD (Roche, Nutley, NJ) reagent according to the manufacturer's instructions. Eight Âµg of plasmid DNA, containing either no insert (empty vector), wild-type c-CBL, S80N/H94Y c-CBL, Q249E c-CBL or W802* CBL was used for transfection in a 6-well culture plate. Cells were harvested 48 h after transfection and analyzed for expression.  c-CBL Knockdown  c-CBL knockdown was performed using lentiviral transduction using MISSION lentiviral transduction particles (Sigma-Aldrich, St. Louis, MO) as per manufacturer's instructions. Briefly, 1Ã—105 H358 cells/well were seeded in 6-well plates and infected the following day with c-CBL lentiviral shRNA constructs. To generate stable c-CBL knockdown cell lines, cells were selected for 2 days with 1 Âµg/ml puromycin. c-CBL levels were determined using whole cell lysates by immunoblotting with anti-CBL antibody (Santa Cruz Biotechnologies, Santa Cruz, CA).  Cell Viability Assay  Cells were transfected as described above in the transfection assay. Forty-eight hours after transfection, viability of cells was assessed using Trypan Blue exclusion.  Wound Healing Assay  A549 cells were seeded in 6-well plates and cultured for 48 h until 100% confluent. The medium was then changed and the cells were transfected as described in the transfection assay. Twelve hours after transfection, a straight scratch was made across the cell layer using a 1 ml pipette tip. The cells were then gently washed with 1Ã— PBS to remove cellular debris and the media was replaced. Photographs were taken of the wound region every 12 h until 48 h.  Western Blot Analysis  Forty eight hours after transfection, cells were collected and washed twice in 1X PBS, then lysed in ice-cold lysis buffer (0.5M Tris-HCl with pH 7.4, 1.5 M NaCl, 2.5% deoxycholic acid, 10 mM EDTA, 10% NP-40, 0.5 mM DTT, 1 mM phenylmethylsulfonyl fluoride, 5 Âµg/mL leupeptin, and 10 Âµg/mL aprotinin) for 5 minutes. The lysate was centrifuged at 13,000 rpm for 20 minutes at 4Â°C, and protein content of the supernatant was measured. Total cell lysates (50 Âµg/well) were separated by SDS-PAGE electrophoresis and the gels transferred onto nitrocellulose membranes (Whatman, Piscataway, NJ). Membranes were blocked with 5% non-fat dry milk in phosphate-buffered saline containing Tween-20 (PBST) (1X PBS, 0.1% Tween-20) for 1 h at room temperature and incubated with the appropriate primary antibody at 4Â°C overnight. Membranes then were washed three times with PBST and probed with appropriate horseradish peroxidase (HRP)-conjugated secondary antibody for 1 h at room temperature. The membranes were again washed three times in PBST and bands were visualized using Western blot chemiluminescence reagent (BioRad, Valencia, CA) on a Chemidoc Gel documentation system (BioRad, Valencia, CA). Antibodies were obtained from Santa Cruz Biotechnologies and used at the following dilutions (c-CBL, 1âˆ¶5000; c-MET, 1âˆ¶5000; EGFR, 1âˆ¶5000; ubiquitin, 1âˆ¶1000; HA, 1âˆ¶5000 and Î²-actin, 1âˆ¶10,000).  Flow Cytometry  Cell cycle analysis was carried out by flow cytometry. Approximately 2Ã—106 cells were grown in media containing 10% FBS. Cells were harvested by trypsin/EDTA treatment, washed with 1X PBS three times and fixed with ice-cold 70% ethanol for 2 h. Cells were washed again with cold PBS and stained with a solution containing 25 Âµg/mL propidium iodide, 200 Âµg/mL RNase A, and 0.1% Triton X-100 for 30 minutes in the dark. Cell cycle analysis was performed using a Guava PCA-96 flow cytometer (Guava Technologies, Millipore, Billerica, MA).  Ubiquitin Ligase Activity  293T cells were maintained in culture in DMEM supplemented with 10% FBS and 1% penicillin (100 units/mL) and streptomycin (100 Âµg/mL) were transfected with 0.2 Âµg EGFR-pcDNA3 and 2 Âµg HA-tagged c-CBL constructs as indicated using calcium phosphate according to manufacturer's protocol (Profection, Promega, Madison, WI). Twenty-four hours post-transfection, cells were starved overnight in DMEM supplemented with 0.5% FBS, and then treated with or without EGF (100 ng/ml) for 15 min. The cells were collected and washed two times in ice-cold PBS containing 0.2 mM sodium orthovanadate then lysed in ice-cold lysis buffer (10 mM Tris HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X100, 10% Glycerol, 2 mM sodium orthovanadate and protease inhibitors). Lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4Â°C. EGFR immunoprecipitations were performed on 200 Âµg of cleared lysate using 250 ng of rabbit-anti-EGFR and Protein A/G Plus Sepharose overnight at 4Â°C. Precipitations were washed 5 times in lysis buffer before boiling in Laemmli buffer. Elutions were immunoblotted with anti-ubiquitin and EGFR. Twenty micrograms of cleared lysate were immunoblotted for each of the c-CBL constructs using anti-HA.  Statistical Analysis  Mutation rates between different groups were compared using Fisher's exact test. For continuous variables, group comparisons were performed using analysis of variance (ANOVA) followed by Sidak's adjustment for multiple comparisons. Experiments involving repeated measurements over time were analyzed using repeated measures ANOVA with the Greenhouse-Geisser adjustment to the degrees of freedom. Analyses were conducted using STATA (v10.1) software (Stata Corporation, College Station, TX).  Go to: Results c-CBL Gene Mutations in Lung Cancer  To investigate the role of c-CBL in lung cancer, we analyzed its genomic DNA in tumor and paired normal samples drawn from multiple ethnicities. The lung tumor samples represented Caucasians (nâ€Š=â€Š50), African-Americans (nâ€Š=â€Š29), and Taiwanese (nâ€Š=â€Š40) lung cancer patients. We designed 12 pairs of primers to sequence the coding region of c-CBL gene that spans exons 2 to 16 (Table S2). We identified 8 unique somatic mutations in c-CBL exons among 8 different patients. A variation L620F, a known SNP (rs2227988) in exon 11 was also detected. Importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-CBL genomic DNA obtained from lung tumor samples (Table 1). Moreover, none of the 8 mutations were detected in the corresponding normal tissue, indicating that these were somatic mutations. Four synonymous single nucleotide variations (SNVs) were also identified but were not used further in this study.  Table 1 Table 1 c-CBL mutation analysis in 119 lung cancer patient tumor tissues. Three of the 8 novel non-synonymous mutations were located in the TKB (tyrosine kinase binding) domain (S80N, H94Y, and Q249E), one in the RING finger domain (V391I), one in the proline-rich region (72515_72517 del ATG), and three in the C-terminal region (W802*, R830K, and A848T) of the c-CBL protein (Figure 1A and Figure S1). In Figure 1B, we show model chromatograms of representative samples.  Figure 1 Figure 1 c-CBL mutations and LOH in non-small cell lung cancer. 11q LOH of c-CBL Gene  Paired lung tumor and normal lung tissue samples from Taiwanese patients (nâ€Š=â€Š37) were investigated for LOH. Eight (21.6%) showed LOH at the c-CBL locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at the microsatellite markers (Figures 1C and D).  c-CBL Mutations in Different Ethnic Groups  The c-CBL double mutant S80N/H94Y was found in the same patient and the overall mutation rate for c-CBL in lung tumors was 6.7% (8/119). The frequency of c-CBL mutation was highest in large cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed in adenocarcinoma (AD) (1.8%; 1 of 53 patients), although these rates were not statistically significant (pâ€Š=â€Š0.292). Mutation rates were 6.0% among Caucasians (0 of 20 in AD; 0 of 10 in SQ; and 3 of 20 in LC), 13.8% in African-Americans (1 of 10 in AD; 1 of 10 in SQ; and 2 of 9 in LC), and 2.5% (0 of 23 in AD; 1 of 12 in SQ; and 0 of 5 in LC) in the Taiwanese population. Additionally two Taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had the known SNP L620F. Ethnic differences were not statistically significant, however the power to detect differences was low.  Mutations in MET and EGFR Can Be Co-Associated with c-CBL Alterations  Since East Asians with lung cancer have a higher frequency of EGFR and MET mutations in lung tumors [21], [22], we also determined mutations in EGFR and MET in the same Taiwanese cohort samples and compared the results with the observed c-CBL alterations (LOH and/or mutations). In the 37 samples tested, we did not find any overlap between c-CBL mutations and c-CBL LOH (Figure 2). Of the three c-CBL mutants (including the known L620F SNP, rs2227988), one of the samples had a MET mutation (N375S) and the other had an EGFR mutation (L858R). Among the 8 samples that had a LOH at the c-CBL locus 5 had an additional mutation in MET (N375S) and 2 had an EGFR exon 19 deletion. Twenty-six samples had neither c-CBL mutation nor c-CBL LOH (3 patients had a c-CBL mutation but no c-CBL LOH). Among these 26 samples 9 had a MET mutation (8 N375S, 1 L211W), 13 had an EGFR mutation (7 exon 9 deletion, 6 L858R) and 4 had no other MET or EGFR mutation. Thus the rate of MET or EGFR mutations among patients with LOH at the c-CBL locus (7 of 8) was similar to that seen in patients without c-CBL mutation or LOH (22 of 26 patients) (pâ€Š=â€Š0.99). These 4 patients with no identifiable mutation in c-CBL, MET or EGFR represented 10.8% of the 37 patients analyzed in the Taiwanese patient cohort. Conversely, 89.2% Taiwanese lung cancer patients have an identifiable mutation in either c-CBL, MET or EGFR or a combination of the three genes (Figure 2). Additionally, we determined p53 and KRAS mutations in these Taiwanese cohorts. Two p53 and 1 KRAS mutation were detected. The single KRAS mutation overlapped with one p53 mutation. This patient also had the EGFR exon 19 deletion but had no c-CBL mutation. The other p53 mutation sample had a c-CBL LOH with concurrent MET N375S mutation. Thus, in the Taiwanese samples analyzed, p53/KRAS mutations and c-CBL mutations were mutually exclusive (data not shown).  Figure 2 Figure 2 c-CBL mutations and relationship to MET and EGFR mutations in lung cancer. Cellular Functions of c-CBL Alterations in the Context of Lung Tumorigenesis  A. E3 activity is intact in the mutant c-CBL proteins To investigate whether the different c-CBL mutations affect the E3 activity, EGFR was chosen as a model substrate for c-CBL E3 function. All of the c-CBL mutants tested enhanced ubiquitination of the activated EGFR similar to the wild-type c-CBL protein. This result demonstrates that the catalytic activity of the c-CBL mutants is not impaired when EGFR was the substrate. (Figure 3A).  Figure 3 Figure 3 Ubiquitination, viability, expression and cell cycle analysis of various c-CBL mutants. B. Effect on lung cancer cell viability The effect of a representative c-CBL mutant from each of the three ethnic backgrounds on lung cancer cell viability in cell lines was determined. S80N/H94Y double mutation, Q249E, and W802* were identified in lung tumor samples obtained from a Caucasian, a Taiwanese and an African-American, respectively. As described in methods, the c-CBL wild-type (WT) and the above three mutants were expressed after cloning them into pAlterMax vector in A549 cells. These cells express relatively low basal levels of endogenous c-CBL (data not shown). Transfection efficiency was comparable between different groups and the number of cells transfected with c-CBL wild-type construct was about 70% compared to control cells that were transfected with the empty vector. Cells transfected with S80N/H94Y, Q249E and W802* c-CBL mutant constructs resulted in increased number of viable cells that was 132.3%, 120.8% and 147.9% higher respectively, relative to the empty vector control transfected cells and significantly different from the wild-type construct (pâ€Š=â€Š0.022, pâ€Š=â€Š0.049, and pâ€Š=â€Š0.008, respectively) (Figure 3B). Relative levels of c-CBL protein in whole cell lysates prepared from samples obtained from a parallel experiment was determined. The c-CBL protein levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c-CBL WT and the three c-CBL mutants were comparable (Figure 3C).  C. Effect on cell cycle To investigate if the increases in cell viability in different c-CBL mutants are due to increased cellular proliferation, a cell cycle analysis was performed. A549 cells were transfected with the c-CBL WT or the three different mutants: S80N/H94Y, Q249E and W802*. The empty vector transfectant was used as a control. Forty-eight hours after transfection cell cycle analysis was performed as described in materials and methods. There was no significant change in the subG1, G1 or the S phase of the cell cycle among the different mutants compared to the WT construct (pâ€Š=â€Š0.64, pâ€Š=â€Š0.40, and pâ€Š=â€Š0.28, respectively). The G2/M phase of the cell cycle showed an increase in cell numbers for the three mutants, S80N/H94Y, Q249E and W802*, when compared to the WT but again the difference was not statistically significant (pâ€Š=â€Š0.25) (Figure 3D).  D. Effect on cell motility To investigate the effect of the expression of the above three c-CBL mutants on cell migration, we carried out wound healing assay as described in materials and methods. The closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (Figure 4A). In all the samples, that represented cells transfected with mutants, the wound gap was much smaller than that seen in the sample that represented cells transfected with c-CBL WT (p<0.001). We also determined the rate of wound closure for all the five groups. At 48 h, wild-type c-CBL transfectants showed 61.1% open wound while the S80N/H94Y, Q249E and W802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p<0.001) (Figure 4B).  Figure 4 Figure 4 c-CBL mutations affect wound healing in A549 cells. E. c-CBL knockdown increases cell viability It is hypothesized that the LOH seen in our samples could lead to decreased expression of c-CBL. Thus we tested the effect of c-CBL knockdown in lung cancer cells. Compared to A549, H358 lung cancer cells express relatively high levels of endogenous c-CBL (data not shown). c-CBL expression was knocked down using lentiviral construct that expressed c-CBL specific shRNA and compared the results with those that were transduced with scrambled shRNA and the results are shown in Figure 5. We identified several clones that revealed varying degrees of c-CBL knockdown showing different sets of c-CBL lentiviral shRNA knockdown efficiency (Figure 5A). Of all the clones tested, Clone 27 was chosen for further experiments. Equal amount of cells were seeded in a 6-well plate and the cell proliferation was measured at various times and the results are depicted in Figure 5B. As expected, number of cells increased in a time dependent fashion from 100 to 190% relative to scrambled shRNA as control in a span of 48 h (pâ€Š=â€Š0.0002) (Figure 5B). The cell cycle phases in H358 cells that were knocked down with c-CBL shRNA were looked at and compared with the scrambled shRNA. There were no discernable differences between these two constructs in the different phases of the cell cycle (data not shown).  Figure 5 Figure 5 Knockdown of c-CBL using an shRNA increases cell proliferation. Go to: Discussion Our results demonstrate that c-CBL is somatically mutated (or has LOH) in lung cancers, and can significantly contribute to enhanced cell viability and motility. There was also a high prevalence of LOH with respect to c-CBL in lung tumors that harbored MET or EGFR mutation.  In the present study, we have demonstrated the occurrence of c-CBL mutations in lung cancer patients, especially with different ancestral variations. Mutations in c-CBL have been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies. In the AML study, the mutation R420Q located in the junction of the RING finger and the linker region inhibited FMS-like tyrosine kinase 3 (FLT3) internalization and ubiquitination [20], thus contributing to the gain-in-function for the RTK. In addition, mutations such as H398Y, C384R, and L380P mapped to the RING finger domain and the linker region of c-CBL that is required for its E3 activity [23], [24], [25], [26], [27]. Additionally, homozygous mutations in the RING finger domain of the c-CBL gene were described as a result of acquired Uniparental Disomy (UPD) [26]. It is important to note that our results indicate LOH at 11q23 locus and these are mutually exclusive from missense mutations of c-CBL. The somatic mutations were all heterozygous. The mutations in AML led to abrogation of the E3 activity leading to prolonged RTK activation. In addition mutants located on the linker region surrounding the RING finger domain exhibited enhanced AKT signaling in response to cytokine stimulation [26]. In addition, it was shown in NH3T3 cells, that neither mutations in the RING finger nor the linker region causes transformation, however while certain mutations perturbs the ubiquitination, others affect receptor recycling and prolong kinase activity [28].  We report here c-CBL mutations that mapped not only to the RING finger domain, but also to the TKB domain, proline-rich domain and the C-terminal region, but none mapped to the linker region as reported in the AML studies described above [23], [24], [25], [26], [29]. In addition, 8 mutants that we detected were found in different ethnic backgrounds. For example, S80N/H94Y, Q249E, W802* were detected in Caucasians, Taiwanese and African-Americans, respectively. The results point out not only the difference between lung cancer and other cancers, but also genetic polymorphism among different races in the same cancer. Interestingly, there is a large disparity between African-American and other ethnic populations with lung cancer [30]. We have previously shown that there was a low frequency of EGFR and MET mutation in African-Americans as compared with Taiwanese and Caucasians [31]. In this study, the number of African-American samples analyzed was relatively fewer and we found 3 mutations that are unique to this ethnicity. It would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for African-Americans. Our results provide evidence of the importance of c-CBL in tumorigenesis and potential signaling. Our prediction, based on the AML data, would be that the V391I RING finger domain mutation would affect the E3 activity. Also, it will be important to determine the binding partners for c-CBL in the TKB domain and proline rich domain mutations. It has previously been shown that the TKB domain can bind to growth factor receptors and it will be important to determine the cross-binding of these mutants to MET and or EGFR. It would also be important in the future to look at fluorescence in situ hybridization/copy number changes in c-CBL in lung cancer.  c-CBL plays an important role in down regulating RTK-mediated signaling through K63 poly-ubiquitination and subsequent downregulation of RTKs followed by lysosomal degredation [3]. Mono-ubiquitination or ubiquitinated with K63-linked chains of substrates by c-CBL may lead to enhancement of biological and biochemical functions (reviewed in Hermann et al, 2007 [32]). The mutations that we analyzed in our studies all point out to the fact that the E3 activity of c-CBL on EGFR is intact; the EGFR levels in the various mutants remain same (Figure S2). Multiple kinases, both RTKs and non-RTKs could be acted upon by c-CBL, including ERBs, PDGFR, FMS, MET, c-Kit, VEGFR, FLT-1, RON, FGFR, IR, as well as SYK, FYN, LCK, FGR, LYN and c-ABL [3]. In lung cancers the relevant substrates of c-CBL in terms of degradation or signal transduction are yet to be identified.  The observation that c-CBL somatic mutations, especially S80N/H94Y, Q249E and W802* showed increased cell viability and cell motility that is in agreement with the physiological role for D-cbl in the regulation of apoptosis and differentiation identified in Drosophila is very significant [33]. It has been previously shown that activating c-CBL mutation downregulates EGFR signaling and decreases cellular proliferation and migration in breast cancer cell lines [34]. Although the role of c-CBL in the negative regulation of RTKs is well substantiated, thereby suggesting that it is a natural tumor suppressor, studies in cancer cells have revealed both tumor suppressor and tumor promoting activities depending on the type of c-CBL mutation and the number of alleles at the c-CBL locus [24]. In agreement with the above, the three c-CBL mutants described here appear to have tumor growth and metastasis promoting properties. Although these mutants are outside of the RING finger or the linker region of c-CBL, their downstream effects are significant so as to cause increased proliferation and migration, but the substrate affected by these mutations are not known yet. This raises the possibility that some of the cellular functions of c-CBL are independent of its ubiquitin-ligase activity, an area that we are currently investigating. The oncogenic nature of RTKs, addiction of cancers to growth signals and given the clustering of c-CBL, EGFR and MET mutations, it is possible that the transforming effect of c-CBL mutations is most likely a combinatorial effect of the three. We also show that LOH for c-CBL was found in a significant number of samples that harbored MET or EGFR mutations. The fact that about 7% of lung tumor samples are likely to have c-CBL mutations and an additional 22% are likely to harbor c-CBL-related LOH, makes c-CBL a highly mutated molecule in lung cancer. Since LOH alone is not enough to cause a transforming event [35], [36], [37], associated mutation in the MET or EGFR locus or yet another RTK discussed above may play a role in carcinogenesis. We predict that this LOH in c-CBL results in haploinsufficiency that downplays RTK ubiquitination leading to hyperactivity of the RTKs. However, whether this is sufficient cause for tumorigenesis remains to be determined. Consistent with our hypothesis is the fact that c-cbl-/- mice have increased kinase activity in lymphocytes, but is not sufficient for tumor formation [35], [36], [37]. c-CBL LOH could also lead to increased expression of c-CBL from the other allele to compensate for a loss of an allele. Alternately, there could be some form of synergy working with reduced c-CBL levels and mutated receptors that exacerbate the phenotype of each alone.  Previous studies from our lab and others have shown that East Asians with lung cancers have relatively high frequencies of gain-of-function of mutations in RTKs such as EGFR and MET [31]. In a cohort of Japanese patients an activating MET mutation was identified in the splice region that deletes the juxtamembrane domain that is involved E3 activity of c-CBL [38]. This study also found that activation of MET is mutually exclusive of EGFR, KRAS and HER2 gene mutations [38]. We failed to detect such mutations in significant numbers in lung tumor samples obtained from African-Americans (nâ€Š=â€Š29) and Caucasian (nâ€Š=â€Š50) patients. One MET mutation was identified in each of the groups whereas 1 and 3 EGFR mutations were identified in the African-American and Caucasian cohorts respectively. EGFR mutations have earlier been identified as one of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. Our study encompasses different histologies of NSCLC. However, the published series did not find any mutations in c-CBL or MET unlike our study that encompassed different subtypes of NSCLC. It is important to note, we have recently shown that MET mutations in lung cancer are in majority germline [31]. We have reported earlier c-CBL mutations in a small cohort of Taiwanese lung cancer samples [40]. In our efforts to understand the ethnic differences in the lung oncogenome, we also looked at PAX transcription factors such as PAX5 and PAX8 that are highly expressed in lung cancers; however there was no preferential expression or mutations of the above genes in lung tumor samples of African-Americans. In this study, we show relatively high frequency of c-CBL mutations in lung cancers, especially in the large cell type among Caucasians and particularly among African-Americans. We therefore propose c-CBL as an efficacious target for lung cancers in African-Americans that needs to be further substantiated. This is all the more important because the prognosis for African-Americans with lung cancer, especially for men is much poorer compared to their Caucasian counterparts [41].  In conclusion, the results presented in this study demonstrate that c-CBL is frequently mutated or even lost in lung cancers. Our results support a role for c-CBL mutants that are independent of its ubiquitination activity. Given the relatively high mutation rates in c-CBL as well as RTKs such as MET and EGFR, it is likely that their combined effect could be synergistic in promoting tumorigenesis.\""
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['Text'][2]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Oncogenic mutations in the monomeric Casitas B-lineage lymphoma (Cbl) gene have been found in many tumors, but their significance remains largely unknown. Several human c-Cbl (CBL) structures have recently been solved depicting the protein at different stages of its activation cycle and thus provide mechanistic insight underlying how stability-activity tradeoffs in cancer-related proteins may influence disease onset and progression. In this study, we computationally modeled the effects of missense cancer mutations on structures representing four stages of the CBL activation cycle to identify driver mutations that affect CBL stability, binding, and activity. We found that recurrent, homozygous, and leukemia-specific mutations had greater destabilizing effects on CBL states than did random non-cancer mutations. We further tested the ability of these computational models assessing the changes in CBL stability and its binding to ubiquitin conjugating enzyme E2, by performing blind CBL-mediated EGFR ubiquitination assays in cells. Experimental CBL ubiquitin ligase activity was in agreement with the predicted changes in CBL stability and, to a lesser extent, with CBL-E2 binding affinity. Two-thirds of all experimentally tested mutations affected the ubiquitin ligase activity by either destabilizing CBL or disrupting CBL-E2 binding, whereas about one-third of tested mutations were found to be neutral. Collectively, our findings demonstrate that computational methods incorporating multiple protein conformations and stability and binding affinity evaluations can successfully predict the functional consequences of cancer mutations on protein activity, and provide a proof of concept for mutations in CBL.  Keywords: CBL, driver mutations, protein interactions Go to: Introduction Whole exome sequencing of cancer patients has produced unprecedented amounts of data to analyze and interpret; these studies report a very large fraction of missense mutations which can potentially be implicated in tumorigenensis (1). Although some missense mutations can provide selective growth advantage to tumor cells (driver mutations), the large majority of them are considered to be neutral (passenger mutations). The mechanisms by which the driver variants may affect protein stability, interactions, and function remain largely unknown. Various computational methods have been developed to estimate the impacts of disease mutations on proteins but most of them exclusively use sequence features and do not explicitly utilize the protein three-dimensional structures, their physico-chemical properties and dynamics (2, 3). Many cancers are characterized by (de)activation of certain proteins which may be a result of missense mutations (4). The interconversion between active and inactive states is highly regulated in proteins and it is not well understood how these regulatory mechanisms are disrupted in cancer. The development of in silico approaches to estimate the effects of disease mutations on protein activity, stability and binding will help to define which are likely to be driver or passenger mutations. Moreover, understanding the mechanisms of their actions would allow for prioritization of potential driver candidates for better targeted therapies to design drugs which might in turn compensate for the reduced/enhanced protein stability or activity.  The monomeric Casitas B-lineage lymphoma (Cbl) RING finger ubiquitin ligase (E3) represents an exceptionally difficult yet important system to study the mechanisms of cancer mutations (5, 6). Strikingly, proteins from this family play both positive and negative regulatory roles in tyrosine kinase signaling which is aberrantly activated in many cancers (5). Oncogenic mutations in the c-Cbl gene (referred to as CBL thereafter) were found in human myeloid neoplasms and other tumors (5) but the significance of these mutations and their impacts on CBL function were studied only for very few mutants (7). The mechanistic aspects of CBL cancer mutations can now be adequately addressed as several CBL structures have become available which represent the snapshots of different stages of the CBL activation cycle (Fig. 1). All CBL proteins share a highly conserved N-terminus which includes a tyrosine kinaseâ€“binding domain (TKBD), a linker helix region (LHR) and a RING finger domain, while the C-terminus comprises a proline-rich region (8). The RING domain of CBL has E3 activity and ubiquitinates activated receptor tyrosine kinases which subsequently targets them for degradation (8). At the same time, since CBL proteins can bind to activated receptor tyrosine kinases via the TKBD domain, they can serve as adaptors by recruiting downstream signal transduction components such as SHP2 and P13K (9, 10).  Figure 1 Figure 1 CBL activation cycle. The structures representing the activation cycle of CBL are shown. In the inactive closed state (nCBL) the protein exists in the cytosol. Upon activation of the RTK, CBL can bind to the phosphorylated RTK (orange peptide) via TKBD ... Another aspect of CBL function which should be accounted for in modeling the effects of cancer mutations is that it can bind to the ubiquitin-conjugating enzyme (E2) in complex with ubiquitin (Ub) and a substrate protein, thereby facilitating the transfer of Ub from E2 to a lysine residue of the substrate (11). The crystal structure of the inactive complex of CBL with E2 (UbcH7) was solved more than ten years ago (12) while the active phosphorylated CBL-E2 (UbcH5B) complex was resolved fairly recently (13). According to the latter study, substrate binding and Tyr371 phosphorylation activates CBL by producing a large conformational change in order to place the RING domain and E2 in close proximity to the substrate. It was further confirmed that phosphorylation-induced conformational change is required for positioning of ubiquitin for effective catalysis (14).  Here we present a new approach, which aims to assess the effects of cancer mutations on stability, binding and activity of cancer related proteins. We apply computational models to four different stages of the CBL activation cycle (Fig. 1) and perform blind in vivo experiments of CBL-mediated EGFR ubiquitination. We show a rather remarkable agreement of experimental EGFR ubiquitination by CBL mutants with the computed changes in CBL thermodynamic stability and to a lesser extent with CBL-E2 binding affinity. The computational models not only quantitatively predict the magnitude of the effects of mutations but also shed light on the mechanisms of their action. Namely, we find that cancer mutations have greater destabilizing effects on four CBL states than random non-cancer mutations for recurrent, homozygous and leukemia mutations. Most damaging cancer mutations happen in the sites involved in Zn-coordination and in the formation of salt bridges and hydrogen bonds within CBL or between CBL and E2. Overall cancer driver mutations affect different or multiple stages of the CBL activation cycle either completely abolishing its E3 activity or partially attenuating it. The computational models based on stability and binding affinity calculations can discriminate experimentally validated driver from passenger mutations (with the exception of two mutations) and outperform several state-of-the-art bioinformatics methods aiming to predict phenotypic impacts of mutations.  Go to: Materials and Methods Computational modeling and analysis  Mapping of CBL mutations The COSMIC database (15) stores data on somatic cancer mutations and integrates the experimental data from full-genome sequencing studies. We extracted 103 missense mutations for the CBL gene from the COSMIC database (15) that could be mapped to four available CBL structures in its activation cycle (Fig. 1). Cancer mutations were classified into different classes according to the frequency of observed samples (single and recurrent mutations), zygosity (homo- and heterozygous mutations), types of cancer (leukemia and sarcoma) and the involvement of mutations in Zn-coordination (Table S1). In addition, all possible single nucleotide substitutions resulting in amino-acid changes in the CBL gene were performed to obtain a â€œRandomâ€� missense mutation reference set. After excluding mutations observed in COSMIC and mutations occurring on residues without known coordinates in crystal structures, we obtained 2,102 random missense mutations (Table S1). We also searched the dbSNP database (16) but found very few benign variations in the CBL gene. Detailed description is provided in Supplementary Materials and Methods.  Model preparation We investigated the effects of mutations on four states of the CBL activation cycle: (1) closed CBL state (nCBL), (2) partially opened CBL state bound to substrate (CBL-S), (3) unphosphorylated autoinhibitory CBL bound to substrate and conjugating enzyme UbcH5B (CBL-E2-S) and (4) phosphorylated CBL bound to substrate and UbcH5B (pCBL-E2-S) (Fig. 1). It was previously noted that both UbcH5B and UbcH7 can bind specifically to CBL but only UbcH5B can facilitate ubiquitination (17). The crystal structures of nCBL (PDB id: 2Y1M) (13), CBL-S (PDB id: 2Y1N) (13) and pCBL-E2-S (PDB id: 4A4C) (13) were obtained directly from the Protein Data Bank (PDB) (18). Only one crystal structure of the unphosphorylated inactive state of CBL bound to E2 (UbcH7) and Zap-70 peptide (PDB id: 1FBV) (12) was available in PDB. Although sequence identity between UbcH7 and UbcH5B proteins is 38%, the structural similarity is very high with the root mean square deviation (RMSD) of 1.04 Ã…. The unphosphorylated autoinhibitory structure of CBL-UbcH5B-S was therefore modeled based on CBL-UbcH7-S (PDB id: 1FBV) using Chimera (19). (Detailed description is provided in Supplementary Materials and Methods.)  Minimization procedure We applied our recently developed optimization protocol for minimizing wild-type and mutant structures (20). Heavy side chain atoms without known coordinates and hydrogen atoms were added to the crystal structures using the VMD (version 1.9.1) program (21) with models immersed into rectangular boxes of water molecules extending up to 10 Ã… from the protein in each direction. Wild-type protein complexes were minimized for 40,000 steps using explicit TIP3P water model. The final minimized models of wild-type protein complexes were used to produce all mutant structures and then an additional 300-step minimization for all mutant structures was performed. The energy minimization was carried out with the NAMD program (version 2.9) (22) using the CHARMM27 force field (23). For unfolding free energy calculations we applied the optimization procedure implemented in the RepairPDB module of the FoldX program (24) which optimizes the side chain configurations to provide a repaired structure. (See Supplementary Materials and Methods for more information.)  Binding and unfolding free energy calculations Binding free energy and effects of mutations on binding affinity were calculated according to the approach introduced by us earlier (20). Energy calculations were based on the modified MM-PBSA method that combined the molecular mechanics terms with the Poissonâ€“Boltzmann continuum representation of the solvent (25) and statistical scoring energy functions with parameters optimized on experimental sets of several thousand mutations (equation 3). Binding free energy calculations were performed on minimized wild-type and mutant structures of inactive CBL-E2-S and active pCBL-E2-S states. The binding energy is defined as a difference between the free energies of the complex and unbound proteins.  Î”Gbind=GCBL-E2-S-GCBL-S-GE2 (1) The change of binding energy due to a mutation can be calculated as:  Î”Î”Gbind=Î”Gbindmut-Î”GbindWT (2) The effect of mutations on binding affinity was calculated in this work using the following energy function:  Î”Î”Gbind=Î±Î”Î”Evdw+Î²Î”Î”Gsolv+Î³Î”SAmut+ÎµÎ”Î”GBM+Î»Î”Î”GFD+Î´ (3) Here Î”Î”Evdw is the change of the van der Waals interaction energy and Î”Î”Gsolv is the change of the polar solvation energy of solute in water. Î”SAmut represents a term proportional to the interface area of the mutant complex. Î”Î”GBM and Î”Î”GFD are the binding energy changes from BeAtMuSiC and FoldX, respectively. BeAtMuSiC (26) estimates the effect of mutations on binding based on statistical potentials.  The FoldX software program (24) was used to estimate the unfolding free energy and to model the unfolding state. FoldX calculates the effects of mutations on protein stability using an empirical force field. The BuildModel module was used to introduce a mutation, optimize the configurations of the neighboring side chains and calculate the difference in stability (unfolding free energy) between mutant ( Î”Gfoldmut) and repaired native structure ( Î”GfoldWT).  Î”Î”Gfold=Î”Gfoldmut-Î”GfoldWT (4) For comparison we used three alternative methods to predict the impacts of mutations on unfolding free energy: the Eris server (27), the Rosetta program (28) and the PoPMuSiC server (29). We also applied four additional webtools to assess the impacts of amino acid substitutions on CBL function: PROVEAN (30), PolyPhen-2 (31), MutationAssessor (32) and InCa (33). (See Supplementary Methods for more information.)  Go to: Experimental procedures Expression constructs  Human CBL cDNA was originally obtained from Wallace Langdon and subsequently cloned into the pCEFL expression plasmid (34). Point mutation constructs, described above, were created from wild-type CBL using the QuikChange II Site-directed Mutagenesis Kit according to the manufacturerâ€™s instructions (Stratagene, La Jolla, CA). All constructs were confirmed by DNA sequencing.  Cell culture and transfections  The human embryonic kidney cell line HEK293T, the human nonsmall cell lung cancer cell line A549 and the human cervical cancer cell line HeLa used in this study were originally obtained from ATCC and maintained in culture using DMEM (Gibco, Grand Island, NY) supplemented with 10% FBS, 100 U/ml penicillin and 100 Î¼g/ml streptomycin sulfate. Cell lines were authenticated by Short Tandem Repeat (STR) analysis by using either Promega Powerplex 16 (Promega, Madison, WI) or the AmpFISTR Identifier Kit (Life Technologies, Carlsbad, CA) and compared to the ATCC or DSMZ databases. HeLa cells were received in the lab in 2013 last authenticated on 7-30-15, A549 were received in the lab in 2015 and last authenticated on 8-12-15, and HEK293T were received in the lab in 1995 and last authenticated on 10-8-15.HEK293T cells were transfected using calcium phosphate according to the instructions accompanying the reagent (Profection; Promega Corp., Madison, WI), incubated 18 h prior to media change and grown for a total of 48 h prior to harvesting. A549 and Hela cells were transfected with Lipofectamine 2000 (Invitrogen, Carlsbad, CA). Transfections were allowed to incubate 6 h prior to media change, and cells were grown an additional 48 h before being harvested. Cells were starved for 4 hours and treated with EGF (100 ng/ul). Each cell-based experiment was repeated at least two times.  Immunoblotting and immunoprecipitation  To harvest proteins, cells were washed twice in ice-cold Dulbeccoâ€™s PBS-containing 200 Î¼M sodium orthovanadate (Fisher Chemicals, Fairlawn, NJ) then lysed in ice-cold lysis buffer (10 mM Tris-HCl pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 10% glycerol, 100 mM iodoacetamide [Sigma-Aldrich Corp., St. Louis, MO], 2 mM sodium orthovanadate, and protease inhibitors [Complete tabsÂ®, Roche Diagnostics Corp., Indianapolis, IN]). All whole cell lysates were cleared of cellular debris by centrifugation at 16,000 Ã— g for 15 m at 4 Â°C. Supernatant protein concentrations were determined using the BioRad protein assay (BioRad, Hercules, CA). For immunoblotting, 20Î¼g of whole cell lysates were boiled in a 1:1 dilution of 2X loading buffer (62.5 mM Tris-HCl pH 6.8, 10% glycerol, 2% SDS, 1 mg/ml bromophenol blue, 0.3573 M Î²-mercaptoethanol) for 5 m. For immunoprecipitations, 150 Î¼g of each of the whole cell lysates were incubated with rabbit anti-EGFR (Ab-3; Millipore) and with Protein A/G+ agarose beads (sc-2003; Santa Cruz Biotechnology, Santa Cruz, CA). All immunoprecipitations were incubated overnight at 4Â°C with tumbling. Immune complexes were washed five times in 1 mL cold lysis buffer, then resuspended in 2X loading buffer, boiled for 5 m, then resolved by SDS-PAGE and transferred to nitrocellulose membranes (Protran BA85; Whatman, Sanford, MA). For immunoblot detection of proteins, the following antibodies were used: rabbit anti-EGFR (2232L; Cell Signaling), rat monoclonal high affinity anti-HA-peroxidase, (clone 3F10; Roche), rabbit anti-Cbl (sc-C-15; Santa Cruz) and mouse anti-Hsc70 (sc-7298; Santa Cruz). Horseradish peroxidase linked donkey anti-rabbit IgG (NA934V; GE Healthcare, Piscataway, NJ), or donkey anti-mouse IgG (NA931: GE Healthcare, Piscataway, NJ) immunoglobulin was used with SuperSignal (Pierce Biotechnology Inc., Rockford, IL) to visualize protein detection.  Densitometric Analysis  Immunoblots were developed on HyBlot CL Autoradiography Film (Denville Scientific Inc., South Plainfield, NJ) with an X-OMAT automated processor (Eastman Kodak, Rochester, NY). Protein expression levels were then recorded using an Epson Perfection V750 PRO scanner (Epson Inc., Long Beach, CA) and densitometric analysis was performed using Adobe Photoshop software version 7.0 (Adobe Systems Inc., U.S.A). EGFR ubiquitination signal intensity was determined by optical density in a set area normalized against the EGFR band intensity in parallel immunoblots and expressed as a densitometric ratio of ubiquitination/EGFR levels. The mean densitometric ratio of EGFR ubiquitination, in the presence of each CBL mutation, was then assessed relative to wild-type CBL where the ratio was set at 1.0.  Go to: Results Cancer mutations impact CBL stability  We estimated the thermodynamic stability (unfolding free energy) changes, Î”Î”Gfold, upon mutation for closed (nCBL) and partially opened (CBL-S) CBL states (Fig. 1). These states did not involve binding to E2. The examination of the Î”Î”Gfold distribution for random non-cancer mutations (Fig. 2A) showed that it was very similar to experimental distributions produced by random mutagenesis on a set of different proteins (35, 36). Namely, the distribution for random non-cancer mutations was asymmetrically centered at positive energy values and there were about 10% of random mutations with highly damaging effects on CBL structures by more than 5 kcal molâˆ’1 (Fig. 2A). Despite the presence of highly damaging random mutations, recurrent cancer mutations overall produced significantly larger destabilizing effects compared to random (p-value â‰ª 0.01) for both nCBL and CBL-S states (Fig. 2A and S1, Table S2). This was not the case for single cancer mutations observed in only one patient (Table S2). Overall, the form of the Î”Î”Gfold distribution for recurrent cancer mutations was different from the Î”Î”Gfold distribution of random mutations (Kolmogorov-Smirnov test p-value â‰ª 0.01) whereas the distribution for single cancer mutations was indistinguishable from a random mutation distribution. However, it does not mean that all single cancer mutants can be considered passenger, as we show later, this is not the case for some of them.  Figure 2 Figure 2 (A) The distribution of unfolding (Î”Î”Gfold) and binding (Î”Î”Gbind) free energy changes for Single, Recurrent and Random class mutations on nCBL and pCBL-E2-S states (see SI for other states). Positive and negative values ... The top 25% cancer mutations with the largest damaging effects are presented in Table S3. Many of them occurred in Zn-coordinating sites while many others did not involve Zn-coordination sites. Some of these latter mutations (G415V, G413R and G415S) introduced large Van der Waals clashes with neighboring residues which could not be accommodated by side chain rearrangements, whereas other mutations (S376F, Y371D and L405P) affected disulfide or hydrogen bonds. For example, L405P located in the middle of an Î±-helix affected the CBL structure by introducing an energetically unfavorable kink in the helix due to its inability to donate an amide hydrogen bond.  Effects of cancer mutations on CBL-E2 binding  The outcome of mutations can be assessed by the extent of structural changes they induce. We calculated the local root mean squared deviation (RMSD) between the minimized wild-type and mutant structures around the mutated site. We found that the protein backbone in the vicinity of a mutation of the CBL-E2 complex underwent larger local conformational changes upon recurrent cancer mutations, especially for mutations occurring in Zn-coordinating site clusters, compared to random mutations (p-value â‰ª 0.01, Fig. S2 and S3). In the previous section we analyzed the original closed (nCBL) state and partially opened (CBL-S) state induced by substrate binding. In this section we study the effects of cancer mutations on two other CBL states: an autoinhibitory CBL state bound to substrate and conjugating enzyme UbcH5B (CBL-E2-S) and phosphorylated active CBL state bound to substrate and UbcH5B (pCBL-E2-S) (Fig. 1) (13). Effects of mutations on binding can be in general linked with their structural locations, therefore we examined the locations of mutations and found that about one third of all cancer mutations were located on the CBL-E2 interface (Fig. S4). This preference was found to be statistically significant (p-value â‰ª 0.01) (Fig. S4).  It was previously experimentally shown that binding between CBL and E2 was rather weak with a micromolar dissociation constant (13). Despite the fact that the interface between CBL and E2 in the inactive state is several residues larger than in the active state, consistent with experiments, we found somewhat stronger binding between CBL and E2 for the active state of CBL (Table S4). Similarly to the impacts of cancer mutations on stability, recurrent (but not single) mutations destabilized CBL-E2 binding significantly (Fig. 2B, S1 and Table S2). Importantly, cancer mutations reduced the CBL-E2 binding in the active state considerably more than in the inactive state (p-value = 0.002) (Fig. 2B) and the effects of cancer mutations on CBL-E2 binding of the active state were noticeably larger compared to random mutations even if we excluded mutations in Zn-coordinating sites (Table S2). This latter observation does not hold true for the autoinhibitory CBL-E2-S conformation.  Next, all cancer mutations were ranked with respect to their effects on CBL-E2 binding. About half of all highly damaging mutations (top 25% of most damaging mutations, Table S3) impacted both stability and CBL-E2 binding for all four CBL states. Among them, several mutations occurred in Zn-coordinating clusters and M400 and L405 sites. Another class of mutations (involving W408 and R420 sites) mostly influenced CBL-E2 binding. For example, the W408S mutation caused the largest perturbation in CBL-E2 binding of the active state by decreasing binding affinity up to 3.5 kcal molâˆ’1 but having a moderate effect on stability of the nCBL and CBL-S states. It was previously suggested that amino acids in positions W408 and I383 constituted specificity of CBL-E2 binding and corresponded to binding hot spots (10, 12). Mutations in the I383 site are not recorded in the COSMIC database although they have a profound destabilizing effect on CBL-E2 binding for both states (see data for random mutations on the ftp site ftp://ftp.ncbi.nih.gov/pub/panch/CBL). Another site affecting binding of CBL (R420) is a highly conserved site in the CBL family, it strongly interacts with the Q92, W93 and S94 residues of E2. It was previously experimentally verified that mutations in this site disrupted the CBL activity and could be associated with cytokine-independent growth (37). There are 24 cancer patient samples where mutations of this site (R420Q, R420L and R420P) are found and they all produce strong destabilizing effects on the active pCBL-E2-S state.  According to the zygosity annotation, there are 49 heterozygous (â€œHeteroâ€�) and 27 homozygous (â€œHomoâ€�) mutations (Table S1) and for the rest of cancer mutations their zygocity status is undetermined. Overall, we found that destabilizing mutations were enriched among homozygous compared to heterozygous mutations for all stages of CBL activation cycle (p-value = 0.021 â€“ 0.035) and were more prevalent in leukemia compared to sarcoma patients (p-value = 0.000 â€“ 0.003) (Fig. 2B, Table S2). This observation could not be attributed to the prevalence of homozygous mutations in Zn-coordinating clusters since these clusters had almost equal numbers of heterozygous and homozygous mutations. It was reported earlier that many homozygous mutations could be connected to uniparental disomy when germline heterozygosity would lead to neoplasia upon reduction to homozygosity (6, 38, 39). If some patients with myeloproliferative neoplasms had germline heterozygous mutations in CBL, then lost the wild-type CBL allele and duplicated mutant allele, it would be possible that the damaging mutant allele would be duplicated in a cancer cell with a higher probability. Indeed, human leukemia samples show the loss of the normal CBL allele.  CBL-mediated EGFR ubiquitination: comparing experiments with computational models  Next we randomly picked fifteen cancer mutations based on their predicted damaging status, trying to equally sample highly damaging and benign mutations irrespective to their frequencies in cancer samples, and experimentally tested the E3 activity of wild-type and mutant CBL proteins (see Methods, Table 1). As can be seen in Fig. 3A, in the presence of wild-type CBL, activation of the EGFR induced more than a 10-fold increase in the ubiquitination of the EGFR compared to empty vector (endogenous CBL levels) in HEK293T cells (compare lane 4 to lane 2 in top panel of Fig. 3A). The E3 activity of wild-type CBL was mirrored by a decrease in the levels of immunoprecipitated EGFR consistent with the targeting of ubiquitinated EGFR for degradation (compare lanes 2 and 4 in second panel of Fig. 3A).  Figure 3 Figure 3 (A) EGFR ubiquitination. HEK293T cells were transfected with plasmids encoding wild-type or mutant CBL proteins along with the EGFR and hemagglutinin (HA)-epitope tagged ubiquitin. The vector control and wild-type, CBL transfectant cells were incubated ... Table 1 Table 1 Experimental relative densitometry values for HEK293T cells, predicted changes in stability (Î”Î”Gfold kcal molâˆ’1), binding affinity (Î”Î”Gbind kcal molâˆ’1) and comparison with other methods for 15 experimentally ... We classified all mutations into three groups (damaging, attenuating and benign) according to their experimental relative densitometry data and compared experimental data with the estimates produced by the computational models. The first group of damaging mutants included C396R, H398Q, Y371H, K382E and C381A which completely abolished CBL activity (relative densitometry was less than 10%) (Fig. 3B) and the levels of total EGFR were not decreased by these mutants (Fig. 3A). Only two of these mutants belonged to the Recurrent class of mutations. Table 1 shows predicted and experimentally verified effects of mutations on all four CBL states. All five mutants were predicted to be damaging by stability and binding affinity calculations but the mechanisms of their action were different. C396R, H398Q and C381A mutations disrupted Zn-coordinating clusters and had very damaging consequences according to the stability model for both nCBL and CBL-S states and damaging effects on CBL-E2 binding even though none of these mutations were located on the CBL-E2 interface. On the other hand, the K382E mutation did not affect Zn-coordination but destabilized the nCBL state. This, in turn, could be explained by the charge substitution that led to the disruption of the K382-E373 salt bridge within the closed state of nCBL (Figure 4 and Figure S5A). Moreover, K382E mutation had a significant impact on CBL-E2 binding due to the disruption of a salt bridge between pY371 and K382 affecting the stability of CBL in the active state (Figure 4 and Figure S5D). No significant changes were observed for CBL-S and CBL-E2-S states (Figure S5B,C). K382E mutation was previously observed in Noonan syndrome and was speculated to affect CBL stability or binding (40). Finally, the Y371H mutation not only abolished phosphorylation at the Y371 site but also had a profound destabilizing effect on all four states as evident from Table 1.  Figure 4 Figure 4 Structures of wild type (green) and K382E mutant (tan) are shown for nCBL and pCBL-E2-S states. Salt bridges are shown as dashed lines and in both cases mutation affects the stability by disrupting the salt bridge. The second group of mutants (M374V, V430M, P428L, Q249E and double mutant S80N/H94Y) maintained the CBL activity equivalent to or greater than wild-type CBL (relative densitometry of 80% or higher) (Fig. 3B). Consistent with this, the levels of the activated EGFR were also decreased in these samples (Fig. 3A). To confirm that the retained E3 activity of these mutants was not cell type specific, we transfected these mutants into the non-small cell lung cancer cell line A549 and the cervical cancer cell line Hela (Fig. S6). The S80N/H94Y and Q249E mutations were previously identified in human non-small cell lung cancers making the A549 cell line a relevant cell type to investigate the function of these CBL mutants while other cancer mutations were found in other cancer types. As in the HEK293T cells, transfection of wild-type CBL into A549 and HeLa cells resulted in a significant increase in EGF stimulated ubiquitination of EGFR compared to the cells transfected with empty vector (Fig. S6). The increase in transfected CBL protein compared to endogenous CBL protein was less in HeLa than in either the A549 or HEK293T cells. Consistent with this, the fold increase in EGF stimulated ubiquitination of the EGFR by wild-type CBL was smaller in Hela cells (about three fold) compared to either A549 or HEK293T cells (more than 10 fold in each). As in HEK293T cells, the Y371H mutant did not stimulate ubiquitination compared to the vector transfected control for A549 or HeLa cells (Fig. S6A and C). The CBL mutants that were fully active in HEK293T cells (Fig. 3A and B) maintained CBL E3 activity in A549 and HeLa cells (Fig. S6). Two of these CBL mutants resulted in more ubiquitination of EGFR compared to wild type CBL. For example, the Q249E mutant showed an increased ubiquitination of EGFR in 293T and HeLa cells while the M374V mutant resulted in higher ubiquitination levels in A549 and HeLa cells (Figure 3 and S6). Only V430M mutant was on the borderline with the densitometry ratio of 0.8. All mutations from the second group were predicted to be benign according to our computational models, whereas V430M, consistent with experimental data, had a borderline destabilizing impact on CBL-E2 binding (Table 1).  Finally, the third group constituted mutations (L399V, G375P, P395A and V391I) which attenuated the CBL E3 activity according to the relative densitometry data (Fig. 3B). Only one of these mutants was observed in two cancer samples while other three belonged to the Single mutation class. Concordant with this, there were intermediate levels of EGFR (Fig. 3A). G375P and P395A mutations were predicted to have partially damaging effects while V391I and L399V were classified as benign in our predictions. Interestingly, mutations from this third group affected only some of the CBL states. In contrast, the highly damaging mutations from the first group had damaging impacts on almost all CBL states.  Next we tested if the reduction in CBL ubiquitination activity was directly correlated with the effects of mutations on stability (Fig. 5A). The relationship between experimental densitometry data and Î”Î”Gfold was better described by an exponential dependence with correlation coefficient (R) of 0.77 and 0.78 for CBL-S and nCBL states respectively (Table S5). Indeed, the change in absorption between wild-type and mutant proteins which refers to a fraction of active mutants can be described by the Boltzmann equation relating the probability of a state with the energy of this state. A mutation may lead to damaging effects and a loss of function if it impacts at least one of the CBL functional states. Taking this into consideration we calculated the correlation between densitometry and stability changes taking a maximum of Î”Î”Gfold values for the nCBL and CBL-S states. As shown in Fig. 5A and Table S5, changes in stability can indeed explain the effects of mutations on CBL ubiquitination activity with high correlation of R = 0.83. For comparison, several alternative methods were applied to predict the effect of mutations on unfolding free energy, they all reported correlation coefficients ranging from 0.21 to 0.52, with only one method PopMusic reporting a statistically significant correlation of 0.52 (Table S5). The relationship between densitometry and binding affinity changes was on the borderline of significance with a linear correlation coefficient of 0.48 and 0.62 (if two highly damaging mutations were excluded) (Fig. 5B and Table S5).  Figure 5 Figure 5 (A) Comparison between experimental densitometry values (exponential model) and maximal predicted changes in stability. (B) Comparison between experimental densitometry values (linear model) and predicted changes in binding affinity of pCBL-E2-S state. ... There are various methods which predict the phenotypic effects of mutations, (30â€“33, 41) and some of them use structural features (31, 41). We applied four state-of-the-art independent methods to predict the impacts of mutations on CBL function: PROVEAN (30), PolyPhen-2 (31), MutationAssessor (32) and InCa (33). Some of these methods outperformed our model in classifying cancer from random mutations since they were trained to distinguish disease from neutral variants (Table S6). However, all four methods had a very limited accuracy in classifying CBL cancer mutations into those that disrupted function and those that did not. As evident from Table 1 and Table S5, all methods except for PROVEAN over-predicted damaging effects of experimentally tested mutations while PROVEAN was the only method which produced a significant correlation between the densitometry data and PROVEANâ€™s score (R=0.58, Table S5).  Go to: Discussion Stability-activity balance in cancer related proteins  Evolutionary selection to maintain structural, foldable and functional proteins eliminates many mutations in protein sequences. On the other hand, thermodynamic stability can be compromised in evolution to ensure certain arrangements of catalytic and binding sites, which might not be energetically optimal (36, 42). In tumorigenesis, protein stability or binding may be reduced (or in some cases increased) due to cancer mutations. As a consequence it can lead to decreased fitness at the protein level, but may confer a fitness advantage for the population of tumor cells (43, 44). However, the extent of the stability-activity tradeoff in oncogenes and tumor suppressors remains largely unknown. Using the example of the CBL protein, here we tried to elucidate the stability-activity balance and to understand whether the loss or gain of activity in cancer-related proteins can be accompanied by compromised stability or binding.  In contrast to many other computational studies, which attempt to link stability with activity by mostly focusing on one protein conformation, we performed an analysis on four different stages of the CBL activation cycle. Our predictions of changes in stability and binding were further elucidated by the experimental CBL-mediated EGFR ubiquitination assays. We found a strong relationship between the effects of mutations on CBL stability and experimentally obtained densitometry data (quantifying activity), while a relatively weaker correlation was observed between changes in densitometry and CBL-E2 binding affinity. It could be explained either by a less significant impact of experimentally tested mutations on CBL-E2 binding or by a limited coupling (compared to CBL stability) between CBL-E2 binding and E3 activity.  Drivers or passengers?  According to our study, about two thirds of all experimentally tested mutations either completely abolished or attenuated E3 activity, while one third of them were neutral. Trying to assess the limitations of our models, we applied state-of- the-art methods commonly used to predict functional effects of mutations. However, these methods failed to distinguish inactivating from neutral mutations as accurately as the computational models which were solely based on accounting for the effects on stability and binding for different CBL states. Indeed, many methods are not specifically designed to discriminate driver from passenger mutations within the pool of cancer mutations, but rather they perform a task of distinguishing cancer from neutral mutations.  Certainly, the large majority of all mutations detected in cancer genomics studies are likely to be neutral although the collective burden of passenger mutations may also alter the course of tumorigenesis (45). When we analyzed the distribution of all single cancer missense mutations observed in the CBL gene, their average effects on stability or binding were not found to be significantly different from the pool of random mutations. These mutations either mostly constitute true passenger mutations or their oncogenic mechanisms are not directly connected to protein destabilization. The story is quite the opposite for the recurrent mutations as they have on average significantly higher destabilizing effects than random mutations and the majority of them should be drivers. An interesting group of mutations includes those which are highly damaging but found only in one cancer sample (10% of all single cancer mutations). These mutations may represent either rare driver or latent driver mutations (46). Several of these mutations were experimentally tested and are listed in Table 1.  CBL-E2 binding in deciphering the mechanisms of cancer  Our analysis showed that cancer mutations reduce CBL-E2 binding in the active state considerably more than in the inactive state. It is in contrast to the stability-activity tradeoff reported for EGFR and other cancer related proteins (47), where cancer mutations may disrupt autoinhibitory interactions and activate the kinase (48, 49). Importantly, all mutations with experimentally tested high inactivating effects have impacts on both CBL stability and CBL-E2 binding (Table 1). In fact, the correlation between Î”Î”Gfold and Î”Î”Gbind for CBL cancer mutations from COSMIC is positive (R = 0.20 â€“ 0.48 depending on the CBL state, p-value < 0.05). It might seem counterintuitive as some residues maintain stability by sustaining the RING-TKBD autoinhibitory interactions within CBL. This interface overlaps with the CBL-E2 interface and competes against E2 binding (13). One might think that disruption of RING-TKBD interactions and destabilization of CBL might facilitate binding to E2 and therefore lead to CBL activation. A slight activation was actually observed for two mutants, one of which (M374V) was directly located on the CBL-E2 interface. However, as we showed through the positive high correlation between stability and activity changes, this mechanism is rarely observed and the vast majority of mutations disrupt CBL-E3 activity by either destabilizing CBL and/or CBL-E2 binding or by directly affecting phosphorylation of the critical tyrosine Y371.  Overall, our results support the idea that decrease of E3 activity, of the ability to ubiquitinate receptor tyrosine kinases, of CBL stability and/or CBL-E2 binding can give a selective advantage for tumor cells. However, there are several factors that complicate deciphering the mechanisms of action of CBL cancer mutations. CBL mutations can be dominant-negative and mutated CBL might not only change the CBL E3 activity but also might affect the concentration and activity of the wild-type CBL. According to a ratiometric method to identify driver genes in cancer (1), CBL can be regarded as an oncogene since it has several mutation hot spots. On the other hand, as shown in our study and in other studies, cancer driver mutations can inactivate the E3 activity of CBL, so it can also be regarded as a tumor suppressor. Although the latter fact complicates the development of CBL targeted therapies, understanding the delicate balance among different CBL-affected pathways may facilitate the indirect drug targeting of damaged CBL proteins. The current genetics-based frameworks to analyze cancer genome-wide sequence data are necessary but not sufficient for understanding the processes of carcinogenesis and developing informed, targeted therapies. Our approach, which can be applied in general to different proteins of interest, emphasizes the importance of the physics of binding and protein conformational ensembles in deducing the mechanisms of cancer.'"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data['Text'][4]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [],
   "source": [
    "t = {\n",
    "        'Gain-of-function': 'probabilities[0]',\n",
    "        'Inconclusive': 'probabilities[1]',\n",
    "        'Likely Gain-of-function': 'probabilities[2]',\n",
    "        'Likely Loss-of-function': 'probabilities[3]',\n",
    "        'Likely Neutral': 'probabilities[4]',\n",
    "        'Likely Switch-of-function': 'probabilities[5]',\n",
    "        'Loss-of-function': 'probabilities[6]',\n",
    "        'Neutral': 'probabilities[7]',\n",
    "        'Switch-of-function': 'probabilities[8]'\n",
    "    }\n",
    "\n",
    "result = []\n",
    "for key, value in t.items():\n",
    "    result.append((key, value))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('Gain-of-function', 'probabilities[0]'),\n",
       " ('Inconclusive', 'probabilities[1]'),\n",
       " ('Likely Gain-of-function', 'probabilities[2]'),\n",
       " ('Likely Loss-of-function', 'probabilities[3]'),\n",
       " ('Likely Neutral', 'probabilities[4]'),\n",
       " ('Likely Switch-of-function', 'probabilities[5]'),\n",
       " ('Loss-of-function', 'probabilities[6]'),\n",
       " ('Neutral', 'probabilities[7]'),\n",
       " ('Switch-of-function', 'probabilities[8]')]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "result"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "a ={\n",
    "    'Name': 'Karan',\n",
    "    'School': 'CBM',\n",
    "    'Diploma': 'Yes',\n",
    "    'Degree': 'BE'\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Karan'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "a.get('Name')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
